diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 062550f..9369f4b 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -9,7 +9,7 @@
             this.$wrapper = $wrapper;
             this.repLogs = [];
 
-            HelperInstances.set(this, new Helper($wrapper));
+            HelperInstances.set(this, new Helper(this.repLogs));
 
             this.loadRepLogs();
 
@@ -46,7 +46,6 @@
                 for (let repLog of data.items) {
                     this._addRow(repLog);
                 }
-                console.log(this.repLogs, this.repLogs.includes(data.items[0]));
             })
         }
 
@@ -188,13 +187,13 @@
      * A "private" object
      */
     class Helper {
-        constructor($wrapper) {
-            this.$wrapper = $wrapper;
+        constructor(repLogs) {
+            this.repLogs = repLogs;
         }
 
         calculateTotalWeight() {
             return Helper._calculateWeights(
-                this.$wrapper.find('tbody tr')
+                this.repLogs
             );
         }
 
@@ -208,10 +207,10 @@
             return weight + ' lbs';
         }
 
-        static _calculateWeights($elements) {
+        static _calculateWeights(repLogs) {
             let totalWeight = 0;
-            for (let element of $elements) {
-                totalWeight += $(element).data('weight');
+            for (let repLog of repLogs) {
+                totalWeight += repLog.totalWeightLifted;
             }
 
             return totalWeight;
