diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 75182473..87f06599 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -7,7 +7,7 @@
     class RepLogApp {
         constructor($wrapper) {
             this.$wrapper = $wrapper;
-            this.repLogs = new Array();
+            this.repLogs = [];
 
             HelperInstances.set(this, new Helper($wrapper));
 
