diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index f8e6bc4..29e8a1b 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -33,7 +33,7 @@
             var self = this;
             $.ajax({
                 url: Routing.generate('rep_log_list'),
-            }).then((data) => {
+            }).then(data => {
                 $.each(data.items, function(key, repLog) {
                     self._addRow(repLog);
                 });
