diff --git a/web/assets/js/rep_log.js b/web/assets/js/rep_log.js
index af0ade3..ca79a28 100644
--- a/web/assets/js/rep_log.js
+++ b/web/assets/js/rep_log.js
@@ -1,5 +1,5 @@
 const $ = require('jquery');
-const RepLogApp = require('babel-loader!./Components/RepLogApp');
+const RepLogApp = require('./Components/RepLogApp');
 
 $(document).ready(function() {
     var $wrapper = $('.js-rep-log-table');
diff --git a/webpack.config.js b/webpack.config.js
index 21f77c0..0653020 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -11,6 +11,16 @@ module.exports = {
         path: path.resolve(__dirname, 'web', 'build'),
         filename: '[name].js',
     },
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                use: {
+                    loader: 'babel-loader'
+                }
+            }
+        ]
+    },
     plugins: [
         new webpack.ProvidePlugin({
             jQuery: 'jquery',
