diff --git a/webpack.config.js b/webpack.config.js
index 0653020..26069b8 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -15,8 +15,12 @@ module.exports = {
         rules: [
             {
                 test: /\.js$/,
+                exclude: /node_modules/,
                 use: {
-                    loader: 'babel-loader'
+                    loader: 'babel-loader',
+                    options: {
+                        cacheDirectory: true
+                    }
                 }
             }
         ]
