diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index b830966..181c3cf 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -94,6 +94,15 @@
                 );
             },
 
+            updateTotalWeightLifted: function() {
+                var totalWeight = 0;
+                this.$wrapper.find('tbody tr').each(function() {
+                    totalWeight += $(this).data('weight');
+                });
+
+                this.$wrapper.find('.js-total-weight').html(totalWeight);
+            },
+
             handleRepLogDelete: function(e) {
                 e.preventDefault();
 
@@ -105,14 +114,14 @@
 
                 var deleteUrl = $(this).data('url');
                 var $row = $(this).closest('tr');
-                var $totalWeightContainer = RepLogTable.$wrapper.find('.js-total-weight');
-                var newWeight = $totalWeightContainer.html() - $row.data('weight');
                 $.ajax({
                     url: deleteUrl,
                     method: 'DELETE',
                     success: function() {
-                        $row.fadeOut();
-                        $totalWeightContainer.html(newWeight);
+                        $row.fadeOut('normal', function() {
+                            $row.remove();
+                            RepLogTable.updateTotalWeightLifted();
+                        });
                     }
                 });
             },
