diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 02c5773..47e7ae2 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -66,10 +66,21 @@
     {{ parent() }}
 
     <script>
-        $(document).ready(function() {
-            var $table = $('.js-rep-log-table');
+        var RepLogApp = {
+            initialize: function($wrapper) {
+                this.$wrapper = $wrapper;
+
+                this.$wrapper.find('.js-delete-rep-log').on(
+                    'click',
+                    this.handleRepLogDelete
+                );
+                this.$wrapper.find('tbody tr').on(
+                    'click',
+                    this.handleRowClick
+                );
+            },
 
-            $table.find('.js-delete-rep-log').on('click', function (e) {
+            handleRepLogDelete: function(e) {
                 e.preventDefault();
 
                 $(this).addClass('text-danger');
@@ -90,11 +101,16 @@
                         $totalWeightContainer.html(newWeight);
                     }
                 });
-            });
+            },
 
-            $table.find('tbody tr').on('click', function() {
+            handleRowClick: function() {
                 console.log('row clicked!');
-            });
+            }
+        };
+
+        $(document).ready(function() {
+            var $table = $('.js-rep-log-table');
+            RepLogApp.initialize($table);
         });
     </script>
 {% endblock %}
