diff --git a/web/assets/js/layout.js b/web/assets/js/layout.js
index 3628a4e..e9d87b7 100644
--- a/web/assets/js/layout.js
+++ b/web/assets/js/layout.js
@@ -1,7 +1,6 @@
 'use strict';
 
 const $ = require('jquery');
-window.jQuery = $;
 require('bootstrap');
 
 $(document).ready(function() {
diff --git a/webpack.config.js b/webpack.config.js
index 308c607..21f77c0 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -1,4 +1,5 @@
 const path = require('path');
+const webpack = require('webpack');
 
 module.exports = {
     entry: {
@@ -9,5 +10,11 @@ module.exports = {
     output: {
         path: path.resolve(__dirname, 'web', 'build'),
         filename: '[name].js',
-    }
+    },
+    plugins: [
+        new webpack.ProvidePlugin({
+            jQuery: 'jquery',
+            $: 'jquery'
+        })
+    ]
 };
