diff --git a/app/Resources/views/base.html.twig b/app/Resources/views/base.html.twig
index 14167bc..74df11b 100644
--- a/app/Resources/views/base.html.twig
+++ b/app/Resources/views/base.html.twig
@@ -9,7 +9,7 @@
     <title>{% block title %}Find Something to Lift!{% endblock %}</title>
 
     {% block stylesheets %}
-        <link href="//netdna.bootstrapcdn.com/bootstrap/3.0.3/css/bootstrap.min.css" rel="stylesheet">
+        <link rel="stylesheet" href="{{ asset('assets/vendor/bootstrap/dist/css/bootstrap.css') }}" />
         <link href="{{ asset('assets/css/main.css') }}" rel="stylesheet" />
     {% endblock %}
 
@@ -98,6 +98,9 @@
 </div>
 <!-- /container -->
 
-{% block javascripts %}{% endblock %}
+{% block javascripts %}
+    <script src="{{ asset('assets/vendor/jquery/jquery.js') }}"></script>
+    <script src="{{ asset('assets/vendor/bootstrap/dist/js/bootstrap.js') }}"></script>
+{% endblock %}
 </body>
 </html>
diff --git a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
index bafd013..38ecf76 100644
--- a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
+++ b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
@@ -61,4 +61,4 @@
             {{ render(controller('AcmeLiftStuffBundle:Lift:leaderboard')) }}
         </div>
     </div>
-{% endblock %}
\ No newline at end of file
+{% endblock %}
