diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 3b81379b..bc55be7e 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -35,7 +35,7 @@ class RepLogApp extends Component {
                         <th>&nbsp;</th>
                     </tr>
                     </thead>
-                    <RepLogList/>
+                    <RepLogList highlightedRowId={highlightedRowId}/>
                     <tfoot>
                     <tr>
                         <td>&nbsp;</td>
diff --git a/assets/js/RepLog/RepLogList.js b/assets/js/RepLog/RepLogList.js
index d19b895d..188c2195 100644
--- a/assets/js/RepLog/RepLogList.js
+++ b/assets/js/RepLog/RepLogList.js
@@ -2,6 +2,8 @@ import React, { Component } from 'react';
 
 class RepLogList extends Component {
     render() {
+        const { highlightedRowId } = this.props;
+
         const repLogs = [
             { id: 1, reps: 25, itemLabel: 'My Laptop', totalWeightLifted: 112.5 },
             { id: 2, reps: 10, itemLabel: 'Big Fat Cat', totalWeightLifted: 180 },
