diff --git a/assets/js/Components/Button.js b/assets/js/Components/Button.js
index 595101cc..cf8f5d57 100644
--- a/assets/js/Components/Button.js
+++ b/assets/js/Components/Button.js
@@ -6,10 +6,7 @@ export default function Button(props) {
         <button
             className="btn"
             {...props}
-        >{props.text}</button>
+        >{props.children}</button>
     );
 }
 
-Button.propTypes = {
-    text: PropTypes.string.isRequired
-};
diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 6f9c2ac2..a1826b4c 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -86,7 +86,9 @@ export default class RepLogCreator extends Component {
                     {quantityInputError && <span className="help-block">{quantityInputError}</span>}
                 </div>
                 {' '}
-                <Button type="submit" text="I Lifted it!" />
+                <Button type="submit">
+                    I Lifted it!
+                </Button>
             </form>
         );
     }
