diff --git a/assets/js/Components/Button.js b/assets/js/Components/Button.js
index 627c75e..36db003 100644
--- a/assets/js/Components/Button.js
+++ b/assets/js/Components/Button.js
@@ -4,8 +4,15 @@ import PropTypes from 'prop-types';
 export default function Button(props) {
     return (
         <button
-            className="btn"
+            className={`btn ${props.className}`}
             {...props}
         >{props.children}</button>
     );
 }
+Button.propTypes = {
+    className: PropTypes.string
+};
+
+Button.defaultProps = {
+    className: ''
+};
diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 93437c5..18a53cb 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -86,7 +86,7 @@ export default class RepLogCreator extends Component {
                     {quantityInputError && <span className="help-block">{quantityInputError}</span>}
                 </div>
                 {' '}
-                <Button type="submit">
+                <Button type="submit" className="btn-primary">
                     I Lifted it <span className="fa fa-plus-circle"></span>
                 </Button>
             </form>
