diff --git a/assets/js/Components/Button.js b/assets/js/Components/Button.js
index 595101ccc..627c75ee3 100644
--- a/assets/js/Components/Button.js
+++ b/assets/js/Components/Button.js
@@ -6,10 +6,6 @@ export default function Button(props) {
         <button
             className="btn"
             {...props}
-        >{props.text}</button>
+        >{props.children}</button>
     );
 }
-
-Button.propTypes = {
-    text: PropTypes.string.isRequired
-};
diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 6f9c2ac28..1e7ec2d9f 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -86,7 +86,9 @@ export default class RepLogCreator extends Component {
                     {quantityInputError && <span className="help-block">{quantityInputError}</span>}
                 </div>
                 {' '}
-                <Button type="submit" text="I Lifted it!" />
+                <Button type="submit">
+                    I Lifted it <span className="fa fa-plus-circle"></span>
+                </Button>
             </form>
         );
     }
