diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index bc55be7e..e6026389 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -10,7 +10,7 @@ class RepLogApp extends Component {
         };
     }
 
-    handleRowClick(repLogId, event) {
+    handleRowClick(repLogId) {
         this.setState({highlightedRowId: repLogId});
     }
 
@@ -35,7 +35,10 @@ class RepLogApp extends Component {
                         <th>&nbsp;</th>
                     </tr>
                     </thead>
-                    <RepLogList highlightedRowId={highlightedRowId}/>
+                    <RepLogList
+                        highlightedRowId={highlightedRowId}
+                        handleRowClick={this.handleRowClick}
+                    />
                     <tfoot>
                     <tr>
                         <td>&nbsp;</td>
diff --git a/assets/js/RepLog/RepLogList.js b/assets/js/RepLog/RepLogList.js
index 188c2195..a0615e9c 100644
--- a/assets/js/RepLog/RepLogList.js
+++ b/assets/js/RepLog/RepLogList.js
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 
 class RepLogList extends Component {
     render() {
-        const { highlightedRowId } = this.props;
+        const { highlightedRowId, handleRowClick } = this.props;
 
         const repLogs = [
             { id: 1, reps: 25, itemLabel: 'My Laptop', totalWeightLifted: 112.5 },
@@ -16,7 +16,7 @@ class RepLogList extends Component {
                 <tr
                     key={repLog.id}
                     className={highlightedRowId === repLog.id ? 'info' : ''}
-                    onClick={(event) => this.handleRowClick(repLog.id, event)}
+                    onClick={() =>  handleRowClick(repLog.id)}
                 >
                     <td>{repLog.itemLabel}</td>
                     <td>{repLog.reps}</td>
