diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 2d38141b..40cd0bc2 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -8,6 +8,8 @@ export default class RepLogApp extends Component {
         this.state = {
             highlightedRowId: null
         };
+
+        this.handleRowClick = this.handleRowClick.bind(this);
     }
 
     handleRowClick(repLogId) {
