diff --git a/assets/js/Components/RepLogApp.js b/assets/js/Components/RepLogApp.js
deleted file mode 100644
index 43c1035..0000000
--- a/assets/js/Components/RepLogApp.js
+++ /dev/null
@@ -1,214 +0,0 @@
-'use strict';
-
-import Helper from './RepLogHelper';
-import $ from 'jquery';
-import swal from 'sweetalert2';
-import Routing from './Routing';
-
-let HelperInstances = new WeakMap();
-
-class RepLogApp {
-    constructor($wrapper) {
-        this.$wrapper = $wrapper;
-        this.repLogs = [];
-
-        HelperInstances.set(this, new Helper(this.repLogs));
-
-        this.loadRepLogs();
-
-        this.$wrapper.on(
-            'click',
-            '.js-delete-rep-log',
-            this.handleRepLogDelete.bind(this)
-        );
-        this.$wrapper.on(
-            'click',
-            'tbody tr',
-            this.handleRowClick.bind(this)
-        );
-        this.$wrapper.on(
-            'submit',
-            RepLogApp._selectors.newRepForm,
-            this.handleNewFormSubmit.bind(this)
-        );
-    }
-
-    /**
-     * Call like this.selectors
-     */
-    static get _selectors() {
-        return {
-            newRepForm: '.js-new-rep-log-form'
-        }
-    }
-
-    loadRepLogs() {
-        $.ajax({
-            url: Routing.generate('rep_log_list'),
-        }).then(data => {
-            for (let repLog of data.items) {
-                this._addRow(repLog);
-            }
-        })
-    }
-
-    updateTotalWeightLifted() {
-        this.$wrapper.find('.js-total-weight').html(
-            HelperInstances.get(this).getTotalWeightString()
-        );
-    }
-
-    handleRepLogDelete(e) {
-        e.preventDefault();
-
-        const $link = $(e.currentTarget);
-
-        swal({
-            title: 'Delete this log?',
-            text: 'What? Did you not actually lift this?',
-            showCancelButton: true,
-            showLoaderOnConfirm: true,
-            preConfirm: () => this._deleteRepLog($link)
-        });
-    }
-
-    _deleteRepLog($link) {
-        $link.addClass('text-danger');
-        $link.find('.fa')
-            .removeClass('fa-trash')
-            .addClass('fa-spinner')
-            .addClass('fa-spin');
-
-        const deleteUrl = $link.data('url');
-        const $row = $link.closest('tr');
-
-        return $.ajax({
-            url: deleteUrl,
-            method: 'DELETE'
-        }).then(() => {
-            $row.fadeOut('normal', () => {
-                // we need to remove the repLog from this.repLogs
-                // the "key" is the index to this repLog on this.repLogs
-                this.repLogs.splice(
-                    $row.data('key'),
-                    1
-                );
-
-                $row.remove();
-
-                this.updateTotalWeightLifted();
-            });
-        })
-    }
-
-    handleRowClick() {
-        console.log('row clicked!');
-    }
-
-    handleNewFormSubmit(e) {
-        e.preventDefault();
-
-        const $form = $(e.currentTarget);
-        const formData = {};
-
-        for (let fieldData of $form.serializeArray()) {
-            formData[fieldData.name] = fieldData.value
-        }
-
-        this._saveRepLog(formData)
-        .then((data) => {
-            this._clearForm();
-            this._addRow(data);
-        }).catch((errorData) => {
-            this._mapErrorsToForm(errorData.errors);
-        });
-    }
-
-    _saveRepLog(data) {
-        return new Promise((resolve, reject) => {
-            const url = Routing.generate('rep_log_new');
-
-            $.ajax({
-                url,
-                method: 'POST',
-                data: JSON.stringify(data)
-            }).then((data, textStatus, jqXHR) => {
-                $.ajax({
-                    url: jqXHR.getResponseHeader('Location')
-                }).then((data) => {
-                    // we're finally done!
-                    resolve(data);
-                });
-            }).catch((jqXHR) => {
-                const errorData = JSON.parse(jqXHR.responseText);
-
-                reject(errorData);
-            });
-        });
-    }
-
-    _mapErrorsToForm(errorData) {
-        this._removeFormErrors();
-        const $form = this.$wrapper.find(RepLogApp._selectors.newRepForm);
-
-        for (let element of $form.find(':input')) {
-            const fieldName = $(element).attr('name');
-            const $wrapper = $(element).closest('.form-group');
-            if (!errorData[fieldName]) {
-                // no error!
-                return;
-            }
-
-            const $error = $('<span class="js-field-error help-block"></span>');
-            $error.html(errorData[fieldName]);
-            $wrapper.append($error);
-            $wrapper.addClass('has-error');
-        }
-    }
-
-    _removeFormErrors() {
-        const $form = this.$wrapper.find(RepLogApp._selectors.newRepForm);
-        $form.find('.js-field-error').remove();
-        $form.find('.form-group').removeClass('has-error');
-    }
-
-    _clearForm() {
-        this._removeFormErrors();
-
-        const $form = this.$wrapper.find(RepLogApp._selectors.newRepForm);
-        $form[0].reset();
-    }
-
-    _addRow(repLog) {
-        this.repLogs.push(repLog);
-        // destructuring example
-        // let {id, itemLabel, reps, totallyMadeUpKey = 'whatever!'} = repLog;
-        // console.log(id, itemLabel, reps, totallyMadeUpKey);
-
-        const html = rowTemplate(repLog);
-        const $row = $($.parseHTML(html));
-        // store the repLogs index
-        $row.data('key', this.repLogs.length - 1);
-        this.$wrapper.find('tbody').append($row);
-
-        this.updateTotalWeightLifted();
-    }
-}
-
-const rowTemplate = (repLog) => `
-<tr data-weight="${repLog.totalWeightLifted}">
-<td>${repLog.itemLabel}</td>
-<td>${repLog.reps}</td>
-<td>${repLog.totalWeightLifted}</td>
-<td>
-    <a href="#"
-       class="js-delete-rep-log"
-       data-url="${repLog.links._self}"
-    >
-        <span class="fa fa-trash"></span>
-    </a>
-</td>
-</tr>
-`;
-
-export default RepLogApp;
diff --git a/assets/js/Components/RepLogHelper.js b/assets/js/Components/RepLogHelper.js
deleted file mode 100644
index d9d65ca..0000000
--- a/assets/js/Components/RepLogHelper.js
+++ /dev/null
@@ -1,34 +0,0 @@
-'use strict';
-
-class Helper {
-    constructor(repLogs) {
-        this.repLogs = repLogs;
-    }
-
-    calculateTotalWeight() {
-        return Helper._calculateWeights(
-            this.repLogs
-        );
-    }
-
-    getTotalWeightString(maxWeight = 500) {
-        let weight = this.calculateTotalWeight();
-
-        if (weight > maxWeight) {
-            weight = maxWeight + '+';
-        }
-
-        return weight + ' lbs';
-    }
-
-    static _calculateWeights(repLogs) {
-        let totalWeight = 0;
-        for (let repLog of repLogs) {
-            totalWeight += repLog.totalWeightLifted;
-        }
-
-        return totalWeight;
-    }
-}
-
-export default Helper;
diff --git a/assets/js/rep_log.js b/assets/js/rep_log.js
deleted file mode 100644
index c9b9521..0000000
--- a/assets/js/rep_log.js
+++ /dev/null
@@ -1,9 +0,0 @@
-'use strict';
-
-import $ from 'jquery';
-import RepLogApp from './Components/RepLogApp';
-
-$(document).ready(function() {
-    var $wrapper = $('.js-rep-log-table');
-    var repLogApp = new RepLogApp($wrapper);
-});
diff --git a/templates/lift/_form.html.twig b/templates/lift/_form.html.twig
deleted file mode 100644
index 722a71f..0000000
--- a/templates/lift/_form.html.twig
+++ /dev/null
@@ -1,29 +0,0 @@
-<form class="form-inline js-new-rep-log-form" novalidate data-url="{{ path('rep_log_new') }}">
-    <div class="form-group">
-        <label class="sr-only control-label required" for="rep_log_item">
-            What did you lift?
-        </label>
-        <select id="rep_log_item"
-                name="item"
-                required="required"
-                class="form-control">
-            <option value="" selected="selected">What did you lift?</option>
-            <option value="cat">Cat</option>
-            <option value="fat_cat">Big Fat Cat</option>
-            <option value="laptop">My Laptop</option>
-            <option value="coffee_cup">Coffee Cup</option>
-        </select>
-    </div>
-
-    <div class="form-group">
-        <label class="sr-only control-label required" for="rep_log_reps">
-            How many times?
-        </label>
-        <input type="number" id="rep_log_reps"
-               name="reps" required="required"
-               placeholder="How many times?"
-               class="form-control"/>
-    </div>
-
-    <button type="submit" class="btn btn-primary">I Lifted it!</button>
-</form>
diff --git a/templates/lift/index.html.twig b/templates/lift/index.html.twig
index 87ee4dd..e8e3efe 100644
--- a/templates/lift/index.html.twig
+++ b/templates/lift/index.html.twig
@@ -3,43 +3,7 @@
 {% block body %}
     <div class="row">
         <div id="lift-stuff-app"></div>
-    </div>
-
-    <br><br><br><br><br><br>
-
-    <hr />
-
-    <div class="row">
-        <div class="col-md-7 js-rep-log-table">
-            <h2>
-                Lift History
-            </h2>
-
-            <table class="table table-striped">
-                <thead>
-                <tr>
-                    <th>What</th>
-                    <th>How many times?</th>
-                    <th>Weight</th>
-                    <th>&nbsp;</th>
-                </tr>
-                </thead>
-                <tbody>
-                </tbody>
-                <tfoot>
-                    <tr>
-                        <td>&nbsp;</td>
-                        <th>Total</th>
-                        <th class="js-total-weight"></th>
-                        <td>&nbsp;</td>
-                    </tr>
-                </tfoot>
-            </table>
 
-            <div class="js-new-rep-log-form-wrapper">
-                {{ include('lift/_form.html.twig') }}
-            </div>
-        </div>
         <div class="col-md-5">
             <div class="leaderboard">
                 <h2 class="text-center">
@@ -67,6 +31,5 @@
     <script>
         window.REP_LOG_APP_PROPS = {{ repLogAppProps|json_encode|raw }};
     </script>
-    <script src="{{ asset('build/rep_log.js') }}"></script>
     <script src="{{ asset('build/rep_log_react.js') }}"></script>
 {% endblock %}
diff --git a/webpack.config.js b/webpack.config.js
index ed07a8c..169f4c3 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -9,7 +9,6 @@ Encore
     .setPublicPath('/build')
 
     .createSharedEntry('layout', './assets/js/layout.js')
-    .addEntry('rep_log', './assets/js/rep_log.js')
     .addEntry('rep_log_react', './assets/js/rep_log_react.js')
     .addEntry('login', './assets/js/login.js')
 
