diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index c40e3f81..dd11f2cb 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -15,7 +15,7 @@ export default class RepLogApp extends Component {
 
         const repLogElements = repLogs.map((repLog) => {
             return (
-                <tr>
+                <tr key={repLog.id}>
                     <td>{repLog.itemLabel}</td>
                     <td>{repLog.reps}</td>
                     <td>{repLog.totalWeightLifted}</td>
