diff --git a/app/Resources/views/base.html.twig b/app/Resources/views/base.html.twig
index 1205992..5e680a4 100644
--- a/app/Resources/views/base.html.twig
+++ b/app/Resources/views/base.html.twig
@@ -95,6 +95,7 @@
 {% block javascripts %}
     <script src="{{ asset('bundles/fosjsrouting/js/router.js') }}"></script>
     <script src="{{ path('fos_js_routing_js', { callback: 'fos.Router.setData' }) }}"></script>
+    <script src="{{ asset('build/manifest.js') }}"></script>
     <script src="{{ asset('build/layout.js') }}"></script>
 {% endblock %}
 
diff --git a/webpack.config.js b/webpack.config.js
index 5347fcc..316b084 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -104,9 +104,13 @@ module.exports = {
         ]),
 
         new webpack.optimize.CommonsChunkPlugin({
-            // "layout" is an entry file
-            // anything included in layout, is not included in other output files
-            name: 'layout',
+            name: [
+                // "layout" is an entry file
+                // anything included in layout, is not included in other output files
+                'layout',
+                // dumps the manifest into a separate file
+                'manifest'
+            ],
             minChunks: Infinity
         })
     ],
