diff --git a/app/Resources/views/base.html.twig b/app/Resources/views/base.html.twig
index 1205992..a3cb3dc 100644
--- a/app/Resources/views/base.html.twig
+++ b/app/Resources/views/base.html.twig
@@ -95,6 +95,7 @@
 {% block javascripts %}
     <script src="{{ asset('bundles/fosjsrouting/js/router.js') }}"></script>
     <script src="{{ path('fos_js_routing_js', { callback: 'fos.Router.setData' }) }}"></script>
+    <script src="{{ asset('build/vendor.js') }}"></script>
     <script src="{{ asset('build/layout.js') }}"></script>
 {% endblock %}
 
diff --git a/webpack.config.js b/webpack.config.js
index ef77a49..65db612 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -102,6 +102,11 @@ module.exports = {
             // copies to {output}/static
             { from: './assets/static', to: 'static' }
         ]),
+
+        new webpack.optimize.CommonsChunkPlugin({
+            name: 'vendor',
+            minChunks: 2,
+        })
     ],
     devtool: 'inline-source-map'
 };
