diff --git a/webpack.config.js b/webpack.config.js
index 65db612..89fcc6d 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -105,7 +105,9 @@ module.exports = {
 
         new webpack.optimize.CommonsChunkPlugin({
             name: 'vendor',
-            minChunks: 2,
+            minChunks: function (module) {
+                return module.context && module.context.indexOf("node_modules") !== -1;
+            }
         })
     ],
     devtool: 'inline-source-map'
