diff --git a/public/assets/js/RepLogApp.js b/public/assets/js/Components/RepLogApp.js
similarity index 100%
rename from public/assets/js/RepLogApp.js
rename to public/assets/js/Components/RepLogApp.js
diff --git a/public/assets/js/RepLogHelper.js b/public/assets/js/Components/RepLogHelper.js
similarity index 100%
rename from public/assets/js/RepLogHelper.js
rename to public/assets/js/Components/RepLogHelper.js
diff --git a/public/assets/js/rep_log.js b/public/assets/js/rep_log.js
index 1069d4cf0..00bdc5a20 100644
--- a/public/assets/js/rep_log.js
+++ b/public/assets/js/rep_log.js
@@ -1,7 +1,7 @@
 'use strict';
 
 const $ = require('jquery');
-const RepLogApp = require('./RepLogApp');
+const RepLogApp = require('./Components/RepLogApp');
 
 $(document).ready(function() {
     var $wrapper = $('.js-rep-log-table');
