diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 9ef1b539..1d03af4d 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -30,7 +30,7 @@ export default function RepLogs(props) {
                 type="number"
                 value={numberOfHearts}
                 onChange={(e) => {
-                    onHeartChange(e.target.value);
+                    onHeartChange(+e.target.value);
                 }}
             />
 
