diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index d4550eca..e3949d54 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -30,6 +30,8 @@ export default class RepLogs extends Component {
             <div className="col-md-7">
                 <h2>Lift Stuff! {heart}</h2>
 
+                <input type="number" />
+
                 <table className="table table-striped">
                     <thead>
                     <tr>
