diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index f4fc1637c..ef756077f 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -35,7 +35,7 @@ export default function RepLogs(props) {
             <h2>Lift Stuff! {heart}</h2>
 
             <input
-                type="number"
+                type="range"
                 value={numberOfHearts}
                 onChange={(e) => {
                     onHeartChange(+e.target.value);
