diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 6a5f7640..46732249 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -26,6 +26,8 @@ export default function RepLogs(props) {
         <div className="col-md-7">
             <h2>Lift Stuff! {heart}</h2>
 
+            <input type="number" />
+
             <table className="table table-striped">
                 <thead>
                 <tr>
