diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index ce69cb02..4448155e 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -62,6 +62,8 @@ export default class RepLogApp extends Component {
             <div className="col-md-7">
                 <h2>Lift Stuff! {heart}</h2>
 
+                <input type="number" />
+
                 <table className="table table-striped">
                     <thead>
                     <tr>
