diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 82943b0d..7fdd3432 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -38,7 +38,7 @@ export default function RepLogs(props) {
                 type="number"
                 value={numberOfHearts}
                 onChange={(e) => {
-                    onHeartChange(e.target.value);
+                    onHeartChange(+e.target.value);
                 }}
             />
 
