diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index f911b9f2..3265b18c 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -29,7 +29,7 @@ export default function RepLogs(props) {
                 type="number"
                 value={numberOfHearts}
                 onChange={(e) => {
-                    onHeartChange(e.target.value);
+                    onHeartChange(+e.target.value);
                 }}
             />
 
