diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 1d03af4d..ccbd84d9 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -27,7 +27,7 @@ export default function RepLogs(props) {
             <h2>Lift Stuff! {heart}</h2>
 
             <input
-                type="number"
+                type="range"
                 value={numberOfHearts}
                 onChange={(e) => {
                     onHeartChange(+e.target.value);
