diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 009fc4ff..ab3c7dba 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -62,7 +62,7 @@ export default class RepLogApp extends Component {
                 <h2>Lift Stuff! {heart}</h2>
 
                 <input
-                    type="number"
+                    type="range"
                     value={this.state.numberOfHearts}
                     onChange={(e) => {
                         this.setState({numberOfHearts: e.target.value});
