diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 36da0d10..ede6fc9a 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import RepLogList from './RepLogList';
+import RepLogs from './RepLogs';
 import PropTypes from 'prop-types';
 
 export default class RepLogApp extends Component {
@@ -21,40 +22,7 @@ export default class RepLogApp extends Component {
         const { highlightedRowId } = this.state;
         const { withHeart } = this.props;
 
-        let heart = '';
-        if (withHeart) {
-            heart = <span>❤️</span>;
-        }
-
-        return (
-            <div className="col-md-7">
-                <h2>Lift Stuff! {heart}</h2>
-
-                <table className="table table-striped">
-                    <thead>
-                    <tr>
-                        <th>What</th>
-                        <th>How many times?</th>
-                        <th>Weight</th>
-                        <th>&nbsp;</th>
-                    </tr>
-                    </thead>
-                    <RepLogList
-                        highlightedRowId={highlightedRowId}
-                        onRowClick={this.handleRowClick}
-                    />
-                    <tfoot>
-                    <tr>
-                        <td>&nbsp;</td>
-                        <th>Total</th>
-                        <th>TODO</th>
-                        <td>&nbsp;</td>
-                    </tr>
-                    </tfoot>
-                </table>
-            </div>
-
-        );
+        return <RepLogs/>
     }
 }
 
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
new file mode 100644
index 00000000..0474d1b4
--- /dev/null
+++ b/assets/js/RepLog/RepLogs.js
@@ -0,0 +1,37 @@
+import React from 'react';
+
+export default function RepLogs(props) {
+    let heart = '';
+    if (withHeart) {
+        heart = <span>❤️</span>;
+    }
+
+    return (
+        <div className="col-md-7">
+            <h2>Lift Stuff! {heart}</h2>
+
+            <table className="table table-striped">
+                <thead>
+                <tr>
+                    <th>What</th>
+                    <th>How many times?</th>
+                    <th>Weight</th>
+                    <th>&nbsp;</th>
+                </tr>
+                </thead>
+                <RepLogList
+                    highlightedRowId={highlightedRowId}
+                    onRowClick={this.handleRowClick}
+                />
+                <tfoot>
+                <tr>
+                    <td>&nbsp;</td>
+                    <th>Total</th>
+                    <th>TODO</th>
+                    <td>&nbsp;</td>
+                </tr>
+                </tfoot>
+            </table>
+        </div>
+    );
+}
