diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 06f1b937..c713c1f5 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -32,6 +32,8 @@ export default function RepLogs(props) {
         <div className="col-md-7">
             <h2>Lift Stuff! {heart}</h2>
 
+            <input type="number" />
+
             <table className="table table-striped">
                 <thead>
                 <tr>
