diff --git a/public/assets/css/login.css b/assets/css/login.css
similarity index 100%
rename from public/assets/css/login.css
rename to assets/css/login.css
diff --git a/public/assets/css/main.css b/assets/css/main.css
similarity index 100%
rename from public/assets/css/main.css
rename to assets/css/main.css
diff --git a/public/assets/images/dumbbell-mini.png b/assets/images/dumbbell-mini.png
similarity index 100%
rename from public/assets/images/dumbbell-mini.png
rename to assets/images/dumbbell-mini.png
diff --git a/public/assets/images/dumbbell.png b/assets/images/dumbbell.png
similarity index 100%
rename from public/assets/images/dumbbell.png
rename to assets/images/dumbbell.png
diff --git a/public/assets/js/Components/RepLogApp.js b/assets/js/Components/RepLogApp.js
similarity index 100%
rename from public/assets/js/Components/RepLogApp.js
rename to assets/js/Components/RepLogApp.js
diff --git a/public/assets/js/Components/RepLogHelper.js b/assets/js/Components/RepLogHelper.js
similarity index 100%
rename from public/assets/js/Components/RepLogHelper.js
rename to assets/js/Components/RepLogHelper.js
diff --git a/public/assets/js/layout.js b/assets/js/layout.js
similarity index 100%
rename from public/assets/js/layout.js
rename to assets/js/layout.js
diff --git a/public/assets/js/login.js b/assets/js/login.js
similarity index 100%
rename from public/assets/js/login.js
rename to assets/js/login.js
diff --git a/public/assets/js/rep_log.js b/assets/js/rep_log.js
similarity index 100%
rename from public/assets/js/rep_log.js
rename to assets/js/rep_log.js
diff --git a/webpack.config.js b/webpack.config.js
index bc6cc09f..55a8c5b0 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -7,9 +7,9 @@ Encore
     // the public path used by the web server to access the previous directory
     .setPublicPath('/build')
 
-    .addEntry('rep_log', './public/assets/js/rep_log.js')
-    .addEntry('login', './public/assets/js/login.js')
-    .addEntry('layout', './public/assets/js/layout.js')
+    .addEntry('rep_log', './assets/js/rep_log.js')
+    .addEntry('login', './assets/js/login.js')
+    .addEntry('layout', './assets/js/layout.js')
 
     .enableBuildNotifications()
     // fixes modules that expect jQuery to be global
