diff --git a/assets/js/RepLog/RepLogCreatorControlledComponents.js b/assets/js/RepLog/RepLogCreatorControlledComponents.js
index 5e12a19..8ac6cf7 100644
--- a/assets/js/RepLog/RepLogCreatorControlledComponents.js
+++ b/assets/js/RepLog/RepLogCreatorControlledComponents.js
@@ -28,6 +28,10 @@ export default class RepLogCreator extends Component {
         const { onAddRepLog } = this.props;
         const { selectedItemId, quantityValue } = this.state;
 
+        const itemLabel = this.itemOptions.find((option) => {
+            return option.id === this.state.selectedItemId
+        }).text;
+
         if (quantityValue <= 0) {
             this.setState({
                 quantityInputError: 'Please enter a value greater than 0'
@@ -38,7 +42,7 @@ export default class RepLogCreator extends Component {
         }
 
         onAddRepLog(
-            'TODO - just wait a second!',
+            itemLabel,
             quantityValue
         );
 
