diff --git a/assets/js/RepLog/RepLogCreatorControlledComponents.js b/assets/js/RepLog/RepLogCreatorControlledComponents.js
index 7f8fdbe7..230528ba 100644
--- a/assets/js/RepLog/RepLogCreatorControlledComponents.js
+++ b/assets/js/RepLog/RepLogCreatorControlledComponents.js
@@ -31,7 +31,9 @@ export default class RepLogCreator extends Component {
         event.preventDefault();
         const { handleAddRepLog } = this.props;
 
-        const itemLabel = 'TODO - just wait a second!';
+        const itemLabel = this.itemOptions.find((option) => {
+            return option.id === this.state.selectedItemId
+        }).text;
 
         if (this.state.quantityValue <= 0) {
             this.setState({
