diff --git a/assets/js/RepLog/RepLogCreatorWithState.js b/assets/js/RepLog/RepLogCreatorWithState.js
index a584c12b..45b4c3f8 100644
--- a/assets/js/RepLog/RepLogCreatorWithState.js
+++ b/assets/js/RepLog/RepLogCreatorWithState.js
@@ -9,22 +9,38 @@ export default class RepLogCreator extends Component {
             selectedItemId: '',
             quantityValue: 0
         };
+
+        this.handleSelectedItemChange = this.handleSelectedItemChange.bind(this);
+        this.handleQuantityInputChange = this.handleQuantityInputChange.bind(this);
+        this.onFormSubmit = this.onFormSubmit.bind(this);
     }
 
-    render() {
-        const { handleAddRepLog } = this.props;
+    handleSelectedItemChange(event) {
+        this.setState({
+            selectedItemId: event.target.value
+        });
+    }
 
-        function handleSubmit(e) {
-            e.preventDefault();
+    handleQuantityInputChange(event) {
+        this.setState({
+            quantityValue: event.target.value
+        });
+    }
 
-            const itemLabel = itemSelect.options[itemSelect.selectedIndex].text;
+    onFormSubmit(event) {
+        event.preventDefault();
 
-            handleAddRepLog(itemLabel, quantityInput.value);
+        const itemLabel = 'TODO - just wait a second!';
 
-            quantityInput.value = '';
-            itemSelect.selectedIndex = 0;
-        }
+        this.props.handleAddRepLog(itemLabel, this.state.quantityValue);
 
+        this.setState({
+            selectedItemId: '',
+            quantityValue: 0
+        });
+    }
+
+    render() {
         const itemOptions = [
             { id: 'cat', text: 'Cat' },
             { id: 'fat_cat', text: 'Big Fat Cat' },
@@ -33,7 +49,7 @@ export default class RepLogCreator extends Component {
         ];
 
         return (
-            <form className="form-inline" onSubmit={handleSubmit}>
+            <form className="form-inline" onSubmit={this.onFormSubmit}>
                 <div className="form-group">
                     <label className="sr-only control-label required"
                            htmlFor="rep_log_item">
@@ -41,6 +57,7 @@ export default class RepLogCreator extends Component {
                     </label>
                     <select id="rep_log_item"
                             value={this.state.selectedItemId}
+                            onChange={this.handleSelectedItemChange}
                             required="required"
                             className="form-control">
                         <option value="">What did you
@@ -59,6 +76,7 @@ export default class RepLogCreator extends Component {
                     </label>
                     <input type="number" id="rep_log_reps"
                            value={this.state.quantityValue}
+                           onChange={this.handleQuantityInputChange}
                            required="required"
                            placeholder="How many times?"
                            className="form-control"/>
