diff --git a/assets/js/RepLog/RepLogCreatorControlledComponents.js b/assets/js/RepLog/RepLogCreatorControlledComponents.js
new file mode 100644
index 00000000..3ebc2894
--- /dev/null
+++ b/assets/js/RepLog/RepLogCreatorControlledComponents.js
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class RepLogCreator extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            quantityInputError: ''
+        };
+
+        this.quantityInput = React.createRef();
+        this.itemSelect = React.createRef();
+
+        this.itemOptions = [
+            { id: 'cat', text: 'Cat' },
+            { id: 'fat_cat', text: 'Big Fat Cat' },
+            { id: 'laptop', text: 'My Laptop' },
+            { id: 'coffee_cup', text: 'Coffee Cup' },
+        ];
+
+        this.handleFormSubmit = this.handleFormSubmit.bind(this);
+    }
+
+    handleFormSubmit(event) {
+        event.preventDefault();
+        const { onAddRepLog } = this.props;
+
+        const quantityInput = this.quantityInput.current;
+        const itemSelect = this.itemSelect.current;
+
+        if (quantityInput.value <= 0) {
+            this.setState({
+                quantityInputError: 'Please enter a value greater than 0'
+            });
+
+            // don't submit, or clear the form
+            return;
+        }
+
+        onAddRepLog(
+            itemSelect.options[itemSelect.selectedIndex].text,
+            quantityInput.value
+        );
+
+        quantityInput.value = '';
+        itemSelect.selectedIndex = 0;
+        this.setState({
+            quantityInputError: ''
+        });
+    }
+
+    render() {
+        const { quantityInputError } = this.state;
+
+        return (
+            <form onSubmit={this.handleFormSubmit}>
+                <div className="form-group">
+                    <label className="sr-only control-label required"
+                           htmlFor="rep_log_item">
+                        What did you lift?
+                    </label>
+                    <select id="rep_log_item"
+                            ref={this.itemSelect}
+                            required="required"
+                            className="form-control">
+                        <option value="">What did you
+                            lift?
+                        </option>
+                        {this.itemOptions.map(option => {
+                            return <option value={option.id} key={option.id}>{option.text}</option>
+                        })}
+                    </select>
+                </div>
+                {' '}
+                <div className={`form-group ${quantityInputError ? 'has-error' : ''}`}>
+                    <label className="sr-only control-label required"
+                           htmlFor="rep_log_reps">
+                        How many times?
+                    </label>
+                    <input type="number" id="rep_log_reps"
+                           ref={this.quantityInput}
+                           required="required"
+                           placeholder="How many times?"
+                           className="form-control"/>
+                    {quantityInputError && <span className="help-block">{quantityInputError}</span>}
+                </div>
+                {' '}
+                <button type="submit" className="btn btn-primary">I Lifted
+                    it!
+                </button>
+            </form>
+        );
+    }
+}
+
+RepLogCreator.propTypes = {
+    onAddRepLog: PropTypes.func.isRequired,
+};
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index a9c94c9f..4ad48ff9 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -1,7 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import RepLogList from './RepLogList';
-import RepLogCreator from './RepLogCreator';
+//import RepLogCreator from './RepLogCreator';
+import RepLogCreator from './RepLogCreatorControlledComponents';
 
 function calculateTotalWeightLifted(repLogs) {
     let total = 0;
