diff --git a/assets/js/RepLog/RepLogCreatorControlledComponents.js b/assets/js/RepLog/RepLogCreatorControlledComponents.js
index d16b949d..3e0b0047 100644
--- a/assets/js/RepLog/RepLogCreatorControlledComponents.js
+++ b/assets/js/RepLog/RepLogCreatorControlledComponents.js
@@ -27,6 +27,10 @@ export default class RepLogCreator extends Component {
         event.preventDefault();
         const { onAddRepLog } = this.props;
 
+        const itemLabel = this.itemOptions.find((option) => {
+            return option.id === this.state.selectedItemId
+        }).text;
+
         if (this.state.quantityValue <= 0) {
             this.setState({
                 quantityInputError: 'Please enter a value greater than 0'
@@ -37,7 +41,7 @@ export default class RepLogCreator extends Component {
         }
 
         onAddRepLog(
-            'TODO - just wait a second!',
+            itemLabel,
             this.state.quantityValue
         );
 
