diff --git a/assets/js/RepLog/RepLogCreatorControlledComponents.js b/assets/js/RepLog/RepLogCreatorControlledComponents.js
index 6bb3fb0f..7f8fdbe7 100644
--- a/assets/js/RepLog/RepLogCreatorControlledComponents.js
+++ b/assets/js/RepLog/RepLogCreatorControlledComponents.js
@@ -23,18 +23,17 @@ export default class RepLogCreator extends Component {
         ];
 
         this.handleSubmit = this.handleSubmit.bind(this);
+        this.handleSelectedItemChange = this.handleSelectedItemChange.bind(this);
+        this.handleQuantityInputChange = this.handleQuantityInputChange.bind(this);
     }
 
     handleSubmit(event) {
         event.preventDefault();
         const { handleAddRepLog } = this.props;
 
-        const itemSelect = this.itemSelect.current;
-        const quantityInput = this.quantityInput.current;
+        const itemLabel = 'TODO - just wait a second!';
 
-        const itemLabel = itemSelect.options[itemSelect.selectedIndex].text;
-
-        if (quantityInput.value <= 0) {
+        if (this.state.quantityValue <= 0) {
             this.setState({
                 quantityInputError: 'Please enter a value greater than 0'
             });
@@ -43,15 +42,27 @@ export default class RepLogCreator extends Component {
             return;
         }
 
-        handleAddRepLog(itemLabel, quantityInput.value);
+        handleAddRepLog(itemLabel, this.state.quantityValue);
 
-        quantityInput.value = '';
-        itemSelect.selectedIndex = 0;
         this.setState({
+            selectedItemId: '',
+            quantityValue: 0,
             quantityInputError: ''
         });
     }
 
+    handleSelectedItemChange(event) {
+        this.setState({
+            selectedItemId: event.target.value
+        });
+    }
+
+    handleQuantityInputChange(event) {
+        this.setState({
+            quantityValue: event.target.value
+        });
+    }
+
     render() {
         let quantityError = '';
         if (this.state.quantityInputError) {
@@ -67,6 +78,7 @@ export default class RepLogCreator extends Component {
                     </label>
                     <select id="rep_log_item"
                             value={this.state.selectedItemId}
+                            onChange={this.handleSelectedItemChange}
                             required="required"
                             className="form-control">
                         <option value="">What did you
@@ -85,6 +97,7 @@ export default class RepLogCreator extends Component {
                     </label>
                     <input type="number" id="rep_log_reps"
                            value={this.state.quantityValue}
+                           onChange={this.handleQuantityInputChange}
                            required="required"
                            placeholder="How many times?"
                            className="form-control"/>
