diff --git a/assets/js/RepLog/RepLogCreatorControlledComponents.js b/assets/js/RepLog/RepLogCreatorControlledComponents.js
index 03f84ecf5..5e12a19bb 100644
--- a/assets/js/RepLog/RepLogCreatorControlledComponents.js
+++ b/assets/js/RepLog/RepLogCreatorControlledComponents.js
@@ -19,16 +19,16 @@ export default class RepLogCreator extends Component {
         ];
 
         this.handleFormSubmit = this.handleFormSubmit.bind(this);
+        this.handleSelectedItemChange = this.handleSelectedItemChange.bind(this);
+        this.handleQuantityInputChange = this.handleQuantityInputChange.bind(this);
     }
 
     handleFormSubmit(event) {
         event.preventDefault();
         const { onAddRepLog } = this.props;
+        const { selectedItemId, quantityValue } = this.state;
 
-        const quantityInput = this.quantityInput.current;
-        const itemSelect = this.itemSelect.current;
-
-        if (quantityInput.value <= 0) {
+        if (quantityValue <= 0) {
             this.setState({
                 quantityInputError: 'Please enter a value greater than 0'
             });
@@ -38,17 +38,29 @@ export default class RepLogCreator extends Component {
         }
 
         onAddRepLog(
-            itemSelect.options[itemSelect.selectedIndex].text,
-            quantityInput.value
+            'TODO - just wait a second!',
+            quantityValue
         );
 
-        quantityInput.value = '';
-        itemSelect.selectedIndex = 0;
         this.setState({
+            selectedItemId: '',
+            quantityValue: 0,
             quantityInputError: ''
         });
     }
 
+    handleSelectedItemChange(event) {
+        this.setState({
+            selectedItemId: event.target.value
+        });
+    }
+
+    handleQuantityInputChange(event) {
+        this.setState({
+            quantityValue: event.target.value
+        });
+    }
+
     render() {
         const { quantityInputError, selectedItemId, quantityValue } = this.state;
 
@@ -61,6 +73,7 @@ export default class RepLogCreator extends Component {
                     </label>
                     <select id="rep_log_item"
                             value={selectedItemId}
+                            onChange={this.handleSelectedItemChange}
                             required="required"
                             className="form-control">
                         <option value="">What did you
@@ -79,6 +92,7 @@ export default class RepLogCreator extends Component {
                     </label>
                     <input type="number" id="rep_log_reps"
                            value={quantityValue}
+                           onChange={this.handleQuantityInputChange}
                            required="required"
                            placeholder="How many times?"
                            className="form-control"/>
