diff --git a/src/EventSubscriber/ApiCsrfValidationSubscriber.php b/src/EventSubscriber/ApiCsrfValidationSubscriber.php
index ce3f3d0d1..fc45b8224 100644
--- a/src/EventSubscriber/ApiCsrfValidationSubscriber.php
+++ b/src/EventSubscriber/ApiCsrfValidationSubscriber.php
@@ -10,12 +10,17 @@ class ApiCsrfValidationSubscriber implements EventSubscriberInterface
     public function onKernelRequest(GetResponseEvent $event)
     {
         $request = $event->getRequest();
-        dump($request->attributes->all());die;
 
         // no validation needed on safe methods
         if ($request->isMethodSafe(false)) {
             return;
         }
+
+        if (!$request->attributes->get('_is_api')) {
+            return;
+        }
+
+        // TODO
     }
 
     public static function getSubscribedEvents()
