diff --git a/src/Controller/RepLogController.php b/src/Controller/RepLogController.php
index baa1601..7d6425b 100644
--- a/src/Controller/RepLogController.php
+++ b/src/Controller/RepLogController.php
@@ -13,6 +13,7 @@ use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
 
 /**
  * @IsGranted("ROLE_USER")
+ * @Route(defaults={"_is_api": true})
  */
 class RepLogController extends BaseController
 {
diff --git a/src/EventSubscriber/ApiCsrfValidationSubscriber.php b/src/EventSubscriber/ApiCsrfValidationSubscriber.php
index ffab748..9eb5166 100644
--- a/src/EventSubscriber/ApiCsrfValidationSubscriber.php
+++ b/src/EventSubscriber/ApiCsrfValidationSubscriber.php
@@ -14,6 +14,7 @@ class ApiCsrfValidationSubscriber implements EventSubscriberInterface
         }
 
         $request = $event->getRequest();
+        dump($request->attributes->all());die;
 
         // no validation needed on safe methods
         if ($request->isMethodSafe(false)) {
