diff --git a/src/EventSubscriber/ApiCsrfValidationSubscriber.php b/src/EventSubscriber/ApiCsrfValidationSubscriber.php
index 696e53250..a8b166c55 100644
--- a/src/EventSubscriber/ApiCsrfValidationSubscriber.php
+++ b/src/EventSubscriber/ApiCsrfValidationSubscriber.php
@@ -9,7 +9,12 @@ class ApiCsrfValidationSubscriber implements EventSubscriberInterface
 {
     public function onKernelRequest(GetResponseEvent $event)
     {
-        // ...
+        $request = $event->getRequest();
+
+        // no validation needed on safe methods
+        if ($request->isMethodSafe(false)) {
+            return;
+        }
     }
 
     public static function getSubscribedEvents()
