diff --git a/src/EventSubscriber/ApiCsrfValidationSubscriber.php b/src/EventSubscriber/ApiCsrfValidationSubscriber.php
index 479a22a37..63887ff62 100644
--- a/src/EventSubscriber/ApiCsrfValidationSubscriber.php
+++ b/src/EventSubscriber/ApiCsrfValidationSubscriber.php
@@ -25,6 +25,7 @@ class ApiCsrfValidationSubscriber implements EventSubscriberInterface
         }
 
         $request = $event->getRequest();
+        $session = $request->getSession();
 
         // no validation needed on safe methods
         if ($request->isMethodSafe(false)) {
@@ -49,6 +50,9 @@ class ApiCsrfValidationSubscriber implements EventSubscriberInterface
         if (!$this->csrfTokenManager->isTokenValid(new CsrfToken('api', $csrfToken))) {
             throw new AccessDeniedException('Invalid CSRF Token');
         }
+
+        // regenerate the csrf id
+        $this->csrfTokenManager->refreshToken('api');
     }
 
     public static function getSubscribedEvents()
