diff --git a/public/assets/js/layout.js b/public/assets/js/layout.js
index e9d87b70..7c3bb5c3 100644
--- a/public/assets/js/layout.js
+++ b/public/assets/js/layout.js
@@ -2,6 +2,7 @@
 
 const $ = require('jquery');
 require('bootstrap');
+require('../css/main.css');
 
 $(document).ready(function() {
     $('[data-toggle="tooltip"]').tooltip();
diff --git a/templates/base.html.twig b/templates/base.html.twig
index 385bf770..3c87e18d 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -11,7 +11,7 @@
     {% block stylesheets %}
         <link href="https://netdna.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" />
         <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous" />
-        <link href="{{ asset('assets/css/main.css') }}" rel="stylesheet" />
+        <link href="{{ asset('build/layout.css') }}" rel="stylesheet" />
     {% endblock %}
 
     <link rel="icon" type="image/x-icon" href="{{ asset('favicon.ico') }}" />
