diff --git a/web/assets/js/login.js b/web/assets/js/login.js
index e92448b..479c6e3 100644
--- a/web/assets/js/login.js
+++ b/web/assets/js/login.js
@@ -1,8 +1,7 @@
 'use strict';
 
 const $ = require('jquery');
-const css = require('../css/login.css');
-console.log(css);
+require('../css/login.css');
 
 $(document).ready(function() {
     $('.js-recommended-login').on('click', '.js-show-login', function(e) {
diff --git a/webpack.config.js b/webpack.config.js
index 4b51972..ebb8896 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -26,7 +26,8 @@ module.exports = {
             {
                 test: /\.css$/,
                 use: [
-                    'css-loader'
+                    'style-loader',
+                    'css-loader',
                 ]
             }
         ]
