diff --git a/package.json b/package.json
index ad255b944..a196cb980 100644
--- a/package.json
+++ b/package.json
@@ -2,6 +2,7 @@
   "devDependencies": {
     "@symfony/webpack-encore": "^0.19.0",
     "bootstrap": "3",
+    "font-awesome": "4",
     "jquery": "^3.3.1",
     "sweetalert2": "^7.11.0",
     "webpack-notifier": "^1.5.1"
diff --git a/public/assets/js/layout.js b/public/assets/js/layout.js
index 7a3268917..40529aef7 100644
--- a/public/assets/js/layout.js
+++ b/public/assets/js/layout.js
@@ -3,6 +3,7 @@
 const $ = require('jquery');
 require('bootstrap');
 require('bootstrap/dist/css/bootstrap.css');
+require('font-awesome/css/font-awesome.css');
 require('../css/main.css');
 
 $(document).ready(function() {
diff --git a/templates/base.html.twig b/templates/base.html.twig
index de0b848e9..5250b8af3 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -9,7 +9,6 @@
     <title>{% block title %}Find Something to Lift!{% endblock %}</title>
 
     {% block stylesheets %}
-        <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous" />
         <link href="{{ asset('build/layout.css') }}" rel="stylesheet" />
     {% endblock %}
 
diff --git a/yarn.lock b/yarn.lock
index 8f5e0b065..41ed6dce3 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -2083,6 +2083,10 @@ flatten@^1.0.2:
   version "1.0.2"
   resolved "https://registry.yarnpkg.com/flatten/-/flatten-1.0.2.tgz#dae46a9d78fbe25292258cc1e780a41d95c03782"
 
+font-awesome@4:
+  version "4.7.0"
+  resolved "https://registry.yarnpkg.com/font-awesome/-/font-awesome-4.7.0.tgz#8fa8cf0411a1a31afd07b06d2902bb9fc815a133"
+
 for-in@^1.0.1, for-in@^1.0.2:
   version "1.0.2"
   resolved "https://registry.yarnpkg.com/for-in/-/for-in-1.0.2.tgz#81068d295a8142ec0ac726c6e2200c30fb6d5e80"
