diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index ace50cd..1716236 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -69,7 +69,7 @@
                 e.preventDefault();
 
                 $(e.target).addClass('text-danger');
-                console.log(e.currentTarget);
+                console.log(e.currentTarget === this);
             });
 
             $table.find('tbody tr').on('click', function() {
