diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 6860db7..a41dc70 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -28,7 +28,6 @@
                         <td>
                             <a href="#" class="js-delete-rep-log">
                                 <span class="fa fa-trash"></span>
-                                Delete
                             </a>
                         </td>
                     </tr>
