diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 7f6d834..fe98407 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -74,7 +74,7 @@
                 e.preventDefault();
 
                 $(e.target).addClass('text-danger');
-                console.log(e.target);
+                console.log(e.currentTarget);
             });
 
             $table.find('tbody tr').on('click', function() {
