diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 31b50b6..dbb2bf0 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -73,8 +73,7 @@
             $table.find('.js-delete-rep-log').on('click', function (e) {
                 e.preventDefault();
 
-                $(e.target).addClass('text-danger');
-                console.log(e.currentTarget === this);
+                $(this).addClass('text-danger');
             });
 
             $table.find('tbody tr').on('click', function() {
