diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index fe98407..31b50b6 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -74,7 +74,7 @@
                 e.preventDefault();
 
                 $(e.target).addClass('text-danger');
-                console.log(e.currentTarget);
+                console.log(e.currentTarget === this);
             });
 
             $table.find('tbody tr').on('click', function() {
