diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 26dbdcb..c45499f 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -26,7 +26,10 @@
                         <td>{{ repLog.reps }}</td>
                         <td>{{ repLog.totalWeightLifted }}</td>
                         <td>
-                            <a href="#" class="js-delete-rep-log">
+                            <a href="#"
+                               class="js-delete-rep-log"
+                               data-url="{{ path('rep_log_delete', {id: repLog.id}) }}"
+                            >
                                 <span class="fa fa-trash"></span>
                             </a>
                         </td>
@@ -77,6 +80,16 @@
                     .removeClass('fa-trash')
                     .addClass('fa-spinner')
                     .addClass('fa-spin');
+
+                var deleteUrl = $(this).data('url');
+                var $row = $(this).closest('tr');
+                $.ajax({
+                    url: deleteUrl,
+                    method: 'DELETE',
+                    success: function() {
+                        $row.fadeOut();
+                    }
+                });
             });
 
             $table.find('tbody tr').on('click', function() {
