diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index a17ceb4d..32dc3ec3 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -20,6 +20,7 @@ export default class RepLogApp extends Component {
         this.handleRowClick = this.handleRowClick.bind(this);
         this.handleAddRepLog = this.handleAddRepLog.bind(this);
         this.handleHeartChange = this.handleHeartChange.bind(this);
+        this.handleDeleteRepLog = this.handleDeleteRepLog.bind(this);
     }
 
     handleRowClick(repLogId) {
@@ -47,6 +48,10 @@ export default class RepLogApp extends Component {
         });
     }
 
+    handleDeleteRepLog(id) {
+        console.log('todo');
+    }
+
     render() {
         return (
             <RepLogs
@@ -55,6 +60,7 @@ export default class RepLogApp extends Component {
                 onRowClick={this.handleRowClick}
                 onAddRepLog={this.handleAddRepLog}
                 onHeartChange={this.handleHeartChange}
+                onDeleteRepLog={this.handleDeleteRepLog}
             />
         );
     }
diff --git a/assets/js/RepLog/RepLogList.js b/assets/js/RepLog/RepLogList.js
index 119ee426..8e2af791 100644
--- a/assets/js/RepLog/RepLogList.js
+++ b/assets/js/RepLog/RepLogList.js
@@ -2,7 +2,13 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 export default function RepLogList(props) {
-    const { highlightedRowId, onRowClick, repLogs } = props;
+    const { highlightedRowId, onRowClick, onDeleteRepLog, repLogs } = props;
+
+    const handleDeleteClick = function(event, repLogId) {
+        event.preventDefault();
+
+        onDeleteRepLog(repLogId);
+    };
 
     return (
         <tbody>
@@ -16,7 +22,7 @@ export default function RepLogList(props) {
                 <td>{repLog.reps}</td>
                 <td>{repLog.totalWeightLifted}</td>
                 <td>
-                    <a href="#">
+                    <a href="#" onClick={(event) => handleDeleteClick(event, repLog.id) }>
                         <span className="fa fa-trash"></span>
                     </a>
                 </td>
@@ -29,5 +35,6 @@ export default function RepLogList(props) {
 RepLogList.propTypes = {
     highlightedRowId: PropTypes.any,
     onRowClick: PropTypes.func.isRequired,
+    onDeleteRepLog: PropTypes.func.isRequired,
     repLogs: PropTypes.array.isRequired,
 };
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 5074116f..906a890a 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -20,7 +20,7 @@ export default class RepLogs extends Component {
     }
 
     render() {
-        const { withHeart, highlightedRowId, onRowClick, repLogs, onAddRepLog, numberOfHearts, onHeartChange } = this.props;
+        const { withHeart, highlightedRowId, onRowClick, repLogs, onAddRepLog, numberOfHearts, onHeartChange, onDeleteRepLog } = this.props;
 
         let heart = '';
         if (withHeart) {
@@ -51,6 +51,7 @@ export default class RepLogs extends Component {
                     <RepLogList
                         highlightedRowId={highlightedRowId}
                         onRowClick={onRowClick}
+                        onDeleteRepLog={onDeleteRepLog}
                         repLogs={repLogs}
                     />
                     <tfoot>
@@ -80,5 +81,6 @@ RepLogs.propTypes = {
     onAddRepLog: PropTypes.func.isRequired,
     repLogs: PropTypes.array.isRequired,
     numberOfHearts: PropTypes.number.isRequired,
-    onHeartChange: PropTypes.func.isRequired
+    onHeartChange: PropTypes.func.isRequired,
+    onDeleteRepLog: PropTypes.func.isRequired,
 };
