diff --git a/assets/js/RepLog/RepLogList.js b/assets/js/RepLog/RepLogList.js
index 1e622e184..119ee4269 100644
--- a/assets/js/RepLog/RepLogList.js
+++ b/assets/js/RepLog/RepLogList.js
@@ -15,7 +15,11 @@ export default function RepLogList(props) {
                 <td>{repLog.itemLabel}</td>
                 <td>{repLog.reps}</td>
                 <td>{repLog.totalWeightLifted}</td>
-                <td>...</td>
+                <td>
+                    <a href="#">
+                        <span className="fa fa-trash"></span>
+                    </a>
+                </td>
             </tr>
         ))}
         </tbody>
