diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index a49803df..7b40d5b5 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -46,9 +46,11 @@ export default class RepLogApp extends Component {
     handleDeleteRepLog(id) {
         // remove the rep log without mutating state
         // filter returns a new array
-        this.setState({
-            repLogs: this.state.repLogs.filter(repLog => repLog.id !== id)}
-        );
+        this.setState((prevState) => {
+            return {
+                repLogs: prevState.repLogs.filter(repLog => repLog.id !== id)
+            };
+        });
     }
 
     calculateTotalWeightLifted() {
