diff --git a/assets/js/RepLog/RepLogList.js b/assets/js/RepLog/RepLogList.js
index 449fcddf..6f744b19 100644
--- a/assets/js/RepLog/RepLogList.js
+++ b/assets/js/RepLog/RepLogList.js
@@ -15,7 +15,11 @@ export default function RepLogList(props) {
                 <td>{repLog.itemLabel}</td>
                 <td>{repLog.reps}</td>
                 <td>{repLog.totalWeightLifted}</td>
-                <td>...</td>
+                <td>
+                    <a href="#">
+                        <span className="fa fa-trash"></span>
+                    </a>
+                </td>
             </tr>
         ))}
         </tbody>
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 3c7d7414..557ca27a 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -1,8 +1,8 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import RepLogList from './RepLogList';
-//import RepLogCreator from './RepLogCreator';
-import RepLogCreator from './RepLogCreatorControlledComponents';
+import RepLogCreator from './RepLogCreator';
+//import RepLogCreator from './RepLogCreatorControlledComponents';
 
 export default class RepLogs extends Component {
     constructor(props) {
