diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 32dc3ec3..daa8cd3a 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -49,7 +49,11 @@ export default class RepLogApp extends Component {
     }
 
     handleDeleteRepLog(id) {
-        console.log('todo');
+        // remove the rep log without mutating state
+        // filter returns a new array
+        this.setState({
+            repLogs: this.state.repLogs.filter(repLog => repLog.id !== id)}
+        );
     }
 
     render() {
