diff --git a/webpack.config.js b/webpack.config.js
index c948a1a..df2a104 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -2,6 +2,9 @@ const path = require('path');
 const webpack = require('webpack');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 
+const useDevServer = true;
+const publicPath = useDevServer ? 'http://localhost:8080/build/' : '/build/';
+
 const styleLoader = {
     loader: 'style-loader',
     options: {
@@ -27,7 +30,6 @@ const resolveUrlLoader = {
     }
 };
 
-
 module.exports = {
     entry: {
         rep_log: './assets/js/rep_log.js',
@@ -37,7 +39,7 @@ module.exports = {
     output: {
         path: path.resolve(__dirname, 'web', 'build'),
         filename: '[name].js',
-        publicPath: '/build/'
+        publicPath: publicPath,
     },
     module: {
         rules: [
@@ -117,5 +119,6 @@ module.exports = {
     devtool: 'inline-source-map',
     devServer: {
         contentBase: './web',
+        headers: { 'Access-Control-Allow-Origin': '*' },
     }
 };
