diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 6d7888c..c0579e1 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -26,7 +26,9 @@
                         <td>{{ repLog.reps }}</td>
                         <td>{{ repLog.totalWeightLifted }}</td>
                         <td>
-                            &nbsp;
+                            <a href="#" class="js-delete-rep-log">
+                                <span class="fa fa-trash"></span>
+                            </a>
                         </td>
                     </tr>
                 {% else %}
@@ -73,4 +75,14 @@
             {{ include('lift/_leaderboard.html.twig') }}
         </div>
     </div>
+{% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+
+    <script>
+        $('.js-delete-rep-log').on('click', function() {
+            console.log('todo delete!');
+        });
+    </script>
 {% endblock %}
\ No newline at end of file
