diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index ca7451e..65fc78a 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -47,14 +47,6 @@
                 </tfoot>
             </table>
 
-            <script>
-                $(document).ready(function() {
-                    $('.js-delete-rep-log').on('click', function () {
-                        console.log('todo delete!');
-                    });
-                });
-            </script>
-
             {{ include('lift/_form.html.twig') }}
         </div>
         <div class="col-md-5">
@@ -73,4 +65,11 @@
 {% block javascripts %}
     {{ parent() }}
 
+    <script>
+        $(document).ready(function() {
+            $('.js-delete-rep-log').on('click', function () {
+                console.log('todo delete!');
+            });
+        });
+    </script>
 {% endblock %}
