diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 8f06fba..0d9b6a7 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -47,6 +47,14 @@
                 </tfoot>
             </table>
 
+            <script>
+                $(document).ready(function() {
+                    $('.js-delete-rep-log').on('click', function () {
+                        console.log('todo delete!');
+                    });
+                });
+            </script>
+
             {{ form_start(form, {
                 'attr': {
                     'class': 'form-inline',
@@ -80,11 +88,4 @@
 {% block javascripts %}
     {{ parent() }}
 
-    <script>
-        $(document).ready(function() {
-            $('.js-delete-rep-log').on('click', function () {
-                console.log('todo delete!');
-            });
-        });
-    </script>
 {% endblock %}
\ No newline at end of file
