diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 7faccf7..714e7dc 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -47,8 +47,15 @@
                 </tfoot>
             </table>
 
-            {{ include('lift/_form.html.twig') }}
+            <script>
+                $(document).ready(function() {
+                    $('.js-delete-rep-log').on('click', function () {
+                        console.log('todo delete!');
+                    });
+                });
+            </script>
 
+            {{ include('lift/_form.html.twig') }}
         </div>
         <div class="col-md-4 col-md-offset-2">
             <h2>Leaderboard</h2>
@@ -61,11 +68,4 @@
 {% block javascripts %}
     {{ parent() }}
 
-    <script>
-        $(document).ready(function() {
-            $('.js-delete-rep-log').on('click', function () {
-                console.log('todo delete!');
-            });
-        });
-    </script>
 {% endblock %}
\ No newline at end of file
