diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 90766e9..1b9c408 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -67,8 +67,10 @@
     {{ parent() }}
 
     <script>
-        $('.js-delete-rep-log').on('click', function() {
-            console.log('todo delete!');
+        $(document).ready(function() {
+            $('.js-delete-rep-log').on('click', function () {
+                console.log('todo delete!');
+            });
         });
     </script>
 {% endblock %}
