diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index e1bfba9..90766e9 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -26,7 +26,9 @@
                         <td>{{ repLog.reps }}</td>
                         <td>{{ repLog.totalWeightLifted }}</td>
                         <td>
-                            &nbsp;
+                            <a href="#" class="js-delete-rep-log">
+                                <span class="fa fa-trash"></span>
+                            </a>
                         </td>
                     </tr>
                 {% else %}
@@ -60,3 +62,13 @@
         </div>
     </div>
 {% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+
+    <script>
+        $('.js-delete-rep-log').on('click', function() {
+            console.log('todo delete!');
+        });
+    </script>
+{% endblock %}
