diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 9925cf2..f3d78a9 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -62,8 +62,10 @@
     {{ parent() }}
 
     <script>
-        $('.js-delete-rep-log').on('click', function() {
-            console.log('todo delete!');
+        $(document).ready(function() {
+            $('.js-delete-rep-log').on('click', function () {
+                console.log('todo delete!');
+            });
         });
     </script>
 {% endblock %}
\ No newline at end of file
