diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 8af15f5..af67b78 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -72,12 +72,8 @@
             $table.find('.js-delete-rep-log').on('click', function (e) {
                 e.preventDefault();
 
-                $(e.target).addClass('text-danger');
-                console.log(
-                    e.target,
-                    $(e.target)[0] === e.target,
-                    $('.fa-trash')[5] === e.target
-                );
+                //$(e.target).addClass('text-danger');
+                e.target.className = e.target.className+' text-danger';
             });
 
             $table.find('tbody tr').on('click', function() {
