diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index e635f74..3baf238 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -73,6 +73,7 @@
                 e.preventDefault();
 
                 $(e.target).addClass('text-danger');
+                console.log(e.target);
             });
 
             $table.find('tbody tr').on('click', function() {
