diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 9e1be38..9afd0d7 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -86,6 +86,8 @@
 
             $table.find('.js-delete-rep-log').on('click', function () {
                 console.log('todo delete!');
+
+                return false;
             });
 
             $table.find('tbody tr').on('click', function() {
