diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index bf8db1a..894455a 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -67,8 +67,7 @@
             $table.find('.js-delete-rep-log').on('click', function (e) {
                 e.preventDefault();
 
-                //$(e.target).addClass('text-danger');
-                e.target.className = e.target.className+' text-danger';
+                $(e.target).addClass('text-danger');
             });
 
             $table.find('tbody tr').on('click', function() {
