diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index f2f9995..365e034 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -71,6 +71,8 @@
 
             $table.find('.js-delete-rep-log').on('click', function () {
                 console.log('todo delete!');
+
+                return false;
             });
 
             $table.find('tbody tr').on('click', function() {
