diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index dfbbaac..374a38f 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -64,10 +64,11 @@
         $(document).ready(function() {
             var $table = $('.js-rep-log-table');
 
-            $table.find('.js-delete-rep-log').on('click', function () {
-                console.log('todo delete!');
+            $table.find('.js-delete-rep-log').on('click', function (e) {
+                e.preventDefault();
+                e.stopPropagation();
 
-                return false;
+                console.log('todo delete!');
             });
 
             $table.find('tbody tr').on('click', function() {
