diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index d7b204a..268183a 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -68,6 +68,7 @@
                 e.preventDefault();
 
                 $(e.target).addClass('text-danger');
+                console.log(e.target);
             });
 
             $table.find('tbody tr').on('click', function() {
