diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 156fd41..a17cca1 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -70,7 +70,7 @@
                 e.preventDefault();
 
                 $(e.target).addClass('text-danger');
-                console.log(e.target);
+                console.dir(e.target);
             });
 
             $table.find('tbody tr').on('click', function() {
