diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index d9c8e1f..82ac8e2 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -66,7 +66,6 @@
 
             $table.find('.js-delete-rep-log').on('click', function (e) {
                 e.preventDefault();
-                e.stopPropagation();
 
                 console.log('todo delete!');
             });
