diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index c607a05..5aff244 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -72,7 +72,8 @@
                 $(e.target).addClass('text-danger');
                 console.log(
                     e.target,
-                    $(e.target)[0] === e.target
+                    $(e.target)[0] === e.target,
+                    $('.fa-trash')[5] === e.target
                 );
             });
 
