diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 268183a..74dedf3 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -68,7 +68,10 @@
                 e.preventDefault();
 
                 $(e.target).addClass('text-danger');
-                console.log(e.target);
+                console.log(
+                    e.target,
+                    $(e.target)[0] === e.target
+                );
             });
 
             $table.find('tbody tr').on('click', function() {
