diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 8ec5814..b4076b5 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -71,7 +71,8 @@
 
                 $(e.target).addClass('text-danger');
                 console.log(
-                    $(e.target)[0] === e.target
+                    $(e.target)[0] === e.target,
+                    $('.fa-trash')[5] === e.target
                 );
             });
 
