diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 0b37745..8af15f5 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -75,7 +75,8 @@
                 $(e.target).addClass('text-danger');
                 console.log(
                     e.target,
-                    $(e.target)[0] === e.target
+                    $(e.target)[0] === e.target,
+                    $('.fa-trash')[5] === e.target
                 );
             });
 
