diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index eae7493..f63b824 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -71,7 +71,6 @@
 
             $table.find('.js-delete-rep-log').on('click', function (e) {
                 e.preventDefault();
-                e.stopPropagation();
 
                 console.log('todo delete!');
             });
