diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index f696f17..564cdb3 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -69,7 +69,7 @@
             $table.find('.js-delete-rep-log').on('click', function (e) {
                 e.preventDefault();
 
-                console.log('todo delete!');
+                $(e.target).addClass('text-danger');
             });
 
             $table.find('tbody tr').on('click', function() {
