diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index b0870a2..62df3f2 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -88,6 +88,7 @@
                 e.preventDefault();
 
                 $(e.target).addClass('text-danger');
+                console.log(e.target);
             });
 
             $table.find('tbody tr').on('click', function() {
