diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 65fc78a..4570fc6 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -10,7 +10,7 @@
                 </a>
             </h2>
 
-            <table class="table table-striped">
+            <table class="table table-striped js-rep-log-table">
                 <thead>
                 <tr>
                     <th>What</th>
@@ -70,6 +70,10 @@
             $('.js-delete-rep-log').on('click', function () {
                 console.log('todo delete!');
             });
+
+            $('.js-rep-log-table tbody tr').on('click', function() {
+                console.log('row clicked!');
+            });
         });
     </script>
 {% endblock %}
