diff --git a/app/Resources/FOSUserBundle/views/Security/login.html.twig b/app/Resources/FOSUserBundle/views/Security/login.html.twig
index 88951e6..5871e32 100644
--- a/app/Resources/FOSUserBundle/views/Security/login.html.twig
+++ b/app/Resources/FOSUserBundle/views/Security/login.html.twig
@@ -8,6 +8,12 @@
     <script src="{{ asset('build/login.js') }}"></script>
 {% endblock %}
 
+{% block stylesheets %}
+    {{ parent() }}
+
+    <link rel="stylesheet" href="{{ asset('build/login.css') }}">
+{% endblock %}
+
 {% block fos_user_content %}
     <div class="container">
         <div class="wrapper">
diff --git a/app/Resources/views/base.html.twig b/app/Resources/views/base.html.twig
index 5e680a4..87d4fe4 100644
--- a/app/Resources/views/base.html.twig
+++ b/app/Resources/views/base.html.twig
@@ -8,7 +8,9 @@
 
     <title>{% block title %}Find Something to Lift!{% endblock %}</title>
 
-    {% block stylesheets %}{% endblock %}
+    {% block stylesheets %}
+        <link rel="stylesheet" href="{{ asset('build/layout.css') }}">
+    {% endblock %}
 
     <link rel="icon" type="image/x-icon" href="{{ asset('favicon.ico') }}" />
 </head>
diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index ef5c18e..8815d3b 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -59,3 +59,9 @@
         });
     </script>
 {% endblock %}
+
+{% block stylesheets %}
+    {{ parent() }}
+
+    <link rel="stylesheet" href="{{ asset('build/rep_log.css') }}">
+{% endblock %}
diff --git a/package.json b/package.json
index 9fea5ac..2f781c3 100644
--- a/package.json
+++ b/package.json
@@ -13,6 +13,7 @@
     "bootstrap-sass": "^3.3.7",
     "copy-webpack-plugin": "^4.0.1",
     "css-loader": "^0.28.4",
+    "extract-text-webpack-plugin": "^3.0.0",
     "file-loader": "^0.11.2",
     "font-awesome": "^4.7.0",
     "jquery": "^3.2.1",
diff --git a/webpack.config.js b/webpack.config.js
index df2a104..e90ab4f 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -1,6 +1,7 @@
 const path = require('path');
 const webpack = require('webpack');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 const useDevServer = true;
 const publicPath = useDevServer ? 'http://localhost:8080/build/' : '/build/';
@@ -55,19 +56,24 @@ module.exports = {
             },
             {
                 test: /\.css$/,
-                use: [
-                    styleLoader,
-                    cssLoader,
-                ]
+                use: ExtractTextPlugin.extract({
+                    use: [
+                        cssLoader
+                    ],
+                    // use this, if CSS isn't extracted
+                    fallback: styleLoader
+                }),
             },
             {
                 test: /\.scss$/,
-                use: [
-                    styleLoader,
-                    cssLoader,
-                    resolveUrlLoader,
-                    sassLoader,
-                ]
+                use: ExtractTextPlugin.extract({
+                    use: [
+                        cssLoader,
+                        resolveUrlLoader,
+                        sassLoader,
+                    ],
+                    fallback: styleLoader
+                }),
             },
             {
                 test: /\.(png|jpg|jpeg|gif|ico|svg)$/,
@@ -114,7 +120,9 @@ module.exports = {
                 'manifest'
             ],
             minChunks: Infinity
-        })
+        }),
+
+        new ExtractTextPlugin('[name].css'),
     ],
     devtool: 'inline-source-map',
     devServer: {
diff --git a/yarn.lock b/yarn.lock
index d91fb3f..a263e6c 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -195,7 +195,7 @@ async@^1.5.2:
   version "1.5.2"
   resolved "https://registry.yarnpkg.com/async/-/async-1.5.2.tgz#ec6a61ae56480c0c3cb241c95618e20892f9672a"
 
-async@^2.1.2, async@^2.1.5:
+async@^2.1.2, async@^2.1.5, async@^2.4.1:
   version "2.5.0"
   resolved "https://registry.yarnpkg.com/async/-/async-2.5.0.tgz#843190fd6b7357a0b9e1c956edddd5ec8462b54d"
   dependencies:
@@ -1665,6 +1665,15 @@ extglob@^0.3.1:
   dependencies:
     is-extglob "^1.0.0"
 
+extract-text-webpack-plugin@^3.0.0:
+  version "3.0.0"
+  resolved "https://registry.yarnpkg.com/extract-text-webpack-plugin/-/extract-text-webpack-plugin-3.0.0.tgz#90caa7907bc449f335005e3ac7532b41b00de612"
+  dependencies:
+    async "^2.4.1"
+    loader-utils "^1.1.0"
+    schema-utils "^0.3.0"
+    webpack-sources "^1.0.1"
+
 extsprintf@1.0.2:
   version "1.0.2"
   resolved "https://registry.yarnpkg.com/extsprintf/-/extsprintf-1.0.2.tgz#e1080e0658e300b06294990cc70e1502235fd550"
