diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 89fef629..4e85b0f9 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -35,12 +35,10 @@ export default class RepLogApp extends Component {
         this.setState({highlightedRowId: repLogId});
     }
 
-    handleAddRepLog(itemLabel, reps) {
+    handleAddRepLog(item, reps) {
         const newRep = {
-            id: uuid(),
             reps: reps,
-            itemLabel: itemLabel,
-            totalWeightLifted: Math.floor(Math.random() * 50)
+            item: item
         };
 
         createRepLog(newRep)
@@ -49,11 +47,11 @@ export default class RepLogApp extends Component {
             })
         ;
 
-        this.setState(prevState => {
-            const newRepLogs = [...prevState.repLogs, newRep];
-
-            return {repLogs: newRepLogs};
-        })
+        // this.setState(prevState => {
+        //     const newRepLogs = [...prevState.repLogs, newRep];
+        //
+        //     return {repLogs: newRepLogs};
+        // })
     }
 
     handleHeartChange(heartCount) {
diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 3ebc2894..06d44bce 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -39,7 +39,7 @@ export default class RepLogCreator extends Component {
         }
 
         onAddRepLog(
-            itemSelect.options[itemSelect.selectedIndex].text,
+            itemSelect.options[itemSelect.selectedIndex].value,
             quantityInput.value
         );
 
