diff --git a/assets/js/api/rep_log_api.js b/assets/js/api/rep_log_api.js
index f1a234b6..0c925a13 100644
--- a/assets/js/api/rep_log_api.js
+++ b/assets/js/api/rep_log_api.js
@@ -1,21 +1,24 @@
+function doFetch(url, options) {
+    return fetch(url, Object.assign({
+        credentials: 'same-origin',
+    }, options))
+        .then(response => {
+            return response.json();
+        });
+}
+
 /**
  * Returns a promise where the data is the rep log collection
  *
  * @return {Promise<Response>}
  */
 export function getRepLogs() {
-    return fetch('/reps', {
-        credentials: 'same-origin'
-    })
-        .then(response => {
-            return response.json();
-        })
+    return doFetch('/reps')
         .then(data => data.items);
 }
 
 export function deleteRepLog(id) {
-    return fetch(`/reps/${id}`, {
-        credentials: 'same-origin',
+    return doFetch(`/reps/${id}`, {
         method: 'DELETE'
     });
 }
