diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index edadc594..98c024c8 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -2,11 +2,17 @@ import React, { Component } from 'react';
 import RepLogs from './RepLogs';
 import PropTypes from 'prop-types';
 import uuid from 'uuid/v4';
+import { getRepLogs } from '../api/rep_log_api';
 
 export default class RepLogApp extends Component {
     constructor(props) {
         super(props);
 
+        getRepLogs()
+            .then((data) => {
+                console.log(data);
+            });
+
         this.state = {
             highlightedRowId: null,
             repLogs: [
diff --git a/assets/js/api/rep_log_api.js b/assets/js/api/rep_log_api.js
new file mode 100644
index 00000000..e9ba9d2a
--- /dev/null
+++ b/assets/js/api/rep_log_api.js
@@ -0,0 +1,11 @@
+/**
+ * Returns a promise where the data is the rep log collection
+ *
+ * @return {Promise<Response>}
+ */
+export function getRepLogs() {
+    return fetch('/reps')
+        .then(response => {
+            return response.json();
+        });
+}
