diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index c4d65dde..caddd034 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -10,18 +10,9 @@ export default class RepLogApp extends Component {
     constructor(props) {
         super(props);
 
-        getRepLogs()
-            .then((data) => {
-                console.log(data);
-            });
-
         this.state = {
             highlightedRowId: null,
-            repLogs: [
-                { id: uuid(), reps: 25, itemLabel: 'My Laptop', totalWeightLifted: 112.5 },
-                { id: uuid(), reps: 10, itemLabel: 'Big Fat Cat', totalWeightLifted: 180 },
-                { id: uuid(), reps: 4, itemLabel: 'Big Fat Cat', totalWeightLifted: 72 }
-            ],
+            repLogs: [],
             numberOfHearts: 1
         };
 
@@ -30,6 +21,15 @@ export default class RepLogApp extends Component {
         this.handleDeleteRepLog = this.handleDeleteRepLog.bind(this);
     }
 
+    componentDidMount() {
+        getRepLogs()
+            .then((data) => {
+                this.setState({
+                    repLogs: data
+                })
+            });
+    }
+
     handleRowClick(repLogId) {
         this.setState({highlightedRowId: repLogId});
     }
diff --git a/assets/js/api/rep_log_api.js b/assets/js/api/rep_log_api.js
index ea1a68fc..49457f77 100644
--- a/assets/js/api/rep_log_api.js
+++ b/assets/js/api/rep_log_api.js
@@ -9,5 +9,6 @@ export function getRepLogs() {
     })
         .then(response => {
             return response.json();
-        });
+        })
+        .then(data => data.items);
 }
