diff --git a/assets/js/api/rep_log_api.js b/assets/js/api/rep_log_api.js
index e9ba9d2a..ea1a68fc 100644
--- a/assets/js/api/rep_log_api.js
+++ b/assets/js/api/rep_log_api.js
@@ -4,7 +4,9 @@
  * @return {Promise<Response>}
  */
 export function getRepLogs() {
-    return fetch('/reps')
+    return fetch('/reps', {
+        credentials: 'same-origin'
+    })
         .then(response => {
             return response.json();
         });
