diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 5ab6dc7b..2f452a0a 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -8,18 +8,9 @@ export default class RepLogApp extends Component {
     constructor(props) {
         super(props);
 
-        getRepLogs()
-            .then((data) => {
-                console.log(data);
-            });
-
         this.state = {
             highlightedRowId: null,
-            repLogs: [
-                { id: uuid(), reps: 25, itemLabel: 'My Laptop', totalWeightLifted: 112.5 },
-                { id: uuid(), reps: 10, itemLabel: 'Big Fat Cat', totalWeightLifted: 180 },
-                { id: uuid(), reps: 4, itemLabel: 'Big Fat Cat', totalWeightLifted: 72 }
-            ],
+            repLogs: [],
             numberOfHearts: 1
         };
 
@@ -29,6 +20,15 @@ export default class RepLogApp extends Component {
         this.handleDeleteRepLog = this.handleDeleteRepLog.bind(this);
     }
 
+    componentDidMount() {
+        getRepLogs()
+            .then((data) => {
+                this.setState({
+                    repLogs: data
+                })
+            });
+    }
+
     handleRowClick(repLogId) {
         this.setState({highlightedRowId: repLogId});
     }
diff --git a/assets/js/api/rep_log_api.js b/assets/js/api/rep_log_api.js
index ea1a68fc..b59b59a6 100644
--- a/assets/js/api/rep_log_api.js
+++ b/assets/js/api/rep_log_api.js
@@ -8,6 +8,6 @@ export function getRepLogs() {
         credentials: 'same-origin'
     })
         .then(response => {
-            return response.json();
+            return response.json().then((data) => data.items)
         });
 }
