diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index d586e52e..a9f26e60 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -11,7 +11,8 @@ export default class RepLogApp extends Component {
         this.state = {
             highlightedRowId: null,
             repLogs: [],
-            numberOfHearts: 1
+            numberOfHearts: 1,
+            isLoaded: false
         };
 
         this.handleRowClick = this.handleRowClick.bind(this);
@@ -24,7 +25,8 @@ export default class RepLogApp extends Component {
         getRepLogs()
             .then((data) => {
                 this.setState({
-                    repLogs: data
+                    repLogs: data,
+                    isLoaded: true
                 })
             });
     }
diff --git a/assets/js/RepLog/RepLogList.js b/assets/js/RepLog/RepLogList.js
index 8e2af791..669fccf7 100644
--- a/assets/js/RepLog/RepLogList.js
+++ b/assets/js/RepLog/RepLogList.js
@@ -2,7 +2,17 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 export default function RepLogList(props) {
-    const { highlightedRowId, onRowClick, onDeleteRepLog, repLogs } = props;
+    const { highlightedRowId, onRowClick, onDeleteRepLog, repLogs, isLoaded } = props;
+
+    if (!isLoaded) {
+        return (
+            <tbody>
+                <tr>
+                    <td colSpan="4" className="text-center">Loading...</td>
+                </tr>
+            </tbody>
+        );
+    }
 
     const handleDeleteClick = function(event, repLogId) {
         event.preventDefault();
@@ -37,4 +47,5 @@ RepLogList.propTypes = {
     onRowClick: PropTypes.func.isRequired,
     onDeleteRepLog: PropTypes.func.isRequired,
     repLogs: PropTypes.array.isRequired,
+    isLoaded: PropTypes.bool.isRequired,
 };
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 01d7e707..1bb3396e 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -15,7 +15,7 @@ function calculateTotalWeightLifted(repLogs) {
 }
 
 export default function RepLogs(props) {
-    const { withHeart, highlightedRowId, onRowClick, repLogs, onAddRepLog, numberOfHearts, onHeartChange, onDeleteRepLog } = props;
+    const { withHeart, highlightedRowId, onRowClick, repLogs, onAddRepLog, numberOfHearts, onHeartChange, onDeleteRepLog, isLoaded } = props;
 
     let heart = '';
     if (withHeart) {
@@ -48,6 +48,7 @@ export default function RepLogs(props) {
                     onRowClick={onRowClick}
                     onDeleteRepLog={onDeleteRepLog}
                     repLogs={repLogs}
+                    isLoaded={isLoaded}
                 />
                 <tfoot>
                 <tr>
@@ -79,4 +80,5 @@ RepLogs.propTypes = {
     numberOfHearts: PropTypes.number.isRequired,
     onHeartChange: PropTypes.func.isRequired,
     onDeleteRepLog: PropTypes.func.isRequired,
+    isLoaded: PropTypes.bool.isRequired,
 };
