diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 03fd1165..18a8e42d 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import RepLogs from './RepLogs';
 import PropTypes from 'prop-types';
 import uuid from 'uuid/v4';
-import { getRepLogs, deleteRepLog } from '../api/rep_log_api';
+import { getRepLogs, deleteRepLog, createRepLog } from '../api/rep_log_api';
 
 export default class RepLogApp extends Component {
     constructor(props) {
@@ -43,6 +43,12 @@ export default class RepLogApp extends Component {
             totalWeightLifted: Math.floor(Math.random() * 50)
         };
 
+        createRepLog(newRep)
+            .then(data => {
+                console.log(data);
+            })
+        ;
+
         this.setState(prevState => {
             const newRepLogs = [...prevState.repLogs, newRep];
 
diff --git a/assets/js/api/rep_log_api.js b/assets/js/api/rep_log_api.js
index 0c925a13..2fb160e2 100644
--- a/assets/js/api/rep_log_api.js
+++ b/assets/js/api/rep_log_api.js
@@ -22,3 +22,13 @@ export function deleteRepLog(id) {
         method: 'DELETE'
     });
 }
+
+export function createRepLog(repLog) {
+    return doFetch('/reps', {
+        method: 'POST',
+        body: JSON.stringify(repLog),
+        headers: {
+            'Content-Type': 'application/json'
+        }
+    });
+}
\ No newline at end of file
