diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 4e85b0f..2c129b4 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -42,16 +42,14 @@ export default class RepLogApp extends Component {
         };
 
         createRepLog(newRep)
-            .then(data => {
-                console.log(data);
+            .then(repLog => {
+                this.setState(prevState => {
+                    const newRepLogs = [...prevState.repLogs, repLog];
+
+                    return {repLogs: newRepLogs};
+                })
             })
         ;
-
-        // this.setState(prevState => {
-        //     const newRepLogs = [...prevState.repLogs, newRep];
-        //
-        //     return {repLogs: newRepLogs};
-        // })
     }
 
     handleHeartChange(heartCount) {
