diff --git a/assets/js/rep_log_react.js b/assets/js/rep_log_react.js
new file mode 100644
index 0000000..b9cd21c
--- /dev/null
+++ b/assets/js/rep_log_react.js
@@ -0,0 +1 @@
+console.log('Oh hallo React peeps! 🏋️');
diff --git a/templates/lift/index.html.twig b/templates/lift/index.html.twig
index e3dfa10..fb49e8e 100644
--- a/templates/lift/index.html.twig
+++ b/templates/lift/index.html.twig
@@ -57,4 +57,5 @@
     {{ parent() }}
 
     <script src="{{ asset('build/rep_log.js') }}"></script>
+    <script src="{{ asset('build/rep_log_react.js') }}"></script>
 {% endblock %}
diff --git a/webpack.config.js b/webpack.config.js
index 9ccc938..61a5168 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -10,6 +10,7 @@ Encore
 
     .createSharedEntry('layout', './assets/js/layout.js')
     .addEntry('rep_log', './assets/js/rep_log.js')
+    .addEntry('rep_log_react', './assets/js/rep_log_react.js')
     .addEntry('login', './assets/js/login.js')
 
     .enableBuildNotifications()
