diff --git a/webpack.config.js b/webpack.config.js
index 24c8152..c2a07b2 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -34,13 +34,23 @@ module.exports = {
             {
                 test: /\.(png|jpg|jpeg|gif|ico|svg)$/,
                 use: [
-                    'file-loader'
+                    {
+                        loader: 'file-loader',
+                        options: {
+                            name: '[name]-[hash:6].[ext]'
+                        },
+                    }
                 ]
             },
             {
                 test: /\.(woff|woff2|eot|ttf|otf)$/,
                 use: [
-                    'file-loader'
+                    {
+                        loader: 'file-loader',
+                        options: {
+                            name: '[name]-[hash:6].[ext]'
+                        },
+                    }
                 ]
             }
         ]
