diff --git a/app/Resources/views/base.html.twig b/app/Resources/views/base.html.twig
index e7f5b11..9c3869e 100644
--- a/app/Resources/views/base.html.twig
+++ b/app/Resources/views/base.html.twig
@@ -9,7 +9,6 @@
     <title>{% block title %}Find Something to Lift!{% endblock %}</title>
 
     {% block stylesheets %}
-        <link href="https://netdna.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" />
         <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous" />
     {% endblock %}
 
diff --git a/web/assets/js/layout.js b/web/assets/js/layout.js
index 0768bae..9c611a8 100644
--- a/web/assets/js/layout.js
+++ b/web/assets/js/layout.js
@@ -4,6 +4,8 @@ const $ = require('jquery');
 require('bootstrap');
 // make sure the polyfill library is loaded in this main entry
 require('babel-polyfill');
+
+require('bootstrap/dist/css/bootstrap.css');
 require('../css/main.css');
 
 $(document).ready(function() {
diff --git a/webpack.config.js b/webpack.config.js
index d60f8b8..24c8152 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -36,6 +36,12 @@ module.exports = {
                 use: [
                     'file-loader'
                 ]
+            },
+            {
+                test: /\.(woff|woff2|eot|ttf|otf)$/,
+                use: [
+                    'file-loader'
+                ]
             }
         ]
     },
