diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 1998b33..6e38621 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -39,9 +39,9 @@
             $.ajax({
                 url: Routing.generate('rep_log_list'),
             }).then(data => {
-                $.each(data.items, (key, repLog) => {
+                for (let repLog of data.items) {
                     this._addRow(repLog);
-                });
+                }
             })
         }
 
@@ -97,9 +97,10 @@
 
             const $form = $(e.currentTarget);
             const formData = {};
-            $.each($form.serializeArray(), (key, fieldData) => {
+
+            for (let fieldData of $form.serializeArray()) {
                 formData[fieldData.name] = fieldData.value
-            });
+            }
 
             this._saveRepLog(formData)
             .then((data) => {
@@ -137,19 +138,19 @@
             this._removeFormErrors();
             const $form = this.$wrapper.find(RepLogApp._selectors.newRepForm);
 
-            $form.find(':input').each((index, element) => {
+            for (let element of $form.find(':input')) {
                 const fieldName = $(element).attr('name');
                 const $wrapper = $(element).closest('.form-group');
                 if (!errorData[fieldName]) {
                     // no error!
-                    return;
+                    continue;
                 }
 
                 const $error = $('<span class="js-field-error help-block"></span>');
                 $error.html(errorData[fieldName]);
                 $wrapper.append($error);
                 $wrapper.addClass('has-error');
-            });
+            }
         }
 
         _removeFormErrors() {
@@ -203,9 +204,9 @@
 
         static _calculateWeights($elements) {
             let totalWeight = 0;
-            $elements.each((index, element) => {
+            for (let element of $elements) {
                 totalWeight += $(element).data('weight');
-            });
+            }
 
             return totalWeight;
         }
