diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
new file mode 100644
index 00000000..8124d698
--- /dev/null
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -0,0 +1,8 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function RepLogCreator(props) {
+    return (
+        <div>I'm going to be a form when I grow up!</div>
+    );
+}
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index f6288e20..fc7e944e 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import RepLogList from './RepLogList';
+import RepLogCreator from './RepLogCreator';
 
 function calculateTotalWeightLifted(repLogs) {
     let total = 0;
@@ -56,6 +57,8 @@ export default function RepLogs(props) {
                 </tfoot>
             </table>
 
+            <RepLogCreator/>
+
             <form className="form-inline" onSubmit={handleFormSubmit}>
                 <div className="form-group">
                     <label className="sr-only control-label required"
