diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 6c348195..6a897b5f 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -21,7 +21,7 @@ function handleFormSubmit(event) {
 
 export default class RepLogs extends Component {
     render() {
-        const { withHeart, highlightedRowId, onRowClick, repLogs, onNewItemSubmit } = props;
+        const { withHeart, highlightedRowId, onRowClick, repLogs, onNewItemSubmit } = this.props;
 
         let heart = '';
         if (withHeart) {
