diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 0bcfcc5c..63580107 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -21,6 +21,13 @@ export default class RepLogApp extends Component {
         this.setState({highlightedRowId: repLogId});
     }
 
+    handleNewItemSubmit(event) {
+        event.preventDefault();
+
+        console.log('I love when a good form submits!');
+        console.log(event.target);
+    }
+
     calculateTotalWeightLifted() {
         let total = 0;
 
@@ -67,7 +74,7 @@ export default class RepLogApp extends Component {
                     </tfoot>
                 </table>
 
-                <form className="form-inline">
+                <form className="form-inline" onSubmit={this.handleNewItemSubmit}>
                     <div className="form-group">
                         <label className="sr-only control-label required"
                                htmlFor="rep_log_item">
