diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 8fe4c419..cea2ea47 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -68,7 +68,7 @@ export default function RepLogs(props) {
                         <option value="coffee_cup">Coffee Cup</option>
                     </select>
                 </div>
-
+                {' '}
                 <div className="form-group">
                     <label className="sr-only control-label required"
                            htmlFor="rep_log_reps">
@@ -79,7 +79,7 @@ export default function RepLogs(props) {
                            placeholder="How many times?"
                            className="form-control"/>
                 </div>
-
+                {' '}
                 <button type="submit" className="btn btn-primary">I Lifted it!
                 </button>
             </form>
