diff --git a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
index bb67de6..2f38772 100644
--- a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
+++ b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
@@ -49,12 +49,13 @@
         };
 
         RepForm.prototype.initialize = function() {
-            this.$form.on('submit', this.handleSubmit);
+            this.$form.on('submit', $.proxy(this.handleSubmit, this));
         };
 
         RepForm.prototype.handleSubmit = function (e) {
             e.preventDefault();
-            var $form = $(this);
+            var self = this;
+            var $form = this.$form;
             var $errorList = $form.find('.error-list');
 
             $.ajax({
@@ -62,7 +63,7 @@
                 url: $form.attr('action') + '.json',
                 data: $form.serialize(),
                 error: function (jqXhr) {
-                    repForm.parseErrorResponse(jqXhr.responseJSON, $form);
+                    self.parseErrorResponse(jqXhr.responseJSON);
 
                 },
                 success: function () {
@@ -76,7 +77,8 @@
             });
         };
 
-        RepForm.prototype.parseErrorResponse = function(errorResponse, $form) {
+        RepForm.prototype.parseErrorResponse = function(errorResponse) {
+            var $form = this.$form;
             var $errorList = $form.find('.error-list');
 
             // remove any field errors
