diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 55f626f2..9c25fccf 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -22,11 +22,8 @@ export default class RepLogApp extends Component {
         this.setState({highlightedRowId: repLogId});
     }
 
-    handleNewItemSubmit(event) {
-        event.preventDefault();
-
-        console.log('I love when a good form submits!');
-        console.log(event.target.elements.namedItem('reps').value);
+    handleNewItemSubmit() {
+        console.log('todo');
     }
 
     render() {
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 254b3bd8..21c8378f 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -13,6 +13,13 @@ function calculateTotalWeightLifted(repLogs) {
 }
 const calculateTotalWeightFancier = repLogs => repLogs.reduce((total, log) => total + log.totalWeightLifted, 0);
 
+function handleFormSubmit(event) {
+    event.preventDefault();
+
+    console.log('I love when a good form submits!');
+    console.log(event.target.elements.namedItem('reps').value);
+}
+
 export default function RepLogs(props) {
     const { withHeart, highlightedRowId, onRowClick, repLogs, onNewItemSubmit } = props;
 
@@ -49,7 +56,7 @@ export default function RepLogs(props) {
                 </tfoot>
             </table>
 
-            <form className="form-inline" onSubmit={onNewItemSubmit}>
+            <form className="form-inline" onSubmit={handleFormSubmit}>
                 <div className="form-group">
                     <label className="sr-only control-label required"
                            htmlFor="rep_log_item">
