diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index ad484ee65..5798212a3 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -2,6 +2,8 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 export default function RepLogCreator(props) {
+    const { onNewItemSubmit } = props;
+
     function handleFormSubmit(event) {
         event.preventDefault();
 
@@ -49,3 +51,7 @@ export default function RepLogCreator(props) {
         </form>
     );
 }
+
+RepLogCreator.propTypes = {
+    onNewItemSubmit: PropTypes.func.isRequired,
+};
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 7f7c4b448..8f9bf8054 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -50,7 +50,9 @@ export default function RepLogs(props) {
                 </tfoot>
             </table>
 
-            <RepLogCreator/>
+            <RepLogCreator
+                onNewItemSubmit={onNewItemSubmit}
+            />
         </div>
     );
 }
