diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 486dc7c7..88bec148 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -25,7 +25,7 @@ export default class RepLogApp extends Component {
         e.preventDefault();
 
         console.log('I love when a good form submits!');
-        console.log(e.target);
+        console.log(e.target.elements.namedItem('reps').value);
     }
 
     calculateTotalWeightLifted() {
