diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index e1757fe8..8fe4c419 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -49,8 +49,7 @@ export default function RepLogs(props) {
                 </tfoot>
             </table>
 
-            <form className="form-inline js-new-rep-log-form" noValidate
-                  data-url="{{ path('rep_log_new') }}">
+            <form className="form-inline">
                 <div className="form-group">
                     <label className="sr-only control-label required"
                            htmlFor="rep_log_item">
@@ -60,7 +59,7 @@ export default function RepLogs(props) {
                             name="item"
                             required="required"
                             className="form-control">
-                        <option value="" selected="selected">What did you
+                        <option value="">What did you
                             lift?
                         </option>
                         <option value="cat">Cat</option>
