diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 254b3bd8..7d18146d 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -21,6 +21,13 @@ export default function RepLogs(props) {
         heart = <span>❤️</span>;
     }
 
+    function handleFormSubmit(event) {
+        event.preventDefault();
+
+        console.log('I love when a good form submits!');
+        console.log(event.target.elements.namedItem('reps').value);
+    }
+
     return (
         <div className="col-md-7">
             <h2>Lift Stuff! {heart}</h2>
@@ -49,7 +56,7 @@ export default function RepLogs(props) {
                 </tfoot>
             </table>
 
-            <form className="form-inline" onSubmit={onNewItemSubmit}>
+            <form className="form-inline" onSubmit={handleFormSubmit}>
                 <div className="form-group">
                     <label className="sr-only control-label required"
                            htmlFor="rep_log_item">
