diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 8124d69..c1a1bcc 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -3,6 +3,40 @@ import PropTypes from 'prop-types';
 
 export default function RepLogCreator(props) {
     return (
-        <div>I'm going to be a form when I grow up!</div>
+        <form className="form-inline" onSubmit={handleFormSubmit}>
+            <div className="form-group">
+                <label className="sr-only control-label required"
+                       htmlFor="rep_log_item">
+                    What did you lift?
+                </label>
+                <select id="rep_log_item"
+                        name="item"
+                        required="required"
+                        className="form-control">
+                    <option value="">What did you
+                        lift?
+                    </option>
+                    <option value="cat">Cat</option>
+                    <option value="fat_cat">Big Fat Cat</option>
+                    <option value="laptop">My Laptop</option>
+                    <option value="coffee_cup">Coffee Cup</option>
+                </select>
+            </div>
+            {' '}
+            <div className="form-group">
+                <label className="sr-only control-label required"
+                       htmlFor="rep_log_reps">
+                    How many times?
+                </label>
+                <input type="number" id="rep_log_reps"
+                       name="reps" required="required"
+                       placeholder="How many times?"
+                       className="form-control"/>
+            </div>
+            {' '}
+            <button type="submit" className="btn btn-primary">I Lifted
+                it!
+            </button>
+        </form>
     );
 }
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 2e309f1..e0334f5 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -60,42 +60,6 @@ export default function RepLogs(props) {
             </table>
 
             <RepLogCreator/>
-
-            <form className="form-inline" onSubmit={handleFormSubmit}>
-                <div className="form-group">
-                    <label className="sr-only control-label required"
-                           htmlFor="rep_log_item">
-                        What did you lift?
-                    </label>
-                    <select id="rep_log_item"
-                            name="item"
-                            required="required"
-                            className="form-control">
-                        <option value="">What did you
-                            lift?
-                        </option>
-                        <option value="cat">Cat</option>
-                        <option value="fat_cat">Big Fat Cat</option>
-                        <option value="laptop">My Laptop</option>
-                        <option value="coffee_cup">Coffee Cup</option>
-                    </select>
-                </div>
-                {' '}
-                <div className="form-group">
-                    <label className="sr-only control-label required"
-                           htmlFor="rep_log_reps">
-                        How many times?
-                    </label>
-                    <input type="number" id="rep_log_reps"
-                           name="reps" required="required"
-                           placeholder="How many times?"
-                           className="form-control"/>
-                </div>
-                {' '}
-                <button type="submit" className="btn btn-primary">I Lifted
-                    it!
-                </button>
-            </form>
         </div>
     );
 }
