diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 932a7e672..e45802f45 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -2,6 +2,11 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
 export default class RepLogCreator extends Component {
+    constructor(props) {
+        super(props);
+
+        this.handleFormSubmit = this.handleFormSubmit.bind(this);
+    }
 
     handleFormSubmit(event) {
         event.preventDefault();
