diff --git a/app/Resources/views/base.html.twig b/app/Resources/views/base.html.twig
index 74df11b..3e4b8a0 100644
--- a/app/Resources/views/base.html.twig
+++ b/app/Resources/views/base.html.twig
@@ -77,6 +77,7 @@
     </div>
 </div>
 
+<div id="js-notifications">
 {% for type, messages in app.session.flashbag.all() %}
     {% for message in messages %}
         <div class="flash-{{ type }} alert alert-info">
@@ -84,6 +85,7 @@
         </div>
     {% endfor %}
 {% endfor %}
+</div>
 
 {% block hero %}{% endblock %}
 
diff --git a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
index 12b297d..3766f21 100644
--- a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
+++ b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
@@ -46,6 +46,7 @@
             $('form.rep-log-form').on('submit', function(e) {
                 e.preventDefault();
                 var $form = $(this);
+                var $errorList = $form.find('.error-list');
 
                 $.ajax({
                     type: 'POST',
@@ -54,7 +55,7 @@
                     error: function(jqXhr) {
                         // remove any field errors
                         $form.find('.form-group').removeClass('has-error');
-                        var $errorList = $form.find('.error-list');
+
                         $errorList.empty();
 
                         var errors = jqXhr.responseJSON.errors.children;
@@ -70,6 +71,14 @@
                         });
 
                         $errorList.toggleClass('hide', $errorList.html() == '');
+                    },
+                    success: function() {
+                        var $notifications = $('#js-notifications');
+                        $notifications.empty();
+                        $notifications.append('<div class="alert alert-success">Reps saved! You\'re killing it!</div>');
+
+                        $errorList.empty();
+                        $errorList.addClass('hide');
                     }
                 });
             });
