diff --git a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
index 51e29b8..f822ca5 100644
--- a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
+++ b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
@@ -6,6 +6,8 @@
             <h2>Record new Lifts</h2>
 
             <form action="{{ path('lift') }}" method="POST" class="form-inline rep-log-form" novalidate="novalidate">
+                <ul class="error-list alert alert-danger hide"></ul>
+
                 {{ form_errors(form) }}
 
                 {{ form_row(form.item, {
@@ -48,7 +50,22 @@
                 $.ajax({
                     type: 'POST',
                     url: $form.attr('action')+'.json',
-                    data: $form.serialize()
+                    data: $form.serialize(),
+                    error: function(jqXhr) {
+                        var $errorList = $form.find('.error-list');
+                        $errorList.empty();
+
+                        var errors = jqXhr.responseJSON.errors.children;
+                        $.each(errors, function(key, data) {
+                            if (!data.errors) {
+                                return;
+                            }
+
+                            $errorList.append('<li>'+data.errors[0]+'</li>');
+                        });
+
+                        $errorList.toggleClass('hide', $errorList.html() == '');
+                    }
                 });
             });
         });
