diff --git a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
index f822ca5..12b297d 100644
--- a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
+++ b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
@@ -52,6 +52,8 @@
                     url: $form.attr('action')+'.json',
                     data: $form.serialize(),
                     error: function(jqXhr) {
+                        // remove any field errors
+                        $form.find('.form-group').removeClass('has-error');
                         var $errorList = $form.find('.error-list');
                         $errorList.empty();
 
@@ -61,6 +63,9 @@
                                 return;
                             }
 
+                            var $input = $form.find('[name*="['+key+']"]');
+                            $input.closest('.form-group').addClass('has-error');
+
                             $errorList.append('<li>'+data.errors[0]+'</li>');
                         });
 
