diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 90442bcf..c787a32f 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -1,6 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+function handleFormSubmit(event) {
+    event.preventDefault();
+
+    console.log('I love when a good form submits!');
+    console.log(event.target.elements.namedItem('reps').value);
+}
+
 export default function RepLogCreator(props) {
     return (
         <form className="form-inline" onSubmit={handleFormSubmit}>
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 0aca769d..f8445e4d 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -14,13 +14,6 @@ function calculateTotalWeightLifted(repLogs) {
 }
 const calculateTotalWeightFancier = repLogs => repLogs.reduce((total, log) => total + log.totalWeightLifted, 0);
 
-function handleFormSubmit(event) {
-    event.preventDefault();
-
-    console.log('I love when a good form submits!');
-    console.log(event.target.elements.namedItem('reps').value);
-}
-
 export default function RepLogs(props) {
     const { withHeart, highlightedRowId, onRowClick, repLogs, onNewItemSubmit } = props;
 
