diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 9571bad3..44c6b25c 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -22,12 +22,20 @@ export default class RepLogApp extends Component {
         this.setState({highlightedRowId: repLogId});
     }
 
+    handleNewItemSubmit(event) {
+        event.preventDefault();
+
+        console.log('I love when a good form submits!');
+        console.log(event.target);
+    }
+
     render() {
         return (
             <RepLogs
                 {...this.props}
                 {...this.state}
                 onRowClick={this.handleRowClick}
+                onNewItemSubmit={this.handleNewItemSubmit}
             />
         )
     }
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 2841bd14..254b3bd8 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -14,7 +14,7 @@ function calculateTotalWeightLifted(repLogs) {
 const calculateTotalWeightFancier = repLogs => repLogs.reduce((total, log) => total + log.totalWeightLifted, 0);
 
 export default function RepLogs(props) {
-    const { withHeart, highlightedRowId, onRowClick, repLogs } = props;
+    const { withHeart, highlightedRowId, onRowClick, repLogs, onNewItemSubmit } = props;
 
     let heart = '';
     if (withHeart) {
@@ -49,7 +49,7 @@ export default function RepLogs(props) {
                 </tfoot>
             </table>
 
-            <form className="form-inline">
+            <form className="form-inline" onSubmit={onNewItemSubmit}>
                 <div className="form-group">
                     <label className="sr-only control-label required"
                            htmlFor="rep_log_item">
@@ -92,5 +92,6 @@ RepLogs.propTypes = {
     withHeart: PropTypes.bool,
     highlightedRowId: PropTypes.any,
     onRowClick: PropTypes.func.isRequired,
+    onNewItemSubmit: PropTypes.func.isRequired,
     repLogs: PropTypes.array.isRequired
 };
