diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index c1a1bccfd..ad484ee65 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -2,6 +2,15 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 export default function RepLogCreator(props) {
+    function handleFormSubmit(event) {
+        event.preventDefault();
+
+        console.log('I love when a good form submits!');
+        console.log(event.target.elements.namedItem('reps').value);
+
+        onNewItemSubmit('Big Fat Cat', event.target.elements.namedItem('reps').value);
+    }
+
     return (
         <form className="form-inline" onSubmit={handleFormSubmit}>
             <div className="form-group">
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index e0334f555..7f7c4b448 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -22,15 +22,6 @@ export default function RepLogs(props) {
         heart = <span>❤️</span>;
     }
 
-    function handleFormSubmit(event) {
-        event.preventDefault();
-
-        console.log('I love when a good form submits!');
-        console.log(event.target.elements.namedItem('reps').value);
-
-        onNewItemSubmit('Big Fat Cat', event.target.elements.namedItem('reps').value);
-    }
-
     return (
         <div className="col-md-7">
             <h2>Lift Stuff! {heart}</h2>
