diff --git a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
index 6c416cb..bb67de6 100644
--- a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
+++ b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
@@ -42,57 +42,65 @@
     {{ parent() }}
 
     <script>
-        var repForm = {
-            handleSubmit: function (e) {
-                e.preventDefault();
-                var $form = $(this);
-                var $errorList = $form.find('.error-list');
-
-                $.ajax({
-                    type: 'POST',
-                    url: $form.attr('action') + '.json',
-                    data: $form.serialize(),
-                    error: function (jqXhr) {
-                        repForm.parseErrorResponse(jqXhr.responseJSON, $form);
-
-                    },
-                    success: function () {
-                        var $notifications = $('#js-notifications');
-                        $notifications.empty();
-                        $notifications.append('<div class="alert alert-success">Reps saved! You\'re killing it!</div>');
-
-                        $errorList.empty();
-                        $errorList.addClass('hide');
-                    }
-                });
-            },
-
-            parseErrorResponse: function(errorResponse, $form) {
-                var $errorList = $form.find('.error-list');
-
-                // remove any field errors
-                $form.find('.form-group').removeClass('has-error');
-
-                $errorList.empty();
-
-                var errors = errorResponse.errors.children;
-                $.each(errors, function (key, data) {
-                    if (!data.errors) {
-                        return;
-                    }
-
-                    var $input = $form.find('[name*="[' + key + ']"]');
-                    $input.closest('.form-group').addClass('has-error');
-
-                    $errorList.append('<li>' + data.errors[0] + '</li>');
-                });
-
-                $errorList.toggleClass('hide', $errorList.html() == '');
-            }
+        var RepForm = function($form) {
+            this.$form = $form;
+
+            this.initialize();
+        };
+
+        RepForm.prototype.initialize = function() {
+            this.$form.on('submit', this.handleSubmit);
+        };
+
+        RepForm.prototype.handleSubmit = function (e) {
+            e.preventDefault();
+            var $form = $(this);
+            var $errorList = $form.find('.error-list');
+
+            $.ajax({
+                type: 'POST',
+                url: $form.attr('action') + '.json',
+                data: $form.serialize(),
+                error: function (jqXhr) {
+                    repForm.parseErrorResponse(jqXhr.responseJSON, $form);
+
+                },
+                success: function () {
+                    var $notifications = $('#js-notifications');
+                    $notifications.empty();
+                    $notifications.append('<div class="alert alert-success">Reps saved! You\'re killing it!</div>');
+
+                    $errorList.empty();
+                    $errorList.addClass('hide');
+                }
+            });
+        };
+
+        RepForm.prototype.parseErrorResponse = function(errorResponse, $form) {
+            var $errorList = $form.find('.error-list');
+
+            // remove any field errors
+            $form.find('.form-group').removeClass('has-error');
+
+            $errorList.empty();
+
+            var errors = errorResponse.errors.children;
+            $.each(errors, function (key, data) {
+                if (!data.errors) {
+                    return;
+                }
+
+                var $input = $form.find('[name*="[' + key + ']"]');
+                $input.closest('.form-group').addClass('has-error');
+
+                $errorList.append('<li>' + data.errors[0] + '</li>');
+            });
+
+            $errorList.toggleClass('hide', $errorList.html() == '');
         };
 
         jQuery(document).ready(function() {
-            $('form.rep-log-form').on('submit', repForm.handleSubmit);
+            var repForm = new RepForm($('form.rep-log-form'));
         });
     </script>
 {% endblock %}
