diff --git a/assets/js/Components/RepLogApp.js b/assets/js/Components/RepLogApp.js
index b3f7321..b5a829e 100644
--- a/assets/js/Components/RepLogApp.js
+++ b/assets/js/Components/RepLogApp.js
@@ -4,7 +4,7 @@ const Helper = require('./RepLogAppHelper');
 const $ = require('jquery');
 const swal = require('sweetalert2');
 require('sweetalert2/dist/sweetalert2.css');
-const Routing = require('../../../vendor/friendsofsymfony/jsrouting-bundle/Resources/public/js/router');
+const Routing = require('./Routing');
 
 let HelperInstances = new WeakMap();
 
diff --git a/assets/js/Components/Routing.js b/assets/js/Components/Routing.js
new file mode 100644
index 0000000..09c7674
--- /dev/null
+++ b/assets/js/Components/Routing.js
@@ -0,0 +1,5 @@
+/**
+ * For now, we rely on the router.js script tag to be included
+ * in the layout. This is just a helper module to get that object.
+ */
+module.exports = window.Routing;
