diff --git a/assets/js/Components/RepLogApp.js b/assets/js/Components/RepLogApp.js
index 42e6f5b..b3f7321 100644
--- a/assets/js/Components/RepLogApp.js
+++ b/assets/js/Components/RepLogApp.js
@@ -4,6 +4,7 @@ const Helper = require('./RepLogAppHelper');
 const $ = require('jquery');
 const swal = require('sweetalert2');
 require('sweetalert2/dist/sweetalert2.css');
+const Routing = require('../../../vendor/friendsofsymfony/jsrouting-bundle/Resources/public/js/router');
 
 let HelperInstances = new WeakMap();
 
