diff --git a/Gruntfile.js b/Gruntfile.js
index efb176d..540b086 100644
--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -60,12 +60,33 @@ module.exports = function (grunt) {
                      dest: '<%= targetDir %>'
                  }]
              }
-         }
+         },
+
+        // run "Grunt watch" and have it automatically update things when files change
+        watch: {
+            // watch all JS files and run jshint
+            scripts: {
+                files: ['<%= sourceDir %>/js/**'],
+                tasks: ['sync', 'jshint'],
+                options: {
+                    spawn: false
+                }
+            },
+            // watch all CSS files
+            css: {
+                files: '<%= sourceDir %>/css/*.css',
+                tasks: ['sync'],
+                options: {
+                    spawn: false
+                }
+            }
+        }
     });
 
     grunt.loadNpmTasks('grunt-contrib-uglify');
     grunt.loadNpmTasks('grunt-contrib-jshint');
     grunt.loadNpmTasks('grunt-contrib-requirejs');
+    grunt.loadNpmTasks('grunt-contrib-watch');
     grunt.loadNpmTasks('grunt-sync');
 
     grunt.registerTask('default', ['jshint', 'sync']);
diff --git a/app/Resources/views/_requirejs.html.twig b/app/Resources/views/_requirejs.html.twig
index baa779b..77af374 100644
--- a/app/Resources/views/_requirejs.html.twig
+++ b/app/Resources/views/_requirejs.html.twig
@@ -1,8 +1,8 @@
 
-<script src="{{ asset(assetsPath~'/vendor/requirejs/require.js') }}"></script>
+<script src="{{ asset('assets-built/vendor/requirejs/require.js') }}"></script>
 <script>
     requirejs.config({
-        baseUrl: '/{{ assetsPath }}/js',
+        baseUrl: '/assets-built/js',
         paths: {
             routes: '{{ path('fos_js_routing_js', {"callback": "fos.Router.setData"}) }}'
         }
diff --git a/app/Resources/views/base.html.twig b/app/Resources/views/base.html.twig
index 924b36e..4c5e466 100644
--- a/app/Resources/views/base.html.twig
+++ b/app/Resources/views/base.html.twig
@@ -9,8 +9,8 @@
     <title>{% block title %}Find Something to Lift!{% endblock %}</title>
 
     {% block stylesheets %}
-        <link rel="stylesheet" href="{{ asset('assets/vendor/bootstrap/dist/css/bootstrap.css') }}" />
-        <link href="{{ asset('assets/css/main.css') }}" rel="stylesheet" />
+        <link rel="stylesheet" href="{{ asset('assets-built/vendor/bootstrap/dist/css/bootstrap.css') }}" />
+        <link href="{{ asset('assets-built/css/main.css') }}" rel="stylesheet" />
     {% endblock %}
 
     <link rel="icon" type="image/x-icon" href="{{ asset('favicon.ico') }}" />
diff --git a/app/config/config.yml b/app/config/config.yml
index 04529c6..e96bbcc 100644
--- a/app/config/config.yml
+++ b/app/config/config.yml
@@ -2,9 +2,6 @@ imports:
     - { resource: parameters.yml }
     - { resource: security.yml }
 
-parameters:
-    assetsPath: 'assets-built'
-
 framework:
     #esi:             ~
     translator:      { fallback: "%locale%" }
@@ -31,8 +28,6 @@ framework:
 twig:
     debug:            "%kernel.debug%"
     strict_variables: "%kernel.debug%"
-    globals:
-        assetsPath:   %assetsPath%
 
 # Assetic Configuration
 assetic:
diff --git a/app/config/config_dev.yml b/app/config/config_dev.yml
index 9c7a58a..914c34a 100644
--- a/app/config/config_dev.yml
+++ b/app/config/config_dev.yml
@@ -1,9 +1,6 @@
 imports:
     - { resource: config.yml }
 
-parameters:
-    assetsPath: 'assets'
-
 framework:
     router:
         resource: "%kernel.root_dir%/config/routing_dev.yml"
diff --git a/package.json b/package.json
index 1748ced..5ae2f93 100644
--- a/package.json
+++ b/package.json
@@ -15,6 +15,7 @@
     "grunt-contrib-uglify": "~0.2.2",
     "grunt-contrib-requirejs": "~0.4.1",
     "jshint-stylish": "~0.1.4",
-    "grunt-sync": "0.0.5"
+    "grunt-sync": "0.0.5",
+    "grunt-contrib-watch": "~0.5.3"
   }
 }
