diff --git a/Gruntfile.js b/Gruntfile.js
index 74c0b28..5af71a9 100644
--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -1,6 +1,30 @@
 module.exports = function (grunt) {
     grunt.initConfig({
-
+        requirejs: {
+            main: {
+                options: {
+                    mainConfigFile: 'web/assets/js/common.js',
+                    baseUrl: './js',
+                    appDir: 'web/assets',
+                    dir: 'web/assets-built',
+                    paths: {
+                        // prevent routes from being loaded dynamically
+                        'routes': 'empty:'
+                    },
+                    modules: [
+                        {
+                            name: 'common',
+                            include: ['jquery', 'bootstrap']
+                        },
+                        {
+                            name: 'app/lift',
+                            exclude: ['common']
+                        }
+                    ],
+                    optimize: "none"
+                }
+            }
+        }
     });
 
     grunt.loadNpmTasks('grunt-contrib-uglify');
diff --git a/build.js b/build.js
deleted file mode 100644
index 156280d..0000000
--- a/build.js
+++ /dev/null
@@ -1,21 +0,0 @@
-({
-    mainConfigFile: 'web/assets/js/common.js',
-    baseUrl: './js',
-    appDir: 'web/assets',
-    dir: 'web/assets-built',
-    paths: {
-        // prevent routes from being loaded dynamically
-        'routes': 'empty:'
-    },
-    modules: [
-        {
-            name: 'common',
-            include: ['jquery', 'bootstrap']
-        },
-        {
-            name: 'app/lift',
-            exclude: ['common']
-        }
-    ],
-    optimize: "none"
-})
