diff --git a/Gruntfile.js b/Gruntfile.js
index 5af71a9..3421fd6 100644
--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -24,6 +24,19 @@ module.exports = function (grunt) {
                     optimize: "none"
                 }
             }
+        },
+
+        uglify: {
+            build: {
+                files: [
+                    {
+                        expand: true,
+                        cwd: 'web/assets-built',
+                        src: 'js/**/*.js',
+                        dest: 'web/assets-built'
+                    }
+                ]
+            }
         }
     });
 
