diff --git a/Gruntfile.js b/Gruntfile.js
index 6d29ff1..efb176d 100644
--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -67,4 +67,7 @@ module.exports = function (grunt) {
     grunt.loadNpmTasks('grunt-contrib-jshint');
     grunt.loadNpmTasks('grunt-contrib-requirejs');
     grunt.loadNpmTasks('grunt-sync');
+
+    grunt.registerTask('default', ['jshint', 'sync']);
+    grunt.registerTask('production', ['jshint', 'sync', 'requirejs', 'uglify']);
 };
