diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index e3d9e90..4213658 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -68,60 +68,9 @@
 {% block javascripts %}
     {{ parent() }}
 
-    <script>
-        var RepLogApp = {
-            initialize: function($wrapper) {
-                this.$wrapper = $wrapper;
-
-                this.$wrapper.find('.js-delete-rep-log').on(
-                    'click',
-                    this.handleRepLogDelete.bind(this)
-                );
-                this.$wrapper.find('tbody tr').on(
-                    'click',
-                    this.handleRowClick.bind(this)
-                );
-            },
-            updateTotalWeightLifted: function() {
-                var totalWeight = 0;
-                this.$wrapper.find('tbody tr').each(function() {
-                    totalWeight += $(this).data('weight');
-                });
-
-                this.$wrapper.find('.js-total-weight').html(totalWeight);
-            },
-
-            handleRepLogDelete: function(e) {
-                e.preventDefault();
-
-                var $link = $(e.currentTarget);
-
-                $link.addClass('text-danger');
-                $link.find('.fa')
-                    .removeClass('fa-trash')
-                    .addClass('fa-spinner')
-                    .addClass('fa-spin');
-
-                var deleteUrl = $link.data('url');
-                var $row = $link.closest('tr');
-                var self = this;
-                $.ajax({
-                    url: deleteUrl,
-                    method: 'DELETE',
-                    success: function() {
-                        $row.fadeOut('normal', function() {
-                            $(this).remove();
-                            self.updateTotalWeightLifted();
-                        });
-                    }
-                });
-            },
-
-            handleRowClick: function() {
-                console.log('row clicked!');
-            }
-        };
+    <script src="{{ asset('assets/js/RepLogApp.js') }}"></script>
 
+    <script>
         $(document).ready(function() {
             var $table = $('.js-rep-log-table');
             RepLogApp.initialize($table);
diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
new file mode 100644
index 0000000..c94b499
--- /dev/null
+++ b/web/assets/js/RepLogApp.js
@@ -0,0 +1,52 @@
+var RepLogApp = {
+    initialize: function ($wrapper) {
+        this.$wrapper = $wrapper;
+
+        this.$wrapper.find('.js-delete-rep-log').on(
+            'click',
+            this.handleRepLogDelete.bind(this)
+        );
+        this.$wrapper.find('tbody tr').on(
+            'click',
+            this.handleRowClick.bind(this)
+        );
+    },
+    updateTotalWeightLifted: function () {
+        var totalWeight = 0;
+        this.$wrapper.find('tbody tr').each(function () {
+            totalWeight += $(this).data('weight');
+        });
+
+        this.$wrapper.find('.js-total-weight').html(totalWeight);
+    },
+
+    handleRepLogDelete: function (e) {
+        e.preventDefault();
+
+        var $link = $(e.currentTarget);
+
+        $link.addClass('text-danger');
+        $link.find('.fa')
+            .removeClass('fa-trash')
+            .addClass('fa-spinner')
+            .addClass('fa-spin');
+
+        var deleteUrl = $link.data('url');
+        var $row = $link.closest('tr');
+        var self = this;
+        $.ajax({
+            url: deleteUrl,
+            method: 'DELETE',
+            success: function () {
+                $row.fadeOut('normal', function () {
+                    $(this).remove();
+                    self.updateTotalWeightLifted();
+                });
+            }
+        });
+    },
+
+    handleRowClick: function () {
+        console.log('row clicked!');
+    }
+};
