diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 5ca6f1c..be015a2 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -68,6 +68,7 @@
     <script src="{{ asset('assets/js/RepLogApp.js') }}"></script>
 
     <script>
+        console.log(Helper);
         $(document).ready(function() {
             var $table = $('.js-rep-log-table');
             RepLogApp.initialize($table);
