diff --git a/web/assets/js/RepLogTable.js b/web/assets/js/RepLogTable.js
index d0f99f4..1e4be40 100644
--- a/web/assets/js/RepLogTable.js
+++ b/web/assets/js/RepLogTable.js
@@ -1,6 +1,7 @@
 var RepLogTable = {
     initialize: function ($table) {
         this.$wrapper = $table;
+        helper.initialize(this.$wrapper);
 
         this.$wrapper.find('.js-delete-rep-log').on(
             'click',
@@ -13,7 +14,7 @@ var RepLogTable = {
     },
     updateTotalWeightLifted: function () {
         this.$wrapper.find('.js-total-weight').html(
-            this._calculateTotalWeight()
+            helper.calculateTotalWeight()
         );
     },
 
@@ -45,9 +46,18 @@ var RepLogTable = {
 
     handleRowClick: function () {
         console.log('row clicked!');
+    }
+};
+
+/**
+ * A "private" object
+ */
+var helper = {
+    initialize: function ($wrapper) {
+        this.$wrapper = $wrapper;
     },
 
-    _calculateTotalWeight: function() {
+    calculateTotalWeight: function() {
         var totalWeight = 0;
         this.$wrapper.find('tbody tr').each(function () {
             totalWeight += $(this).data('weight');
