diff --git a/app/Resources/views/base.html.twig b/app/Resources/views/base.html.twig
index 5af3b1e..e7f5b11 100644
--- a/app/Resources/views/base.html.twig
+++ b/app/Resources/views/base.html.twig
@@ -11,7 +11,6 @@
     {% block stylesheets %}
         <link href="https://netdna.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" />
         <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous" />
-        <link href="{{ asset('assets/css/main.css') }}" rel="stylesheet" />
     {% endblock %}
 
     <link rel="icon" type="image/x-icon" href="{{ asset('favicon.ico') }}" />
diff --git a/web/assets/js/layout.js b/web/assets/js/layout.js
index 6df59b1..0768bae 100644
--- a/web/assets/js/layout.js
+++ b/web/assets/js/layout.js
@@ -4,6 +4,7 @@ const $ = require('jquery');
 require('bootstrap');
 // make sure the polyfill library is loaded in this main entry
 require('babel-polyfill');
+require('../css/main.css');
 
 $(document).ready(function() {
     $('[data-toggle="tooltip"]').tooltip();
