diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 24af9f63..2a653daa 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -32,15 +32,14 @@ export default class RepLogApp extends Component {
         const quantityInput = this.quantityInput.current;
         const itemSelect = this.itemSelect.current;
 
-        const repLogs = this.state.repLogs;
         const newRep = {
             id: uuid(),
             reps: quantityInput.value,
             itemLabel: itemSelect.options[itemSelect.selectedIndex].text,
             totalWeightLifted: Math.floor(Math.random() * 50)
         };
-        repLogs.push(newRep);
-        this.setState({repLogs: repLogs});
+        const newRepLogs = [...this.state.repLogs, newRep];
+        this.setState({repLogs: newRepLogs});
 
         quantityInput.value = '';
         itemSelect.selectedIndex = 0;
