diff --git a/assets/js/Components/RepLogApp.js b/assets/js/Components/RepLogApp.js
index badc2da..58ec68c 100644
--- a/assets/js/Components/RepLogApp.js
+++ b/assets/js/Components/RepLogApp.js
@@ -5,7 +5,7 @@ const $ = require('jquery');
 const swal = require('sweetalert2');
 require('sweetalert2/dist/sweetalert2.css');
 const Routing = require('./Routing');
-import _ from 'lodash';
+import { random } from 'lodash';
 
 let HelperInstances = new WeakMap();
 
@@ -181,7 +181,7 @@ class RepLogApp {
         $form[0].reset();
 
         // pre-fill with a random rep number
-        $form.find('[name="reps"]').val(_.random(1, 10));
+        $form.find('[name="reps"]').val(random(1, 10));
     }
 
     _addRow(repLog) {
