diff --git a/assets/js/Components/RepLogApp.js b/assets/js/Components/RepLogApp.js
index b5a829e..5ce804a 100644
--- a/assets/js/Components/RepLogApp.js
+++ b/assets/js/Components/RepLogApp.js
@@ -5,6 +5,7 @@ const $ = require('jquery');
 const swal = require('sweetalert2');
 require('sweetalert2/dist/sweetalert2.css');
 const Routing = require('./Routing');
+const _ = require('lodash');
 
 let HelperInstances = new WeakMap();
 
@@ -18,6 +19,7 @@ class RepLogApp {
         for (let repLog of initialRepLogs) {
             this._addRow(repLog);
         }
+        this._clearForm();
 
         this.$wrapper.on(
             'click',
@@ -177,6 +179,9 @@ class RepLogApp {
 
         const $form = this.$wrapper.find(RepLogApp._selectors.newRepForm);
         $form[0].reset();
+
+        // pre-fill with a random rep number
+        $form.find('[name="reps"]').val(_.random(1, 10));
     }
 
     _addRow(repLog) {
diff --git a/package.json b/package.json
index 573a018..0120d4d 100644
--- a/package.json
+++ b/package.json
@@ -16,6 +16,7 @@
     "file-loader": "^0.11.2",
     "font-awesome": "^4.7.0",
     "jquery": "^3.2.1",
+    "lodash": "^4.17.4",
     "node-sass": "^4.5.3",
     "resolve-url-loader": "^2.1.0",
     "sass-loader": "^6.0.6",
diff --git a/yarn.lock b/yarn.lock
index d82c01e..56913c9 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -2248,7 +2248,7 @@ lodash.uniq@^4.5.0:
   version "4.5.0"
   resolved "https://registry.yarnpkg.com/lodash.uniq/-/lodash.uniq-4.5.0.tgz#d0225373aeb652adc1bc82e4945339a842754773"
 
-lodash@^4.0.0, lodash@^4.14.0, lodash@^4.2.0, lodash@^4.3.0, lodash@~4.17.4:
+lodash@^4.0.0, lodash@^4.14.0, lodash@^4.17.4, lodash@^4.2.0, lodash@^4.3.0, lodash@~4.17.4:
   version "4.17.4"
   resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.4.tgz#78203a4d1c328ae1d86dca6460e369b57f4055ae"
 
