diff --git a/assets/js/Components/RepLogApp.js b/assets/js/Components/RepLogApp.js
index 5ce804a..af49f15 100644
--- a/assets/js/Components/RepLogApp.js
+++ b/assets/js/Components/RepLogApp.js
@@ -5,7 +5,7 @@ const $ = require('jquery');
 const swal = require('sweetalert2');
 require('sweetalert2/dist/sweetalert2.css');
 const Routing = require('./Routing');
-const _ = require('lodash');
+const _ = require('lodash/number');
 
 let HelperInstances = new WeakMap();
 
