diff --git a/assets/js/Components/RepLogApp.js b/assets/js/Components/RepLogApp.js
index 58ec68c..79cbbb4 100644
--- a/assets/js/Components/RepLogApp.js
+++ b/assets/js/Components/RepLogApp.js
@@ -5,7 +5,7 @@ const $ = require('jquery');
 const swal = require('sweetalert2');
 require('sweetalert2/dist/sweetalert2.css');
 const Routing = require('./Routing');
-import { random } from 'lodash';
+import random from 'lodash/random';
 
 let HelperInstances = new WeakMap();
 
