diff --git a/assets/js/Components/RepLogApp.js b/assets/js/Components/RepLogApp.js
index af49f15..badc2da 100644
--- a/assets/js/Components/RepLogApp.js
+++ b/assets/js/Components/RepLogApp.js
@@ -5,7 +5,7 @@ const $ = require('jquery');
 const swal = require('sweetalert2');
 require('sweetalert2/dist/sweetalert2.css');
 const Routing = require('./Routing');
-const _ = require('lodash/number');
+import _ from 'lodash';
 
 let HelperInstances = new WeakMap();
 
