diff --git a/assets/js/Components/RepLogApp.js b/assets/js/Components/RepLogApp.js
index 99eec81a..43c10356 100644
--- a/assets/js/Components/RepLogApp.js
+++ b/assets/js/Components/RepLogApp.js
@@ -1,9 +1,9 @@
 'use strict';
 
 import Helper from './RepLogHelper';
-const $ = require('jquery');
-const swal = require('sweetalert2');
-const Routing = require('./Routing');
+import $ from 'jquery';
+import swal from 'sweetalert2';
+import Routing from './Routing';
 
 let HelperInstances = new WeakMap();
 
@@ -211,4 +211,4 @@ const rowTemplate = (repLog) => `
 </tr>
 `;
 
-module.exports = RepLogApp;
+export default RepLogApp;
diff --git a/assets/js/Components/Routing.js b/assets/js/Components/Routing.js
index 09c7674a..1e184886 100644
--- a/assets/js/Components/Routing.js
+++ b/assets/js/Components/Routing.js
@@ -2,4 +2,4 @@
  * For now, we rely on the router.js script tag to be included
  * in the layout. This is just a helper module to get that object.
  */
-module.exports = window.Routing;
+export default window.Routing;
diff --git a/assets/js/layout.js b/assets/js/layout.js
index 0cc3a1ac..8d4c5cec 100644
--- a/assets/js/layout.js
+++ b/assets/js/layout.js
@@ -1,10 +1,10 @@
 'use strict';
 
-const $ = require('jquery');
-require('bootstrap');
-require('bootstrap/dist/css/bootstrap.css');
-require('font-awesome/css/font-awesome.css');
-require('../css/main.scss');
+import $ from 'jquery';
+import 'bootstrap';
+import 'bootstrap/dist/css/bootstrap.css';
+import 'font-awesome/css/font-awesome.css';
+import '../css/main.scss';
 
 $(document).ready(function() {
     $('[data-toggle="tooltip"]').tooltip();
diff --git a/assets/js/login.js b/assets/js/login.js
index 479c6e39..afb516d4 100644
--- a/assets/js/login.js
+++ b/assets/js/login.js
@@ -1,7 +1,7 @@
 'use strict';
 
-const $ = require('jquery');
-require('../css/login.css');
+import $ from 'jquery';
+import '../css/login.css';
 
 $(document).ready(function() {
     $('.js-recommended-login').on('click', '.js-show-login', function(e) {
diff --git a/assets/js/rep_log.js b/assets/js/rep_log.js
index 00bdc5a2..c9b9521d 100644
--- a/assets/js/rep_log.js
+++ b/assets/js/rep_log.js
@@ -1,7 +1,7 @@
 'use strict';
 
-const $ = require('jquery');
-const RepLogApp = require('./Components/RepLogApp');
+import $ from 'jquery';
+import RepLogApp from './Components/RepLogApp';
 
 $(document).ready(function() {
     var $wrapper = $('.js-rep-log-table');
