diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index f07a334bc..21137a380 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -16,7 +16,7 @@ export default class RepLogApp extends Component {
             isSavingNewRepLog: false,
             successMessage: '',
             newRepLogValidationErrorMessage: '',
-            itemOptions: this.props.itemOptions
+            itemOptions: this.props.itemOptions,
         };
         this.clearSuccessMessageTimeout = 0;
 
diff --git a/assets/js/rep_log_react.js b/assets/js/rep_log_react.js
index 1a65693b4..3ed1614c1 100644
--- a/assets/js/rep_log_react.js
+++ b/assets/js/rep_log_react.js
@@ -4,18 +4,10 @@ import RepLogApp from './RepLog/RepLogApp';
 
 const shouldShowHeart = true;
 
-const itemOptions = [
-    {id: 'cat', text: 'Cat'},
-    {id: 'fat_cat', text: 'Big Fat Cat'},
-    {id: 'laptop', text: 'My Laptop'},
-    {id: 'coffee_cup', text: 'Coffee Cup'},
-    {id: 'invalid_item', text: 'Dark Matter'}
-];
-
 render(
     <RepLogApp
         withHeart={shouldShowHeart}
-        itemOptions={itemOptions}
+        itemOptions={window.REP_LOG_APP_PROPS.itemOptions}
     />,
     document.getElementById('lift-stuff-app')
 );
diff --git a/templates/lift/index.html.twig b/templates/lift/index.html.twig
index 71218baab..d31e05817 100644
--- a/templates/lift/index.html.twig
+++ b/templates/lift/index.html.twig
@@ -64,6 +64,17 @@
 {% block javascripts %}
     {{ parent() }}
 
+    <script>
+        window.REP_LOG_APP_PROPS = {
+            itemOptions: [
+                {id: 'cat', text: 'Cat'},
+                {id: 'fat_cat', text: 'Big Fat Cat'},
+                {id: 'laptop', text: 'My Laptop'},
+                {id: 'coffee_cup', text: 'Coffee Cup'},
+                {id: 'invalid_item', text: 'Dark Matter'}
+            ]
+        }
+    </script>
     <script src="{{ asset('build/rep_log.js') }}"></script>
     <script src="{{ asset('build/rep_log_react.js') }}"></script>
 {% endblock %}
