diff --git a/src/Controller/LiftController.php b/src/Controller/LiftController.php
index d39bbd4..b49e991 100644
--- a/src/Controller/LiftController.php
+++ b/src/Controller/LiftController.php
@@ -20,6 +20,8 @@ class LiftController extends BaseController
     {
         $this->denyAccessUnlessGranted('IS_AUTHENTICATED_REMEMBERED');
 
+        dump(RepLog::getThingsYouCanLiftChoices());die;
+
         return $this->render('lift/index.html.twig', array(
             'leaderboard' => $this->getLeaders($replogRepo, $userRepo),
         ));
