diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 9baee151..e219dc34 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -8,7 +8,7 @@ export default class RepLogApp extends Component {
     constructor(props) {
         super(props);
 
-        this.state = {
+        this.state = Object.assign({
             highlightedRowId: null,
             repLogs: [],
             numberOfHearts: 1,
@@ -16,14 +16,8 @@ export default class RepLogApp extends Component {
             isSavingNewRepLog: false,
             successMessage: '',
             newRepLogValidationErrorMessage: '',
-            itemOptions: [
-                {id: 'cat', text: 'Cat'},
-                {id: 'fat_cat', text: 'Big Fat Cat'},
-                {id: 'laptop', text: 'My Laptop'},
-                {id: 'coffee_cup', text: 'Coffee Cup'},
-                {id: 'invalid_item', text: 'Dark Matter'}
-            ]
-        };
+            itemOptions: []
+        }, this.props.initialState);
 
         this.handleRowClick = this.handleRowClick.bind(this);
         this.handleAddRepLog = this.handleAddRepLog.bind(this);
@@ -141,5 +135,6 @@ export default class RepLogApp extends Component {
 }
 
 RepLogApp.propTypes = {
-    withHeart: PropTypes.bool
+    withHeart: PropTypes.bool,
+    initialState: PropTypes.object
 }
\ No newline at end of file
diff --git a/assets/js/rep_log_react.js b/assets/js/rep_log_react.js
index 7fc40512..0b183c80 100644
--- a/assets/js/rep_log_react.js
+++ b/assets/js/rep_log_react.js
@@ -5,6 +5,9 @@ import RepLogApp from './RepLog/RepLogApp';
 const shouldShowHeart = true;
 
 ReactDom.render(
-    <RepLogApp withHeart={shouldShowHeart} />,
+    <RepLogApp
+        withHeart={shouldShowHeart}
+        initialState={window.REP_LOG_APP_INITIAL_STATE}
+    />,
     document.getElementById('lift-stuff-app')
 );
diff --git a/templates/lift/index.html.twig b/templates/lift/index.html.twig
index 71218baa..f9664527 100644
--- a/templates/lift/index.html.twig
+++ b/templates/lift/index.html.twig
@@ -64,6 +64,17 @@
 {% block javascripts %}
     {{ parent() }}
 
+    <script>
+        window.REP_LOG_APP_INITIAL_STATE = {
+            itemOptions: [
+                {id: 'cat', text: 'Cat'},
+                {id: 'fat_cat', text: 'Big Fat Cat'},
+                {id: 'laptop', text: 'My Laptop'},
+                {id: 'coffee_cup', text: 'Coffee Cup'},
+                {id: 'invalid_item', text: 'Dark Matter'}
+            ]
+        }
+    </script>
     <script src="{{ asset('build/rep_log.js') }}"></script>
     <script src="{{ asset('build/rep_log_react.js') }}"></script>
 {% endblock %}
