diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index b7d6451f..82e40640 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -15,7 +15,14 @@ export default class RepLogApp extends Component {
             isLoaded: false,
             isSavingNewRepLog: false,
             successMessage: '',
-            newRepLogValidationErrorMessage: ''
+            newRepLogValidationErrorMessage: '',
+            itemOptions: [
+                {id: 'cat', text: 'Cat'},
+                {id: 'fat_cat', text: 'Big Fat Cat'},
+                {id: 'laptop', text: 'My Laptop'},
+                {id: 'coffee_cup', text: 'Coffee Cup'},
+                {id: 'invalid_item', text: 'Dark Matter'}
+            ]
         };
         this.clearSuccessMessageTimeout = 0;
 
diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index b2e0748e..fd7cd271 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -12,14 +12,6 @@ export default class RepLogCreator extends Component {
         this.quantityInput = React.createRef();
         this.itemSelect = React.createRef();
 
-        this.itemOptions = [
-            { id: 'cat', text: 'Cat' },
-            { id: 'fat_cat', text: 'Big Fat Cat' },
-            { id: 'laptop', text: 'My Laptop' },
-            { id: 'coffee_cup', text: 'Coffee Cup' },
-            { id: 'invalid_item', text: 'Dark Matter' }
-        ];
-
         this.handleFormSubmit = this.handleFormSubmit.bind(this);
     }
 
@@ -53,7 +45,7 @@ export default class RepLogCreator extends Component {
 
     render() {
         const { quantityInputError } = this.state;
-        const { validationErrorMessage } = this.props;
+        const { validationErrorMessage, itemOptions } = this.props;
 
         return (
             <form onSubmit={this.handleFormSubmit}>
@@ -74,7 +66,7 @@ export default class RepLogCreator extends Component {
                         <option value="">What did you
                             lift?
                         </option>
-                        {this.itemOptions.map(option => {
+                        {itemOptions.map(option => {
                             return <option value={option.id} key={option.id}>{option.text}</option>
                         })}
                     </select>
@@ -104,4 +96,5 @@ export default class RepLogCreator extends Component {
 RepLogCreator.propTypes = {
     onAddRepLog: PropTypes.func.isRequired,
     validationErrorMessage: PropTypes.string.isRequired,
+    itemOptions: PropTypes.array.isRequired
 };
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 5f5ee84d..14172fb9 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -28,7 +28,8 @@ export default function RepLogs(props) {
         isLoaded,
         isSavingNewRepLog,
         successMessage,
-        newRepLogValidationErrorMessage
+        newRepLogValidationErrorMessage,
+        itemOptions
     } = props;
 
     let heart = '';
@@ -89,6 +90,7 @@ export default function RepLogs(props) {
                     <RepLogCreator
                         onAddRepLog={onAddRepLog}
                         validationErrorMessage={newRepLogValidationErrorMessage}
+                        itemOptions={itemOptions}
                     />
                 </div>
             </div>
@@ -109,4 +111,5 @@ RepLogs.propTypes = {
     isSavingNewRepLog: PropTypes.bool.isRequired,
     successMessage: PropTypes.string.isRequired,
     newRepLogValidationErrorMessage: PropTypes.string.isRequired,
+    itemOptions: PropTypes.array.isRequired
 };
