diff --git a/src/Controller/LiftController.php b/src/Controller/LiftController.php
index d0898c1c4..5c3cf7991 100644
--- a/src/Controller/LiftController.php
+++ b/src/Controller/LiftController.php
@@ -21,19 +21,20 @@ class LiftController extends BaseController
     {
         $this->denyAccessUnlessGranted('IS_AUTHENTICATED_REMEMBERED');
 
-        $repLogAppState = [
+        $repLogAppProps = [
+            'withHeart' => true,
             'itemOptions' => [],
         ];
         foreach (RepLog::getThingsYouCanLiftChoices() as $label => $id) {
-            $repLogAppState['itemOptions'][] = [
+            $repLogAppProps['itemOptions'][] = [
                 'id' => $id,
                 'text' => $translator->trans($label),
             ];
         }
-        dump($repLogAppState);die;
 
         return $this->render('lift/index.html.twig', array(
             'leaderboard' => $this->getLeaders($replogRepo, $userRepo),
+            'repLogAppProps' => $repLogAppProps,
         ));
     }
 
diff --git a/templates/lift/index.html.twig b/templates/lift/index.html.twig
index eddda19d4..87ee4ddf0 100644
--- a/templates/lift/index.html.twig
+++ b/templates/lift/index.html.twig
@@ -65,16 +65,7 @@
     {{ parent() }}
 
     <script>
-        window.REP_LOG_APP_PROPS = {
-            itemOptions: [
-                {id: 'cat', text: 'Cat'},
-                {id: 'fat_cat', text: 'Big Fat Cat'},
-                {id: 'laptop', text: 'My Laptop'},
-                {id: 'coffee_cup', text: 'Coffee Cup'},
-                {id: 'invalid_item', text: 'Dark Matter'}
-            ],
-            withHeart: true
-        }
+        window.REP_LOG_APP_PROPS = {{ repLogAppProps|json_encode|raw }};
     </script>
     <script src="{{ asset('build/rep_log.js') }}"></script>
     <script src="{{ asset('build/rep_log_react.js') }}"></script>
