diff --git a/assets/js/rep_log_react.js b/assets/js/rep_log_react.js
index ebe9152b..98b61b59 100644
--- a/assets/js/rep_log_react.js
+++ b/assets/js/rep_log_react.js
@@ -5,9 +5,6 @@ import RepLogApp from './RepLog/RepLogApp';
 const shouldShowHeart = true;
 
 render(
-    <div>
-        <RepLogApp withHeart={shouldShowHeart} />
-        <RepLogApp withHeart={false} />
-    </div>,
+    <RepLogApp withHeart={shouldShowHeart} />,
     document.getElementById('lift-stuff-app')
 );
