diff --git a/src/Acme/LiftStuffBundle/Controller/LiftController.php b/src/Acme/LiftStuffBundle/Controller/LiftController.php
index 1d1bd9f..750e1d7 100644
--- a/src/Acme/LiftStuffBundle/Controller/LiftController.php
+++ b/src/Acme/LiftStuffBundle/Controller/LiftController.php
@@ -66,6 +66,24 @@ class LiftController extends Controller
     }
 
     /**
+     * @Route(
+     *      "/leaderboard.{_format}",
+     *      name="leaderboard",
+     *      requirements= {"_format": "json"},
+     *      options={"expose"=true}
+     * )
+     */
+    public function leaderboardAction()
+    {
+        $leaders = $this->getLeaders();
+
+        $view = View::create($leaders);
+        $handler = $this->get('fos_rest.view_handler');
+
+        return $handler->handle($view);
+    }
+
+    /**
      * @return \Doctrine\Common\Persistence\ObjectRepository
      */
     private function getUserRepository()
diff --git a/src/Acme/LiftStuffBundle/Resources/views/Lift/leaderboard.dust.twig b/src/Acme/LiftStuffBundle/Resources/views/Lift/leaderboard.dust.twig
index 9af9407..6025732 100644
--- a/src/Acme/LiftStuffBundle/Resources/views/Lift/leaderboard.dust.twig
+++ b/src/Acme/LiftStuffBundle/Resources/views/Lift/leaderboard.dust.twig
@@ -1,3 +1,4 @@
+<button class="js-leaderboard-update btn btn-primary pull-right">Update Leaderboard</button>
 
 <table class="table table-striped">
     <thead>
diff --git a/web/assets/js/Leaderboard.js b/web/assets/js/Leaderboard.js
index b311613..a57ef0a 100644
--- a/web/assets/js/Leaderboard.js
+++ b/web/assets/js/Leaderboard.js
@@ -1,7 +1,7 @@
 /**
  * Utility to for handling generic functionality related to a course
  */
-window.Leaderboard = (function($, dust, undefined){
+window.Leaderboard = (function($, dust, Routing, undefined){
 
     var Leaderboard = function($wrapper, template, leaders) {
         this.$wrapper = $wrapper;
@@ -14,6 +14,8 @@ window.Leaderboard = (function($, dust, undefined){
     $.extend(Leaderboard.prototype, {
         initialize: function() {
             this.render();
+
+            this.$wrapper.on('click', '.js-leaderboard-update', $.proxy(this._handleUpdateLeaderboard, this));
         },
 
         render: function() {
@@ -29,8 +31,34 @@ window.Leaderboard = (function($, dust, undefined){
                     self.$wrapper.html(out);
                 }
             );
+        },
+
+        /**
+         * Loads fresh leaders data from the server and re-draws the leaderboard
+         */
+        refreshFromServer: function() {
+            var self = this;
+
+            var url = Routing.generate('leaderboard', {
+                '_format': 'json'
+            });
+
+            $.ajax({
+                type: 'GET',
+                url: url,
+                success: function(data) {
+                    self.leaders = data;
+                    self.render();
+                }
+            })
+        },
+
+        _handleUpdateLeaderboard: function(e) {
+            e.preventDefault();
+
+            this.refreshFromServer();
         }
     });
 
     return Leaderboard;
-})(jQuery, dust);
+})(jQuery, dust, Routing);
