diff --git a/app/Resources/views/base.html.twig b/app/Resources/views/base.html.twig
index bea5bce..7f2e7b8 100644
--- a/app/Resources/views/base.html.twig
+++ b/app/Resources/views/base.html.twig
@@ -103,9 +103,15 @@
 {% block javascripts %}
     <script src="{{ asset('assets/vendor/jquery/jquery.js') }}"></script>
     <script src="{{ asset('assets/vendor/bootstrap/dist/js/bootstrap.js') }}"></script>
+    <script src="{{ asset('assets/vendor/eventEmitter/EventEmitter.js') }}"></script>
 
     <script src="{{ asset('bundles/fosjsrouting/js/router.js') }}"></script>
     <script src="{{ path('fos_js_routing_js', {"callback": "fos.Router.setData"}) }}"></script>
+
+    <script>
+        // create a global eventEmitter
+        window.eventEmitter = new EventEmitter();
+    </script>
 {% endblock %}
 </body>
 </html>
diff --git a/bower.json b/bower.json
index d7ce684..8738f4d 100644
--- a/bower.json
+++ b/bower.json
@@ -17,6 +17,7 @@
   ],
   "dependencies": {
     "bootstrap": "~3.0.3",
-    "dustjs-linkedin": "~2.3.1"
+    "dustjs-linkedin": "~2.3.1",
+    "eventEmitter": "~4.2.7"
   }
 }
diff --git a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
index 4d5a39c..c3fbde3 100644
--- a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
+++ b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
@@ -52,11 +52,12 @@
 
     <script>
         jQuery(document).ready(function() {
-            var repForm = new RepForm($('form.rep-log-form'));
+            var repForm = new RepForm($('form.rep-log-form'), eventEmitter);
             var leaderboard = new Leaderboard(
                 $('#leaderboard'),
                 $('#js-template-leaderboard').html(),
-                {{ leaders|json_encode|raw }}
+                {{ leaders|json_encode|raw }},
+                eventEmitter
             );
         });
     </script>
diff --git a/web/assets/js/Leaderboard.js b/web/assets/js/Leaderboard.js
index 58fe1a7..2278f86 100644
--- a/web/assets/js/Leaderboard.js
+++ b/web/assets/js/Leaderboard.js
@@ -3,10 +3,11 @@
  */
 window.Leaderboard = (function($, dust, Routing, undefined){
 
-    var Leaderboard = function($wrapper, template, leaders) {
+    var Leaderboard = function($wrapper, template, leaders, eventEmitter) {
         this.$wrapper = $wrapper;
         this.template = template;
         this.leaders = leaders;
+        this.eventEmitter = eventEmitter;
 
         this.initialize();
     };
@@ -16,6 +17,8 @@ window.Leaderboard = (function($, dust, Routing, undefined){
             this.render();
 
             this.$wrapper.on('click', '.js-leaderboard-update', $.proxy(this._handleUpdateLeaderboard, this));
+
+            this.eventEmitter.addListener('reps.update', $.proxy(this._handleRepsUpdate, this));
         },
 
         /**
@@ -83,6 +86,10 @@ window.Leaderboard = (function($, dust, Routing, undefined){
             e.preventDefault();
 
             this.refreshFromServer();
+        },
+
+        _handleRepsUpdate: function() {
+            this.refreshFromServer();
         }
     });
 
diff --git a/web/assets/js/RepForm.js b/web/assets/js/RepForm.js
index 4440981..0948554 100644
--- a/web/assets/js/RepForm.js
+++ b/web/assets/js/RepForm.js
@@ -3,8 +3,9 @@
  */
 window.RepForm = (function($, SymfonyErrorUtil, ErrorList, undefined){
 
-    var RepForm = function ($form) {
+    var RepForm = function ($form, eventEmitter) {
         this.$form = $form;
+        this.eventEmitter = eventEmitter;
 
         this.initialize();
     };
@@ -40,6 +41,9 @@ window.RepForm = (function($, SymfonyErrorUtil, ErrorList, undefined){
                     $notifications.append('<div class="alert alert-success">Reps saved! You\'re killing it!</div>');
 
                     self.errorList.clear();
+
+                    // trigger that the reps updated
+                    self.eventEmitter.emitEvent('reps.update');
                 }
             });
         },
