diff --git a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
index a0f69c8..455ade3 100644
--- a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
+++ b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
@@ -31,7 +31,7 @@
         <div class="col-md-6">
             <h2>Leaderboard</h2>
 
-            {{ render(controller('AcmeLiftStuffBundle:Lift:leaderboard')) }}
+            <div id="leaderboard"></div>
         </div>
     </div>
 {% endblock %}
@@ -39,13 +39,22 @@
 {% block javascripts %}
     {{ parent() }}
 
+    <script type="text/template" id="js-template-leaderboard">
+        {{ include('AcmeLiftStuffBundle:Lift:leaderboard.dust.twig') }}
+    </script>
+
     <script src="{{ asset('assets/js/SymfonyErrorUtil.js') }}"></script>
     <script src="{{ asset('assets/js/ErrorList.js') }}"></script>
     <script src="{{ asset('assets/js/RepForm.js') }}"></script>
+    <script src="{{ asset('assets/js/Leaderboard.js') }}"></script>
 
     <script>
         jQuery(document).ready(function() {
             var repForm = new RepForm($('form.rep-log-form'));
+            var leaderboard = new Leaderboard(
+                $('#leaderboard'),
+                $('#js-template-leaderboard').html()
+            );
         });
     </script>
 {% endblock %}
diff --git a/src/Acme/LiftStuffBundle/Resources/views/Lift/leaderboard.dust.twig b/src/Acme/LiftStuffBundle/Resources/views/Lift/leaderboard.dust.twig
new file mode 100644
index 0000000..9af9407
--- /dev/null
+++ b/src/Acme/LiftStuffBundle/Resources/views/Lift/leaderboard.dust.twig
@@ -0,0 +1,23 @@
+
+<table class="table table-striped">
+    <thead>
+    <tr>
+        <th>Username</th>
+        <th>Weight lifted</th>
+    </tr>
+    </thead>
+    <tbody>
+    {{ '{#leaders}' }}
+        <tr>
+            <td>{username}</td>
+            <td style="width: 175px;">
+                {weight}lbs
+
+                <span class="label label-primary pull-right">
+                    (~{in_cats} fat cats)
+                </span>
+            </td>
+        </tr>
+    {/leaders}
+    </tbody>
+</table>
diff --git a/src/Acme/LiftStuffBundle/Resources/views/Lift/leaderboard.html.twig b/src/Acme/LiftStuffBundle/Resources/views/Lift/leaderboard.html.twig
deleted file mode 100644
index b08f335..0000000
--- a/src/Acme/LiftStuffBundle/Resources/views/Lift/leaderboard.html.twig
+++ /dev/null
@@ -1,22 +0,0 @@
-<table class="table table-striped">
-    <thead>
-    <tr>
-        <th>Username</th>
-        <th>Weight lifted</th>
-    </tr>
-    </thead>
-    <tbody>
-    {% for leader in leaderboard %}
-        <tr>
-            <td>{{ leader.username }}</td>
-            <td style="width: 175px;">
-                {{ leader.weight }}lbs
-
-                <span class="label label-primary pull-right">
-                    (~{{ leader.in_cats|number_format(0) }} fat cats)
-                </span>
-            </td>
-        </tr>
-    {% endfor %}
-    </tbody>
-</table>
diff --git a/web/assets/js/Leaderboard.js b/web/assets/js/Leaderboard.js
new file mode 100644
index 0000000..906ac88
--- /dev/null
+++ b/web/assets/js/Leaderboard.js
@@ -0,0 +1,21 @@
+window.Leaderboard = (function($, undefined){
+
+    var Leaderboard = function($wrapper, template) {
+        this.$wrapper = $wrapper;
+        this.template = template;
+
+        this.initialize();
+    };
+
+    $.extend(Leaderboard.prototype, {
+        initialize: function() {
+            this.render();
+        },
+
+        render: function() {
+            this.$wrapper.html(this.template);
+        }
+    });
+
+    return Leaderboard;
+})(jQuery);
