diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 8ead14a..0ff7bd5 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -34,7 +34,7 @@
         </div>
         <div class="col-md-5">
             <div class="leaderboard">
-                <h2 class="text-center">
+                <h2 class="text-center js-custom-popover" data-toggle="popover" title="About the Leaderboard" data-content="Want to be the leader? Lift stuff!">
                     <img class="dumbbell" src="{{ asset('build/static/dumbbell.png') }}" />
                     Leaderboard
                 </h2>
@@ -49,4 +49,13 @@
     {{ parent() }}
 
     <script src="{{ asset('build/rep_log.js') }}"></script>
+
+    <script>
+        $(document).ready(function() {
+            $('.js-custom-popover').popover({
+                trigger: 'hover',
+                placement: 'left'
+            });
+        });
+    </script>
 {% endblock %}
