diff --git a/webpack.config.js b/webpack.config.js
index f539b4e..d924e7d 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -94,7 +94,9 @@ module.exports = {
     plugins: [
         new webpack.ProvidePlugin({
             jQuery: 'jquery',
-            $: 'jquery'
+            $: 'jquery',
+            'window.jQuery': 'jquery',
+            'window.$': 'jquery',
         }),
 
         new CopyWebpackPlugin([
