diff --git a/assets/js/rep_log.js b/assets/js/rep_log.js
index bbaaa09..9bdfc1d 100644
--- a/assets/js/rep_log.js
+++ b/assets/js/rep_log.js
@@ -4,7 +4,7 @@ const RepLogApp = require('./Components/RepLogApp');
 
 // expose $ globally so I can use it in the template
 // ... even though I should put all my code here!
-window.$ = $;
+global.$ = $;
 
 $(document).ready(function() {
     var $wrapper = $('.js-rep-log-table');
diff --git a/webpack.config.js b/webpack.config.js
index d924e7d..ef77a49 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -96,7 +96,6 @@ module.exports = {
             jQuery: 'jquery',
             $: 'jquery',
             'window.jQuery': 'jquery',
-            'window.$': 'jquery',
         }),
 
         new CopyWebpackPlugin([
