diff --git a/public/assets/js/layout.js b/public/assets/js/layout.js
index 3628a4e..e9d87b7 100644
--- a/public/assets/js/layout.js
+++ b/public/assets/js/layout.js
@@ -1,7 +1,6 @@
 'use strict';
 
 const $ = require('jquery');
-window.jQuery = $;
 require('bootstrap');
 
 $(document).ready(function() {
diff --git a/webpack.config.js b/webpack.config.js
index ac02bd9..bc6cc09 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -12,6 +12,8 @@ Encore
     .addEntry('layout', './public/assets/js/layout.js')
 
     .enableBuildNotifications()
+    // fixes modules that expect jQuery to be global
+    .autoProvidejQuery()
 ;
 
 // export the final configuration
