diff --git a/assets/js/rep_log.js b/assets/js/rep_log.js
index ca79a28..bbaaa09 100644
--- a/assets/js/rep_log.js
+++ b/assets/js/rep_log.js
@@ -1,6 +1,11 @@
 const $ = require('jquery');
+require('bootstrap-sass');
 const RepLogApp = require('./Components/RepLogApp');
 
+// expose $ globally so I can use it in the template
+// ... even though I should put all my code here!
+window.$ = $;
+
 $(document).ready(function() {
     var $wrapper = $('.js-rep-log-table');
     var repLogApp = new RepLogApp($wrapper);
