diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index d2260de..9e53651 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -75,7 +75,10 @@ export default class RepLogApp extends Component {
     }
 
     handleDeleteRepLog(id) {
-        deleteRepLog(id);
+        deleteRepLog(id)
+            .then(() => {
+                this.setSuccessMessage('Item was Un-lifted!');
+            });
 
         // remove the rep log without mutating state
         // filter returns a new array
