diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index f8d1420fb..db510752e 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -75,6 +75,21 @@ export default class RepLogApp extends Component {
     }
 
     handleDeleteRepLog(id) {
+        this.setState((prevState) => {
+            return {
+                repLogs: prevState.repLogs.map(repLog => {
+                    if (repLog.id !== id) {
+                        return repLog;
+                    }
+
+                    return Object.assign({}, repLog, {isDeleting: true});
+                })
+            };
+        });
+
+        // temporary return
+        return;
+
         deleteRepLog(id)
             .then(() => {
                 // remove the rep log without mutating state
diff --git a/assets/js/RepLog/RepLogList.js b/assets/js/RepLog/RepLogList.js
index 995535c3b..e6b64da5c 100644
--- a/assets/js/RepLog/RepLogList.js
+++ b/assets/js/RepLog/RepLogList.js
@@ -27,6 +27,9 @@ export default function RepLogList(props) {
                 key={repLog.id}
                 className={highlightedRowId === repLog.id ? 'info' : ''}
                 onClick={() =>  onRowClick(repLog.id)}
+                style={{
+                    opacity: repLog.isDeleting ? .3 : 1
+                }}
             >
                 <td>{repLog.itemLabel}</td>
                 <td>{repLog.reps}</td>
