diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 8d19aa6a..2273b3e0 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -33,6 +33,12 @@ export default class RepLogApp extends Component {
             });
     }
 
+    componentWillUnmount() {
+        if (this.clearSuccessMessageTimeout) {
+            clearTimeout(this.clearSuccessMessageTimeout);
+        }
+    }
+
     handleRowClick(repLogId) {
         this.setState({highlightedRowId: repLogId});
     }
@@ -55,9 +61,10 @@ export default class RepLogApp extends Component {
                     return {
                         repLogs: newRepLogs,
                         isSavingNewRepLog: false,
-                        successMessage: 'Rep Log Saved!'
                     };
-                })
+                });
+
+                this.setSuccessMessage('Rep Log Saved!');
             })
         ;
     }
@@ -80,6 +87,19 @@ export default class RepLogApp extends Component {
         });
     }
 
+    setSuccessMessage(message) {
+        this.setState({
+            successMessage: message
+        });
+
+        this.clearSuccessMessageTimeout = setTimeout(() => {
+            this.setState({
+                successMessage: ''
+            });
+            this.clearSuccessMessageTimeout = null;
+        }, 3000)
+    }
+
     render() {
         return (
             <RepLogs
