diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 50ca7e437..4c6029adb 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -13,7 +13,8 @@ export default class RepLogApp extends Component {
             repLogs: [],
             numberOfHearts: 1,
             isLoaded: false,
-            isSavingNewRepLog: false
+            isSavingNewRepLog: false,
+            successMessage: ''
         };
 
         this.handleRowClick = this.handleRowClick.bind(this);
@@ -53,7 +54,8 @@ export default class RepLogApp extends Component {
 
                     return {
                         repLogs: newRepLogs,
-                        isSavingNewRepLog: false
+                        isSavingNewRepLog: false,
+                        successMessage: 'Rep Log Saved!'
                     };
                 })
             })
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 66d55d1c4..b7f04966e 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -26,7 +26,8 @@ export default function RepLogs(props) {
         onHeartChange,
         onDeleteRepLog,
         isLoaded,
-        isSavingNewRepLog
+        isSavingNewRepLog,
+        successMessage
     } = props;
 
     let heart = '';
@@ -34,6 +35,11 @@ export default function RepLogs(props) {
         heart = <span>{'❤️'.repeat(numberOfHearts)}</span>;
     }
 
+    let successMessageElement = '';
+    if (successMessage) {
+        successMessageElement = <div className="alert alert-success text-center">{successMessage}</div>
+    }
+
     return (
         <div className="col-md-7">
             <h2>Lift Stuff! {heart}</h2>
@@ -46,6 +52,10 @@ export default function RepLogs(props) {
                 }}
             />
 
+            <hr/>
+
+            {successMessageElement}
+
             <table className="table table-striped">
                 <thead>
                 <tr>
@@ -95,4 +105,5 @@ RepLogs.propTypes = {
     onDeleteRepLog: PropTypes.func.isRequired,
     isLoaded: PropTypes.bool.isRequired,
     isSavingNewRepLog: PropTypes.bool.isRequired,
+    successMessage: PropTypes.string.isRequired
 };
