diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 1799679..d2260de 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -16,6 +16,7 @@ export default class RepLogApp extends Component {
             isSavingNewRepLog: false,
             successMessage: ''
         };
+        this.successMessageTimeoutHandle = 0;
 
         this.handleRowClick = this.handleRowClick.bind(this);
         this.handleAddRepLog = this.handleAddRepLog.bind(this);
@@ -33,6 +34,10 @@ export default class RepLogApp extends Component {
             });
     }
 
+    componentWillUnmount() {
+        clearTimeout(this.successMessageTimeoutHandle);
+    }
+
     handleRowClick(repLogId) {
         this.setState({highlightedRowId: repLogId});
     }
@@ -86,10 +91,12 @@ export default class RepLogApp extends Component {
             successMessage: message
         });
 
-        setTimeout(() => {
+        clearTimeout(this.successMessageTimeoutHandle);
+        this.successMessageTimeoutHandle = setTimeout(() => {
             this.setState({
                 successMessage: ''
             });
+            this.successMessageTimeoutHandle = 0;
         }, 3000)
     }
 
