diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 6fb8b587..f5f7dd03 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -13,7 +13,8 @@ export default class RepLogApp extends Component {
             repLogs: [],
             numberOfHearts: 1,
             isLoaded: false,
-            isSavingNewRepLog: false
+            isSavingNewRepLog: false,
+            successMessage: ''
         };
 
         this.handleRowClick = this.handleRowClick.bind(this);
@@ -53,7 +54,8 @@ export default class RepLogApp extends Component {
 
                     return {
                         repLogs: newRepLogs,
-                        isSavingNewRepLog: false
+                        isSavingNewRepLog: false,
+                        successMessage: 'Rep Log Saved!'
                     };
                 })
             })
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 1b1b9473..6a899796 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -20,13 +20,18 @@ export default class RepLogs extends Component {
     }
 
     render() {
-        const { withHeart, highlightedRowId, onRowClick, repLogs, onAddRepLog, numberOfHearts, onHeartChange, onDeleteRepLog, isLoaded, isSavingNewRepLog } = this.props;
+        const { withHeart, highlightedRowId, onRowClick, repLogs, onAddRepLog, numberOfHearts, onHeartChange, onDeleteRepLog, isLoaded, isSavingNewRepLog, successMessage } = this.props;
 
         let heart = '';
         if (withHeart) {
             heart = <span>{'❤️'.repeat(numberOfHearts)}</span>;
         }
 
+        let successMessageElement = '';
+        if (successMessage) {
+            successMessageElement = <div className="alert alert-success text-center">{successMessage}</div>
+        }
+
         return (
             <div className="col-md-7">
                 <h2>Lift Stuff! {heart}</h2>
@@ -39,6 +44,10 @@ export default class RepLogs extends Component {
                     }}
                 />
 
+                <hr/>
+
+                {successMessageElement}
+
                 <table className="table table-striped">
                     <thead>
                     <tr>
@@ -87,4 +96,5 @@ RepLogs.propTypes = {
     onDeleteRepLog: PropTypes.func.isRequired,
     isLoaded: PropTypes.bool.isRequired,
     isSavingNewRepLog: PropTypes.bool.isRequired,
+    successMessage: PropTypes.string.isRequired
 };
