diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 4c6029adb..a4b696807 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -16,6 +16,7 @@ export default class RepLogApp extends Component {
             isSavingNewRepLog: false,
             successMessage: ''
         };
+        this.clearSuccessMessageTimeout = 0;
 
         this.handleRowClick = this.handleRowClick.bind(this);
         this.handleAddRepLog = this.handleAddRepLog.bind(this);
@@ -33,6 +34,10 @@ export default class RepLogApp extends Component {
             });
     }
 
+    componentWillUnmount() {
+        clearTimeout(this.clearSuccessMessageTimeout);
+    }
+
     handleRowClick(repLogId) {
         this.setState({highlightedRowId: repLogId});
     }
@@ -55,9 +60,10 @@ export default class RepLogApp extends Component {
                     return {
                         repLogs: newRepLogs,
                         isSavingNewRepLog: false,
-                        successMessage: 'Rep Log Saved!'
                     };
-                })
+                });
+
+                this.setSuccessMessage('Rep Log Saved!');
             })
         ;
     }
@@ -80,6 +86,20 @@ export default class RepLogApp extends Component {
         });
     }
 
+    setSuccessMessage(message) {
+        this.setState({
+            successMessage: message
+        });
+
+        clearTimeout(this.clearSuccessMessageTimeout);
+        this.clearSuccessMessageTimeout = setTimeout(() => {
+            this.setState({
+                successMessage: ''
+            });
+            this.clearSuccessMessageTimeout = 0;
+        }, 3000)
+    }
+
     render() {
         return (
             <RepLogs
