diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index f88b53f0..15708a5d 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -80,7 +80,8 @@
                 );
 
                 var newThis = {cat: 'meow', dog: 'woof'};
-                this.whatIsThis.call(newThis, 'hello');
+                var boundWhatIsThis = this.whatIsThis.bind(this);
+                boundWhatIsThis.call(newThis, 'hello');
             },
 
             whatIsThis: function(greeting) {
