diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 27af7f4..45d896d 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -105,7 +105,7 @@
                     method: 'DELETE',
                     success: function() {
                         $row.fadeOut('normal', function() {
-                            $row.remove();
+                            $(this).remove();
                             RepLogApp.updateTotalWeightLifted();
                         });
                     }
