diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 43e4b39..38939f3 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -82,7 +82,8 @@
                     this.handleRowClick
                 );
 
-                this.whatIsThis('hello');
+                var newThis = {cat: 'meow', dog: 'woof'};
+                this.whatIsThis.call(newThis, 'hello');
             },
 
             whatIsThis: function(greeting) {
