diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 01f7294..ca35dc3 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -98,7 +98,8 @@
                 );
 
                 var newThis = {cat: 'meow', dog: 'woof'};
-                this.whatIsThis.call(newThis, 'hello');
+                var boundWhatIsThis = this.whatIsThis.bind(this);
+                boundWhatIsThis.call(newThis, 'hello');
             },
 
             whatIsThis: function(greeting) {
