diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 1b8fa88..eddddb5 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -123,7 +123,7 @@
                     method: 'DELETE',
                     success: function() {
                         $row.fadeOut('normal', function() {
-                            $row.remove();
+                            $(this).remove();
                             RepLogTable.updateTotalWeightLifted();
                         });
                     }
