diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 45d896d3..3694329a 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -78,6 +78,12 @@
                     'click',
                     this.handleRowClick
                 );
+
+                this.whatIsThis('hello');
+            },
+
+            whatIsThis: function(greeting) {
+                console.log(this, greeting);
             },
 
             updateTotalWeightLifted: function() {
