diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 00dd60a..65cfffe 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -77,7 +77,8 @@
                     this.handleRowClick
                 );
 
-                this.whatIsThis('hello');
+                var newThis = {cat: 'meow', dog: 'woof'};
+                this.whatIsThis.call(newThis, 'hello');
             },
 
             whatIsThis: function(greeting) {
