diff --git a/play.js b/play.js
index 1bff962..8d69190 100644
--- a/play.js
+++ b/play.js
@@ -3,7 +3,10 @@ foods.set('italian', 'gelato');
 foods.set('mexican', 'tortas');
 foods.set('canadian', 'poutine');
 
+let southernUsStates = ['Tennessee', 'Kentucky', 'Texas'];
+foods.set(southernUsStates, 'hot chicken');
+
 console.log(
     foods.get('italian'),
-    foods.has('french')
+    foods.get(southernUsStates)
 );
