diff --git a/web/assets/js/Components/RepLogApp.js b/web/assets/js/Components/RepLogApp.js
new file mode 100644
index 0000000..51ba1d2
--- /dev/null
+++ b/web/assets/js/Components/RepLogApp.js
@@ -0,0 +1,220 @@
+'use strict';
+
+const Helper = require('./RepLogAppHelper');
+const $ = require('jquery');
+const swal = require('sweetalert2');
+
+let HelperInstances = new WeakMap();
+
+class RepLogApp {
+    constructor($wrapper) {
+        this.$wrapper = $wrapper;
+        this.repLogs = [];
+
+        HelperInstances.set(this, new Helper(this.repLogs));
+
+        this.loadRepLogs();
+
+        this.$wrapper.on(
+            'click',
+            '.js-delete-rep-log',
+            this.handleRepLogDelete.bind(this)
+        );
+        this.$wrapper.on(
+            'click',
+            'tbody tr',
+            this.handleRowClick.bind(this)
+        );
+        this.$wrapper.on(
+            'submit',
+            RepLogApp._selectors.newRepForm,
+            this.handleNewFormSubmit.bind(this)
+        );
+    }
+
+    /**
+     * Call like this.selectors
+     */
+    static get _selectors() {
+        return {
+            newRepForm: '.js-new-rep-log-form'
+        }
+    }
+
+    loadRepLogs() {
+        $.ajax({
+            url: Routing.generate('rep_log_list'),
+        }).then(data => {
+            for (let repLog of data.items) {
+                this._addRow(repLog);
+            }
+        })
+    }
+
+    updateTotalWeightLifted() {
+        this.$wrapper.find('.js-total-weight').html(
+            HelperInstances.get(this).getTotalWeightString()
+        );
+    }
+
+    handleRepLogDelete(e) {
+        e.preventDefault();
+
+        const $link = $(e.currentTarget);
+
+        swal({
+            title: 'Delete this log???',
+            text: 'What? Did you not actually lift this?',
+            showCancelButton: true,
+            showLoaderOnConfirm: true,
+            preConfirm: () => this._deleteRepLog($link)
+        }).catch((arg) => {
+            // canceling is cool!
+        });
+    }
+
+    _deleteRepLog($link) {
+        $link.addClass('text-danger');
+        $link.find('.fa')
+            .removeClass('fa-trash')
+            .addClass('fa-spinner')
+            .addClass('fa-spin');
+
+        const deleteUrl = $link.data('url');
+        const $row = $link.closest('tr');
+
+        return $.ajax({
+            url: deleteUrl,
+            method: 'DELETE'
+        }).then(() => {
+            $row.fadeOut('normal', () => {
+                // we need to remove the repLog from this.repLogs
+                // the "key" is the index to this repLog on this.repLogs
+                this.repLogs.splice(
+                    $row.data('key'),
+                    1
+                );
+
+                $row.remove();
+
+                this.updateTotalWeightLifted();
+            });
+        })
+    }
+
+    handleRowClick() {
+        console.log('row clicked!');
+    }
+
+    handleNewFormSubmit(e) {
+        e.preventDefault();
+
+        const $form = $(e.currentTarget);
+        const formData = {};
+
+        for (let fieldData of $form.serializeArray()) {
+            formData[fieldData.name] = fieldData.value
+        }
+
+        this._saveRepLog(formData)
+        .then((data) => {
+            this._clearForm();
+            this._addRow(data);
+        }).catch((errorData) => {
+            this._mapErrorsToForm(errorData.errors);
+        });
+    }
+
+    _saveRepLog(data) {
+        return new Promise((resolve, reject) => {
+            const url = Routing.generate('rep_log_new');
+
+            $.ajax({
+                url,
+                method: 'POST',
+                data: JSON.stringify(data)
+            }).then((data, textStatus, jqXHR) => {
+                $.ajax({
+                    url: jqXHR.getResponseHeader('Location')
+                }).then((data) => {
+                    // we're finally done!
+                    resolve(data);
+                });
+            }).catch((jqXHR) => {
+                const errorData = JSON.parse(jqXHR.responseText);
+
+                reject(errorData);
+            });
+        });
+    }
+
+    _mapErrorsToForm(errorData) {
+        this._removeFormErrors();
+        const $form = this.$wrapper.find(RepLogApp._selectors.newRepForm);
+
+        for (let element of $form.find(':input')) {
+            const fieldName = $(element).attr('name');
+            const $wrapper = $(element).closest('.form-group');
+            if (!errorData[fieldName]) {
+                // no error!
+                continue;
+            }
+
+            const $error = $('<span class="js-field-error help-block"></span>');
+            $error.html(errorData[fieldName]);
+            $wrapper.append($error);
+            $wrapper.addClass('has-error');
+        }
+
+        const $error = $('<span class="js-field-error help-block"></span>');
+        $error.html(errorData[fieldName]);
+        $wrapper.append($error);
+        $wrapper.addClass('has-error');
+    }
+
+    _removeFormErrors() {
+        const $form = this.$wrapper.find(RepLogApp._selectors.newRepForm);
+        $form.find('.js-field-error').remove();
+        $form.find('.form-group').removeClass('has-error');
+    }
+
+    _clearForm() {
+        this._removeFormErrors();
+
+        const $form = this.$wrapper.find(RepLogApp._selectors.newRepForm);
+        $form[0].reset();
+    }
+
+    _addRow(repLog) {
+        this.repLogs.push(repLog);
+        // destructuring example
+        // let {id, itemLabel, reps, totallyMadeUpKey = 'whatever!'} = repLog;
+        // console.log(id, itemLabel, reps, totallyMadeUpKey);
+
+        const html = rowTemplate(repLog);
+        const $row = $($.parseHTML(html));
+        // store the repLogs index
+        $row.data('key', this.repLogs.length - 1);
+        this.$wrapper.find('tbody').append($row);
+
+        this.updateTotalWeightLifted();
+    }
+}
+
+const rowTemplate = (repLog) => `
+<tr data-weight="${repLog.totalWeightLifted}">
+<td>${repLog.itemLabel}</td>
+<td>${repLog.reps}</td>
+<td>${repLog.totalWeightLifted}</td>
+<td>
+    <a href="#"
+       class="js-delete-rep-log"
+       data-url="${repLog.links._self}"
+    >
+        <span class="fa fa-trash"></span>
+    </a>
+</td>
+</tr>
+`;
+
+module.exports = RepLogApp;
diff --git a/web/assets/js/Components/RepLogAppHelper.js b/web/assets/js/Components/RepLogAppHelper.js
new file mode 100644
index 0000000..bb2d5c9
--- /dev/null
+++ b/web/assets/js/Components/RepLogAppHelper.js
@@ -0,0 +1,37 @@
+'use strict';
+
+/**
+ * A "private" object
+ */
+class Helper {
+    constructor(repLogs) {
+        this.repLogs = repLogs;
+    }
+
+    calculateTotalWeight() {
+        return Helper._calculateWeights(
+            this.repLogs
+        );
+    }
+
+    getTotalWeightString(maxWeight = 500) {
+        let weight = this.calculateTotalWeight();
+
+        if (weight > maxWeight) {
+            weight = maxWeight + '+';
+        }
+
+        return weight + ' lbs';
+    }
+
+    static _calculateWeights(repLogs) {
+        let totalWeight = 0;
+        for (let repLog of repLogs) {
+            totalWeight += repLog.totalWeightLifted;
+        }
+
+        return totalWeight;
+    }
+}
+
+module.exports = Helper;
diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
deleted file mode 100644
index 51ba1d2..0000000
--- a/web/assets/js/RepLogApp.js
+++ /dev/null
@@ -1,220 +0,0 @@
-'use strict';
-
-const Helper = require('./RepLogAppHelper');
-const $ = require('jquery');
-const swal = require('sweetalert2');
-
-let HelperInstances = new WeakMap();
-
-class RepLogApp {
-    constructor($wrapper) {
-        this.$wrapper = $wrapper;
-        this.repLogs = [];
-
-        HelperInstances.set(this, new Helper(this.repLogs));
-
-        this.loadRepLogs();
-
-        this.$wrapper.on(
-            'click',
-            '.js-delete-rep-log',
-            this.handleRepLogDelete.bind(this)
-        );
-        this.$wrapper.on(
-            'click',
-            'tbody tr',
-            this.handleRowClick.bind(this)
-        );
-        this.$wrapper.on(
-            'submit',
-            RepLogApp._selectors.newRepForm,
-            this.handleNewFormSubmit.bind(this)
-        );
-    }
-
-    /**
-     * Call like this.selectors
-     */
-    static get _selectors() {
-        return {
-            newRepForm: '.js-new-rep-log-form'
-        }
-    }
-
-    loadRepLogs() {
-        $.ajax({
-            url: Routing.generate('rep_log_list'),
-        }).then(data => {
-            for (let repLog of data.items) {
-                this._addRow(repLog);
-            }
-        })
-    }
-
-    updateTotalWeightLifted() {
-        this.$wrapper.find('.js-total-weight').html(
-            HelperInstances.get(this).getTotalWeightString()
-        );
-    }
-
-    handleRepLogDelete(e) {
-        e.preventDefault();
-
-        const $link = $(e.currentTarget);
-
-        swal({
-            title: 'Delete this log???',
-            text: 'What? Did you not actually lift this?',
-            showCancelButton: true,
-            showLoaderOnConfirm: true,
-            preConfirm: () => this._deleteRepLog($link)
-        }).catch((arg) => {
-            // canceling is cool!
-        });
-    }
-
-    _deleteRepLog($link) {
-        $link.addClass('text-danger');
-        $link.find('.fa')
-            .removeClass('fa-trash')
-            .addClass('fa-spinner')
-            .addClass('fa-spin');
-
-        const deleteUrl = $link.data('url');
-        const $row = $link.closest('tr');
-
-        return $.ajax({
-            url: deleteUrl,
-            method: 'DELETE'
-        }).then(() => {
-            $row.fadeOut('normal', () => {
-                // we need to remove the repLog from this.repLogs
-                // the "key" is the index to this repLog on this.repLogs
-                this.repLogs.splice(
-                    $row.data('key'),
-                    1
-                );
-
-                $row.remove();
-
-                this.updateTotalWeightLifted();
-            });
-        })
-    }
-
-    handleRowClick() {
-        console.log('row clicked!');
-    }
-
-    handleNewFormSubmit(e) {
-        e.preventDefault();
-
-        const $form = $(e.currentTarget);
-        const formData = {};
-
-        for (let fieldData of $form.serializeArray()) {
-            formData[fieldData.name] = fieldData.value
-        }
-
-        this._saveRepLog(formData)
-        .then((data) => {
-            this._clearForm();
-            this._addRow(data);
-        }).catch((errorData) => {
-            this._mapErrorsToForm(errorData.errors);
-        });
-    }
-
-    _saveRepLog(data) {
-        return new Promise((resolve, reject) => {
-            const url = Routing.generate('rep_log_new');
-
-            $.ajax({
-                url,
-                method: 'POST',
-                data: JSON.stringify(data)
-            }).then((data, textStatus, jqXHR) => {
-                $.ajax({
-                    url: jqXHR.getResponseHeader('Location')
-                }).then((data) => {
-                    // we're finally done!
-                    resolve(data);
-                });
-            }).catch((jqXHR) => {
-                const errorData = JSON.parse(jqXHR.responseText);
-
-                reject(errorData);
-            });
-        });
-    }
-
-    _mapErrorsToForm(errorData) {
-        this._removeFormErrors();
-        const $form = this.$wrapper.find(RepLogApp._selectors.newRepForm);
-
-        for (let element of $form.find(':input')) {
-            const fieldName = $(element).attr('name');
-            const $wrapper = $(element).closest('.form-group');
-            if (!errorData[fieldName]) {
-                // no error!
-                continue;
-            }
-
-            const $error = $('<span class="js-field-error help-block"></span>');
-            $error.html(errorData[fieldName]);
-            $wrapper.append($error);
-            $wrapper.addClass('has-error');
-        }
-
-        const $error = $('<span class="js-field-error help-block"></span>');
-        $error.html(errorData[fieldName]);
-        $wrapper.append($error);
-        $wrapper.addClass('has-error');
-    }
-
-    _removeFormErrors() {
-        const $form = this.$wrapper.find(RepLogApp._selectors.newRepForm);
-        $form.find('.js-field-error').remove();
-        $form.find('.form-group').removeClass('has-error');
-    }
-
-    _clearForm() {
-        this._removeFormErrors();
-
-        const $form = this.$wrapper.find(RepLogApp._selectors.newRepForm);
-        $form[0].reset();
-    }
-
-    _addRow(repLog) {
-        this.repLogs.push(repLog);
-        // destructuring example
-        // let {id, itemLabel, reps, totallyMadeUpKey = 'whatever!'} = repLog;
-        // console.log(id, itemLabel, reps, totallyMadeUpKey);
-
-        const html = rowTemplate(repLog);
-        const $row = $($.parseHTML(html));
-        // store the repLogs index
-        $row.data('key', this.repLogs.length - 1);
-        this.$wrapper.find('tbody').append($row);
-
-        this.updateTotalWeightLifted();
-    }
-}
-
-const rowTemplate = (repLog) => `
-<tr data-weight="${repLog.totalWeightLifted}">
-<td>${repLog.itemLabel}</td>
-<td>${repLog.reps}</td>
-<td>${repLog.totalWeightLifted}</td>
-<td>
-    <a href="#"
-       class="js-delete-rep-log"
-       data-url="${repLog.links._self}"
-    >
-        <span class="fa fa-trash"></span>
-    </a>
-</td>
-</tr>
-`;
-
-module.exports = RepLogApp;
diff --git a/web/assets/js/RepLogAppHelper.js b/web/assets/js/RepLogAppHelper.js
deleted file mode 100644
index bb2d5c9..0000000
--- a/web/assets/js/RepLogAppHelper.js
+++ /dev/null
@@ -1,37 +0,0 @@
-'use strict';
-
-/**
- * A "private" object
- */
-class Helper {
-    constructor(repLogs) {
-        this.repLogs = repLogs;
-    }
-
-    calculateTotalWeight() {
-        return Helper._calculateWeights(
-            this.repLogs
-        );
-    }
-
-    getTotalWeightString(maxWeight = 500) {
-        let weight = this.calculateTotalWeight();
-
-        if (weight > maxWeight) {
-            weight = maxWeight + '+';
-        }
-
-        return weight + ' lbs';
-    }
-
-    static _calculateWeights(repLogs) {
-        let totalWeight = 0;
-        for (let repLog of repLogs) {
-            totalWeight += repLog.totalWeightLifted;
-        }
-
-        return totalWeight;
-    }
-}
-
-module.exports = Helper;
diff --git a/web/assets/js/rep_log.js b/web/assets/js/rep_log.js
index b8460ca..ca79a28 100644
--- a/web/assets/js/rep_log.js
+++ b/web/assets/js/rep_log.js
@@ -1,5 +1,5 @@
 const $ = require('jquery');
-const RepLogApp = require('./RepLogApp');
+const RepLogApp = require('./Components/RepLogApp');
 
 $(document).ready(function() {
     var $wrapper = $('.js-rep-log-table');
