diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 2702274..a8f2a3c 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -55,11 +55,4 @@
     {{ parent() }}
 
     <script src="{{ asset('build/rep_log.js') }}"></script>
-
-    <script>
-        $(document).ready(function() {
-            var $wrapper = $('.js-rep-log-table');
-            var repLogApp = new RepLogApp($wrapper);
-        });
-    </script>
 {% endblock %}
diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 3917c20..51ba1d2 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -217,4 +217,4 @@ const rowTemplate = (repLog) => `
 </tr>
 `;
 
-window.RepLogApp = RepLogApp;
+module.exports = RepLogApp;
diff --git a/web/assets/js/rep_log.js b/web/assets/js/rep_log.js
new file mode 100644
index 0000000..b8460ca
--- /dev/null
+++ b/web/assets/js/rep_log.js
@@ -0,0 +1,7 @@
+const $ = require('jquery');
+const RepLogApp = require('./RepLogApp');
+
+$(document).ready(function() {
+    var $wrapper = $('.js-rep-log-table');
+    var repLogApp = new RepLogApp($wrapper);
+});
diff --git a/webpack.config.js b/webpack.config.js
index 38724e7..2f389cc 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -1,7 +1,7 @@
 const path = require('path');
 
 module.exports = {
-    entry: './web/assets/js/RepLogApp.js',
+    entry: './web/assets/js/rep_log.js',
     output: {
         path: path.resolve(__dirname, 'web', 'build'),
         filename: 'rep_log.js',
