diff --git a/app/Resources/views/base.html.twig b/app/Resources/views/base.html.twig
index 7d6993a..6c6daac 100644
--- a/app/Resources/views/base.html.twig
+++ b/app/Resources/views/base.html.twig
@@ -102,7 +102,7 @@
     <script src="{{ asset('bundles/fosjsrouting/js/router.js') }}"></script>
     <script src="{{ path('fos_js_routing_js', { callback: 'fos.Router.setData' }) }}"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/es6-promise/4.0.5/es6-promise.auto.min.js"></script>
-    <script src="{{ asset('assets/js/layout.js') }}"></script>
+    <script src="{{ asset('build/layout.js') }}"></script>
 {% endblock %}
 
 </body>
diff --git a/webpack.config.js b/webpack.config.js
index f009e7e..308c607 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -4,6 +4,7 @@ module.exports = {
     entry: {
         rep_log: './web/assets/js/rep_log.js',
         login: './web/assets/js/login.js',
+        layout: './web/assets/js/layout.js',
     },
     output: {
         path: path.resolve(__dirname, 'web', 'build'),
