diff --git a/app/Resources/FOSUserBundle/views/Security/login.html.twig b/app/Resources/FOSUserBundle/views/Security/login.html.twig
index e79438b..1e2ce3c 100644
--- a/app/Resources/FOSUserBundle/views/Security/login.html.twig
+++ b/app/Resources/FOSUserBundle/views/Security/login.html.twig
@@ -11,7 +11,7 @@
 {% block javascripts %}
     {{ parent() }}
 
-    <script src="{{ asset('assets/js/login.js') }}"></script>
+    <script src="{{ asset('build/login.js') }}"></script>
 {% endblock %}
 
 {% block fos_user_content %}
diff --git a/webpack.config.js b/webpack.config.js
index 2f389cc..f009e7e 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -1,9 +1,12 @@
 const path = require('path');
 
 module.exports = {
-    entry: './web/assets/js/rep_log.js',
+    entry: {
+        rep_log: './web/assets/js/rep_log.js',
+        login: './web/assets/js/login.js',
+    },
     output: {
         path: path.resolve(__dirname, 'web', 'build'),
-        filename: 'rep_log.js',
+        filename: '[name].js',
     }
 };
