diff --git a/app/Resources/views/lift/_form.html.twig b/app/Resources/views/lift/_form.html.twig
index 6cb6ddc..a980408 100644
--- a/app/Resources/views/lift/_form.html.twig
+++ b/app/Resources/views/lift/_form.html.twig
@@ -1,6 +1,6 @@
 {{ form_start(form, {
     'attr': {
-        'class': 'form-inline',
+        'class': 'form-inline js-new-rep-log-form',
         'novalidate': 'novalidate'
     }
 }) }}
diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 47b5c59..e2dee2d 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -2,7 +2,7 @@
 
 {% block body %}
     <div class="row">
-        <div class="col-md-7">
+        <div class="col-md-7 js-rep-log-table">
             <h2>
                 Lift History
                 <a href="#list-stuff-form" class="btn btn-md btn-success pull-right">
@@ -10,7 +10,7 @@
                 </a>
             </h2>
 
-            <table class="table table-striped js-rep-log-table">
+            <table class="table table-striped">
                 <thead>
                 <tr>
                     <th>What</th>
@@ -72,8 +72,8 @@
 
     <script>
         $(document).ready(function() {
-            var $table = $('.js-rep-log-table');
-            var repLogApp = new RepLogApp($table);
+            var $wrapper = $('.js-rep-log-table');
+            var repLogApp = new RepLogApp($wrapper);
         });
     </script>
 {% endblock %}
diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 2ffc230..723ec8f 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -13,6 +13,10 @@
             'click',
             this.handleRowClick.bind(this)
         );
+        this.$wrapper.find('.js-new-rep-log-form').on(
+            'submit',
+            this.handleNewFormSubmit.bind(this)
+        );
     };
 
     $.extend(window.RepLogApp.prototype, {
@@ -50,6 +54,11 @@
 
         handleRowClick: function () {
             console.log('row clicked!');
+        },
+
+        handleNewFormSubmit: function(e) {
+            e.preventDefault();
+            console.log('submitting!');
         }
     });
 
