diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index e2dee2d..2d9f71a 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -50,7 +50,9 @@
                 </tfoot>
             </table>
 
-            {{ include('lift/_form.html.twig') }}
+            <div class="js-new-rep-log-form-wrapper">
+                {{ include('lift/_form.html.twig') }}
+            </div>
         </div>
         <div class="col-md-5">
             <div class="leaderboard">
diff --git a/src/AppBundle/Controller/LiftController.php b/src/AppBundle/Controller/LiftController.php
index 4a7f017..654f3bb 100644
--- a/src/AppBundle/Controller/LiftController.php
+++ b/src/AppBundle/Controller/LiftController.php
@@ -40,6 +40,13 @@ class LiftController extends BaseController
             $totalWeight += $repLog->getTotalWeightLifted();
         }
 
+        // render just the form for AJAX, there is a validation error
+        if ($request->isXmlHttpRequest()) {
+            return $this->render('lift/_form.html.twig', [
+                'form' => $form->createView()
+            ]);
+        }
+
         return $this->render('lift/index.html.twig', array(
             'form' => $form->createView(),
             'repLogs' => $repLogs,
diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 75bcfc0..85a67a3 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -63,7 +63,10 @@
             $.ajax({
                 url: $form.attr('action'),
                 method: 'POST',
-                data: $form.serialize()
+                data: $form.serialize(),
+                success: function(data) {
+                    $form.closest('.js-new-rep-log-form-wrapper').html(data);
+                }
             });
         }
     });
