diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 585d1bf..2702274 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -48,13 +48,12 @@
 {% block stylesheets %}
     {{ parent() }}
 
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/sweetalert2/6.1.0/sweetalert2.min.css" />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/sweetalert2/6.6.0/sweetalert2.min.css" />
 {% endblock %}
 
 {% block javascripts %}
     {{ parent() }}
 
-    <script src="https://cdn.jsdelivr.net/sweetalert2/6.1.0/sweetalert2.min.js"></script>
     <script src="{{ asset('build/rep_log.js') }}"></script>
 
     <script>
diff --git a/package.json b/package.json
index 6f4afd9..3eef178 100644
--- a/package.json
+++ b/package.json
@@ -7,6 +7,7 @@
   "license": "MIT",
   "devDependencies": {
     "jquery": "^3.2.1",
+    "sweetalert2": "^6.6.6",
     "webpack": "^3.4.1"
   }
 }
diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 79f74ae..accad2b 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -2,8 +2,9 @@
 
 const Helper = require('./RepLogAppHelper');
 const $ = require('jquery');
+const swal = require('sweetalert2');
 
-(function(window, Routing, swal) {
+(function(window, Routing) {
 
     let HelperInstances = new WeakMap();
 
@@ -214,4 +215,4 @@ const $ = require('jquery');
 `;
 
     window.RepLogApp = RepLogApp;
-})(window, Routing, swal);
+})(window, Routing);
diff --git a/yarn.lock b/yarn.lock
index 3efe86d..f2420b1 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -1726,6 +1726,10 @@ supports-color@^4.2.1:
   dependencies:
     has-flag "^2.0.0"
 
+sweetalert2@^6.6.6:
+  version "6.6.6"
+  resolved "https://registry.yarnpkg.com/sweetalert2/-/sweetalert2-6.6.6.tgz#3630279d98f04fb8b89f2e8b098fc42f03bcb555"
+
 tapable@^0.2.7:
   version "0.2.7"
   resolved "https://registry.yarnpkg.com/tapable/-/tapable-0.2.7.tgz#e46c0daacbb2b8a98b9b0cea0f4052105817ed5c"
