diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 99d8178..d1efd6f 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -109,7 +109,7 @@
                 const url = Routing.generate('rep_log_new');
 
                 $.ajax({
-                    url: url,
+                    url,
                     method: 'POST',
                     data: JSON.stringify(data)
                 }).then((data, textStatus, jqXHR) => {
