diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 0ff7bd5..ef5c18e 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -2,7 +2,7 @@
 
 {% block body %}
     <div class="row">
-        <div class="col-md-7 js-rep-log-table">
+        <div class="col-md-7 js-rep-log-table" data-rep-logs="{{ repLogsJson|e('html_attr') }}">
             <h2>
                 Lift History
             </h2>
diff --git a/assets/js/rep_log.js b/assets/js/rep_log.js
index 26e6084..be2c3d2 100644
--- a/assets/js/rep_log.js
+++ b/assets/js/rep_log.js
@@ -6,24 +6,7 @@ const RepLogApp = require('./Components/RepLogApp');
 // ... even though I should put all my code here!
 global.$ = $;
 
-const logs = [
-    {
-        "links": {"_self": "\/reps\/78"},
-        "id": 78,
-        "reps": 1,
-        "itemLabel": "Big Fat Cat",
-        "totalWeightLifted": 18
-    },
-    {
-        "links": {"_self": "\/reps\/79"},
-        "id": 79,
-        "reps": 2,
-        "itemLabel": "Big Fat Cat",
-        "totalWeightLifted": 36
-    }
-];
-
 $(document).ready(function() {
     var $wrapper = $('.js-rep-log-table');
-    var repLogApp = new RepLogApp($wrapper, logs);
+    var repLogApp = new RepLogApp($wrapper, $wrapper.data('rep-logs'));
 });
diff --git a/src/AppBundle/Controller/LiftController.php b/src/AppBundle/Controller/LiftController.php
index 86c0f86..c9ef61b 100644
--- a/src/AppBundle/Controller/LiftController.php
+++ b/src/AppBundle/Controller/LiftController.php
@@ -33,9 +33,14 @@ class LiftController extends BaseController
             return $this->redirectToRoute('lift');
         }
 
+        $repLogModels = $this->findAllUsersRepLogModels();
+        $repLogsJson = $this->get('serializer')
+            ->serialize($repLogModels, 'json');
+
         return $this->render('lift/index.html.twig', array(
             'form' => $form->createView(),
             'leaderboard' => $this->getLeaders(),
+            'repLogsJson' => $repLogsJson,
         ));
     }
 
