diff --git a/assets/js/Components/RepLogApp.js b/assets/js/Components/RepLogApp.js
index 6c287f6..42e6f5b 100644
--- a/assets/js/Components/RepLogApp.js
+++ b/assets/js/Components/RepLogApp.js
@@ -8,13 +8,15 @@ require('sweetalert2/dist/sweetalert2.css');
 let HelperInstances = new WeakMap();
 
 class RepLogApp {
-    constructor($wrapper) {
+    constructor($wrapper, initialRepLogs) {
         this.$wrapper = $wrapper;
         this.repLogs = [];
 
         HelperInstances.set(this, new Helper(this.repLogs));
 
-        this.loadRepLogs();
+        for (let repLog of initialRepLogs) {
+            this._addRow(repLog);
+        }
 
         this.$wrapper.on(
             'click',
@@ -42,16 +44,6 @@ class RepLogApp {
         }
     }
 
-    loadRepLogs() {
-        $.ajax({
-            url: Routing.generate('rep_log_list'),
-        }).then(data => {
-            for (let repLog of data.items) {
-                this._addRow(repLog);
-            }
-        })
-    }
-
     updateTotalWeightLifted() {
         this.$wrapper.find('.js-total-weight').html(
             HelperInstances.get(this).getTotalWeightString()
diff --git a/assets/js/rep_log.js b/assets/js/rep_log.js
index 9bdfc1d..26e6084 100644
--- a/assets/js/rep_log.js
+++ b/assets/js/rep_log.js
@@ -6,7 +6,24 @@ const RepLogApp = require('./Components/RepLogApp');
 // ... even though I should put all my code here!
 global.$ = $;
 
+const logs = [
+    {
+        "links": {"_self": "\/reps\/78"},
+        "id": 78,
+        "reps": 1,
+        "itemLabel": "Big Fat Cat",
+        "totalWeightLifted": 18
+    },
+    {
+        "links": {"_self": "\/reps\/79"},
+        "id": 79,
+        "reps": 2,
+        "itemLabel": "Big Fat Cat",
+        "totalWeightLifted": 36
+    }
+];
+
 $(document).ready(function() {
     var $wrapper = $('.js-rep-log-table');
-    var repLogApp = new RepLogApp($wrapper);
+    var repLogApp = new RepLogApp($wrapper, logs);
 });
