diff --git a/templates/lift/index.html.twig b/templates/lift/index.html.twig
index 6342f65..e3dfa10 100644
--- a/templates/lift/index.html.twig
+++ b/templates/lift/index.html.twig
@@ -35,7 +35,9 @@
         <div class="col-md-5">
             <div class="leaderboard">
                 <h2 class="text-center">
+                    {#
                     <img class="dumbbell" src="{{ asset('build/static/dumbbell.png') }}" />
+                    #}
                     Leaderboard
                 </h2>
 
diff --git a/webpack.config.js b/webpack.config.js
index c300f11..0ad3d24 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -24,7 +24,7 @@ Encore
     .enableSassLoader()
     .enableSourceMaps(!Encore.isProduction())
     .cleanupOutputBeforeBuild()
-    .enableVersioning()
+    .enableVersioning(Encore.isProduction())
 ;
 
 // export the final configuration
