diff --git a/src/Controller/RepLogController.php b/src/Controller/RepLogController.php
index 11753354..0f751a03 100644
--- a/src/Controller/RepLogController.php
+++ b/src/Controller/RepLogController.php
@@ -4,12 +4,16 @@ namespace App\Controller;
 
 use App\Entity\RepLog;
 use App\Form\Type\RepLogType;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
 
+/**
+ * @IsGranted("ROLE_USER")
+ */
 class RepLogController extends BaseController
 {
     /**
