diff --git a/webpack.config.js b/webpack.config.js
index 9f11919..b98e8c7 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -147,6 +147,12 @@ if (isProduction) {
             debug: false
         })
     );
+
+    webpackConfig.plugins.push(
+        new webpack.DefinePlugin({
+            'process.env.NODE_ENV': JSON.stringify('production')
+        })
+    );
 }
 
-module.exports = webpackConfig;
\ No newline at end of file
+module.exports = webpackConfig;
