diff --git a/webpack.config.js b/webpack.config.js
index f19249e6..3922895c 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -5,29 +5,30 @@ const ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 const useDevServer = false;
 const publicPath = useDevServer ? 'http://localhost:8080/build/' : '/build/';
+const useSourcemaps = process.env.NODE_ENV !== 'production';
 
 const styleLoader = {
     loader: 'style-loader',
     options: {
-        sourceMap: true
+        sourceMap: useSourcemaps
     }
 };
 const cssLoader = {
     loader: 'css-loader',
     options: {
-        sourceMap: true
+        sourceMap: useSourcemaps
     }
 };
 const sassLoader = {
     loader: 'sass-loader',
     options: {
-        sourceMap: true
+        sourceMap: useSourcemaps
     }
 };
 const resolveUrlLoader = {
     loader: 'resolve-url-loader',
     options: {
-        sourceMap: true
+        sourceMap: useSourcemaps
     }
 };
 
@@ -124,7 +125,7 @@ const webpackConfig = {
 
         new ExtractTextPlugin('[name].css'),
     ],
-    devtool: 'inline-source-map',
+    devtool: useSourcemaps ? 'inline-source-map' : false,
     devServer: {
         contentBase: './web',
         headers: { 'Access-Control-Allow-Origin': '*' },
