diff --git a/webpack.config.js b/webpack.config.js
index cf26af2..525f14d 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -6,6 +6,8 @@ const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const useDevServer = false;
 const publicPath = useDevServer ? 'http://localhost:8080/build/' : '/build/';
 
+console.log(process.env.NODE_ENV);
+
 const styleLoader = {
     loader: 'style-loader',
     options: {
