diff --git a/webpack.config.js b/webpack.config.js
index 525f14d..de9d50d 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -6,8 +6,6 @@ const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const useDevServer = false;
 const publicPath = useDevServer ? 'http://localhost:8080/build/' : '/build/';
 
-console.log(process.env.NODE_ENV);
-
 const styleLoader = {
     loader: 'style-loader',
     options: {
@@ -33,7 +31,7 @@ const resolveUrlLoader = {
     }
 };
 
-module.exports = {
+const webpackConfig = {
     entry: {
         rep_log: './assets/js/rep_log.js',
         login: './assets/js/login.js',
@@ -132,3 +130,11 @@ module.exports = {
         headers: { 'Access-Control-Allow-Origin': '*' },
     }
 };
+
+if (process.env.NODE_ENV === 'production') {
+    webpackConfig.plugins.push(
+        new webpack.optimize.UglifyJsPlugin()
+    );
+}
+
+module.exports = webpackConfig;
\ No newline at end of file
