diff --git a/webpack.config.js b/webpack.config.js
index f19249e..86d8731 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -5,17 +5,19 @@ const ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 const useDevServer = false;
 const publicPath = useDevServer ? 'http://localhost:8080/build/' : '/build/';
+const isProduction = process.env.NODE_ENV === 'production';
+const useSourcemaps = !isProduction;
 
 const styleLoader = {
     loader: 'style-loader',
     options: {
-        sourceMap: true
+        sourceMap: useSourcemaps
     }
 };
 const cssLoader = {
     loader: 'css-loader',
     options: {
-        sourceMap: true
+        sourceMap: useSourcemaps
     }
 };
 const sassLoader = {
@@ -27,7 +29,7 @@ const sassLoader = {
 const resolveUrlLoader = {
     loader: 'resolve-url-loader',
     options: {
-        sourceMap: true
+        sourceMap: useSourcemaps
     }
 };
 
@@ -124,14 +126,14 @@ const webpackConfig = {
 
         new ExtractTextPlugin('[name].css'),
     ],
-    devtool: 'inline-source-map',
+    devtool: useSourcemaps ? 'inline-source-map' : false,
     devServer: {
         contentBase: './web',
         headers: { 'Access-Control-Allow-Origin': '*' },
     }
 };
 
-if (process.env.NODE_ENV === 'production') {
+if (isProduction) {
     webpackConfig.plugins.push(
         new webpack.optimize.UglifyJsPlugin()
     );
