diff --git a/webpack.config.js b/webpack.config.js
index de9d50d..f19249e 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -135,6 +135,15 @@ if (process.env.NODE_ENV === 'production') {
     webpackConfig.plugins.push(
         new webpack.optimize.UglifyJsPlugin()
     );
+
+    // passes these options to all loaders
+    // but we should really pass these ourselves
+    webpackConfig.plugins.push(
+        new webpack.LoaderOptionsPlugin({
+            minimize: true,
+            debug: false
+        })
+    );
 }
 
 module.exports = webpackConfig;
\ No newline at end of file
