diff --git a/src/AppBundle/Controller/RepLogController.php b/src/AppBundle/Controller/RepLogController.php
index cbd4957..2d01311 100644
--- a/src/AppBundle/Controller/RepLogController.php
+++ b/src/AppBundle/Controller/RepLogController.php
@@ -59,7 +59,7 @@ class RepLogController extends BaseController
     }
 
     /**
-     * @Route("/reps", name="rep_log_new")
+     * @Route("/reps", name="rep_log_new", options={"expose" = true})
      * @Method("POST")
      */
     public function newRepLogAction(Request $request)
diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 0a25ba9..20cfa89 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -81,13 +81,9 @@
             var $form = $(e.currentTarget);
 
             var self = this;
-            $.ajax({
-                url: $form.data('url'),
-                method: 'POST',
-                data: JSON.stringify({
-                    item: $form.find('[name="item"]').val(),
-                    reps: $form.find('[name="reps"]').val()
-                })
+            this._saveRepLog({
+                item: $form.find('[name="item"]').val(),
+                reps: $form.find('[name="reps"]').val()
             }).done(function(data) {
                 self._clearForm();
                 self._addRow(data);
@@ -97,6 +93,14 @@
             });
         },
 
+        _saveRepLog: function(data) {
+            return $.ajax({
+                url: Routing.generate('rep_log_new'),
+                method: 'POST',
+                data: JSON.stringify(data)
+            });
+        },
+
         _mapErrorsToForm: function(errorData) {
             this._removeFormErrors();
             var $form = this.$wrapper.find(this._selectors.newRepForm);
