diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 04e736c..b1584c9 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -89,6 +89,7 @@
                 self._clearForm();
                 self._addRow(data);
             }).catch(function(jqXHR) {
+                console.log(jqXHR);
                 var errorData = JSON.parse(jqXHR.responseText);
                 self._mapErrorsToForm(errorData.errors);
             });
