diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 7006d85..8b1323d 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -89,11 +89,13 @@
                 self._clearForm();
                 self._addRow(data);
             }).catch(function(jqXHR) {
-                if (jqXHR instanceof ReferenceError) {
-                    console.log('wow!');
+                if (typeof jqXHR.responseText === 'undefined') {
+                    throw jqXHR;
                 }
                 var errorData = JSON.parse(jqXHR.responseText);
                 self._mapErrorsToForm(errorData.errors);
+            }).catch(function(e) {
+                console.log(e);
             });
         },
 
