diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index f33f70e..0010e1c 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -98,10 +98,10 @@
                     var errorData = JSON.parse(jqXHR.responseText);
                     self._mapErrorsToForm(errorData.errors);
                 }
-            }).done(function(data) {
+            }).then(function(data) {
                 console.log('I am successful!');
                 console.log(data);
-            }).done(function(data) {
+            }).then(function(data) {
                 console.log('another handler!');
                 console.log(data);
             })
