diff --git a/src/AppBundle/Controller/RepLogController.php b/src/AppBundle/Controller/RepLogController.php
index f93e19f..76b2bd1 100644
--- a/src/AppBundle/Controller/RepLogController.php
+++ b/src/AppBundle/Controller/RepLogController.php
@@ -84,7 +84,8 @@ class RepLogController extends BaseController
 
         $apiModel = $this->createRepLogApiModel($repLog);
 
-        $response = $this->createApiResponse($apiModel);
+        //$response = $this->createApiResponse($apiModel);
+        $response = new Response(null, 204);
         // setting the Location header... it's a best-practice
         $response->headers->set(
             'Location',
