diff --git a/src/AppBundle/Controller/RepLogController.php b/src/AppBundle/Controller/RepLogController.php
index 44cc485..f93e19f 100644
--- a/src/AppBundle/Controller/RepLogController.php
+++ b/src/AppBundle/Controller/RepLogController.php
@@ -52,7 +52,7 @@ class RepLogController extends BaseController
     }
 
     /**
-     * @Route("/reps", name="rep_log_new")
+     * @Route("/reps", name="rep_log_new", options={"expose" = true})
      * @Method("POST")
      */
     public function newRepLogAction(Request $request)
diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 8e9a68f..04e736c 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -84,11 +84,8 @@
                 formData[fieldData.name] = fieldData.value
             });
             var self = this;
-            $.ajax({
-                url: $form.data('url'),
-                method: 'POST',
-                data: JSON.stringify(formData)
-            }).then(function(data) {
+            this._saveRepLog(formData)
+            .then(function(data) {
                 self._clearForm();
                 self._addRow(data);
             }).catch(function(jqXHR) {
@@ -97,6 +94,14 @@
             });
         },
 
+        _saveRepLog: function(data) {
+            return $.ajax({
+                url: Routing.generate('rep_log_new'),
+                method: 'POST',
+                data: JSON.stringify(data)
+            });
+        },
+
         _mapErrorsToForm: function(errorData) {
             this._removeFormErrors();
             var $form = this.$wrapper.find(this._selectors.newRepForm);
