diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 34e5a05..8e9a68f 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -33,12 +33,11 @@
             var self = this;
             $.ajax({
                 url: Routing.generate('rep_log_list'),
-                success: function(data) {
-                    $.each(data.items, function(key, repLog) {
-                        self._addRow(repLog);
-                    });
-                }
-            });
+            }).then(function(data) {
+                $.each(data.items, function(key, repLog) {
+                    self._addRow(repLog);
+                });
+            })
         },
 
         updateTotalWeightLifted: function () {
@@ -63,14 +62,13 @@
             var self = this;
             $.ajax({
                 url: deleteUrl,
-                method: 'DELETE',
-                success: function () {
-                    $row.fadeOut('normal', function () {
-                        $(this).remove();
-                        self.updateTotalWeightLifted();
-                    });
-                }
-            });
+                method: 'DELETE'
+            }).then(function() {
+                $row.fadeOut('normal', function () {
+                    $(this).remove();
+                    self.updateTotalWeightLifted();
+                });
+            })
         },
 
         handleRowClick: function () {
@@ -89,26 +87,13 @@
             $.ajax({
                 url: $form.data('url'),
                 method: 'POST',
-                data: JSON.stringify(formData),
-                success: function(data) {
-                    self._clearForm();
-                    self._addRow(data);
-                },
-                error: function(jqXHR) {
-                    var errorData = JSON.parse(jqXHR.responseText);
-                    self._mapErrorsToForm(errorData.errors);
-                }
-            }).then(function(data) {
-                console.log('I am successful!');
-                console.log(data);
-
-                return data;
+                data: JSON.stringify(formData)
             }).then(function(data) {
-                console.log('another handler!');
-                console.log(data);
+                self._clearForm();
+                self._addRow(data);
             }).catch(function(jqXHR) {
-                console.log('failed!');
-                console.log(jqXHR.responseText);
+                var errorData = JSON.parse(jqXHR.responseText);
+                self._mapErrorsToForm(errorData.errors);
             });
         },
 
