diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 70929f86..79bb715b 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -6,7 +6,7 @@ export default class RepLogApp extends Component {
         super(props);
 
         this.state = {
-            highlightedRowId: 'cool!'
+            highlightedRowId: null,
         };
 
         this.handleRowClick = this.handleRowClick.bind(this);
@@ -40,7 +40,6 @@ export default class RepLogApp extends Component {
                     </thead>
                     <RepLogList
                         highlightedRowId={highlightedRowId}
-                        onRowClick={this.handleRowClick}
                     />
                     <tfoot>
                     <tr>
diff --git a/assets/js/RepLog/RepLogList.js b/assets/js/RepLog/RepLogList.js
index 2ecc1e32..f05345eb 100644
--- a/assets/js/RepLog/RepLogList.js
+++ b/assets/js/RepLog/RepLogList.js
@@ -30,5 +30,5 @@ export default function RepLogList(props) {
 
 RepLogList.propTypes = {
     highlightedRowId: PropTypes.any,
-    onRowClick: PropTypes.func
+    onRowClick: PropTypes.func.isRequired
 };
