diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 3dcb6fc4..8ebb3dec 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -24,6 +24,7 @@ export default function RepLogs(props) {
                 </thead>
                 <RepLogList
                     highlightedRowId={highlightedRowId}
+                    onRowClick={onRowClick}
                 />
                 <tfoot>
                 <tr>
