diff --git a/assets/js/RepLog/RepLogList.js b/assets/js/RepLog/RepLogList.js
index 6a02498d..2ecc1e32 100644
--- a/assets/js/RepLog/RepLogList.js
+++ b/assets/js/RepLog/RepLogList.js
@@ -1,4 +1,5 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 export default function RepLogList(props) {
     const { highlightedRowId, onRowClick } = props;
@@ -26,3 +27,8 @@ export default function RepLogList(props) {
         </tbody>
     );
 }
+
+RepLogList.propTypes = {
+    highlightedRowId: PropTypes.any,
+    onRowClick: PropTypes.func
+};
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 8ebb3dec..6c690928 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -24,7 +24,7 @@ export default function RepLogs(props) {
                 </thead>
                 <RepLogList
                     highlightedRowId={highlightedRowId}
-                    onRowClick={onRowClick}
+                    onRowClick="Cool!"
                 />
                 <tfoot>
                 <tr>
