diff --git a/webpack.config.js b/webpack.config.js
index 9572541..07d8256 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -31,10 +31,10 @@ Encore
     .enableReactPreset()
 
     .configureBabel((babelConfig) => {
-        babelConfig.env = {
-            "production": {
-              "plugins": ["transform-react-remove-prop-types"]
-            }
+        if (Encore.isProduction()) {
+            babelConfig.plugins.push(
+                'transform-react-remove-prop-types'
+            );
         }
     }, {
         useBuiltIns: 'usage',
