diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index d1d86e3a..1416e9ef 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import RepLogList from './RepLogList';
+import PropTypes from 'prop-types';
 
 export default class RepLogApp extends Component {
     constructor(props) {
@@ -92,3 +93,7 @@ export default class RepLogApp extends Component {
         );
     }
 }
+
+RepLogApp.propTypes = {
+    withHeart: PropTypes.bool
+}
\ No newline at end of file
