diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 1921530..d622ae5 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -5,15 +5,11 @@
         initialize: function ($wrapper) {
             this.$wrapper = $wrapper;
             this.helper = new Helper(this.$wrapper);
-            console.log(this.helper, Object.keys(this.helper));
-            console.log(Helper, Object.keys(Helper));
-            console.log(this.helper.calculateTotalWeight());
-
-            var playObject = {
-                lift: 'stuff'
-            };
-            playObject.__proto__.cat = 'meow';
-            console.log(playObject.lift, playObject.cat);
+            console.log(
+                'foo'.__proto__,
+                [].__proto__,
+                (new Date()).__proto__
+            );
 
             this.$wrapper.find('.js-delete-rep-log').on(
                 'click',
