diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index e2c73b4..213b6a7 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -7,6 +7,7 @@
             this.helper = new Helper(this.$wrapper);
             console.log(this.helper, Object.keys(this.helper));
             console.log(Helper, Object.keys(Helper));
+            console.log(this.helper.calculateTotalWeight());
 
             this.$wrapper.find('.js-delete-rep-log').on(
                 'click',
@@ -60,7 +61,7 @@
     var Helper = function ($wrapper) {
         this.$wrapper = $wrapper;
     };
-    Helper.calculateTotalWeight = function() {
+    Helper.prototype.calculateTotalWeight = function() {
         var totalWeight = 0;
         this.$wrapper.find('tbody tr').each(function () {
             totalWeight += $(this).data('weight');
