diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index af7dcf7..85c6f37 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -4,7 +4,7 @@
     window.RepLogApp = {
         initialize: function ($wrapper) {
             this.$wrapper = $wrapper;
-            Helper.initialize(this.$wrapper);
+            this.helper = new Helper(this.$wrapper);
 
             this.$wrapper.find('.js-delete-rep-log').on(
                 'click',
@@ -17,7 +17,7 @@
         },
         updateTotalWeightLifted: function () {
             this.$wrapper.find('.js-total-weight').html(
-                Helper.calculateTotalWeight()
+                this.helper.calculateTotalWeight()
             );
         },
 
@@ -55,9 +55,7 @@
     /**
      * A "private" object
      */
-    var Helper = {};
-
-    Helper.initialize = function ($wrapper) {
+    var Helper = function ($wrapper) {
         this.$wrapper = $wrapper;
     };
     Helper.calculateTotalWeight = function() {
