diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index e55d963..cd866a1 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -5,11 +5,6 @@
         initialize: function ($wrapper) {
             this.$wrapper = $wrapper;
             this.helper = new Helper(this.$wrapper);
-            var helper2 = new Helper($('footer'));
-            console.log(
-                this.helper.calculateTotalWeight(),
-                helper2.calculateTotalWeight()
-            );
 
             this.$wrapper.find('.js-delete-rep-log').on(
                 'click',
