diff --git a/web/assets/js/RepLogTable.js b/web/assets/js/RepLogTable.js
index e91b5b9..e7fa3a7 100644
--- a/web/assets/js/RepLogTable.js
+++ b/web/assets/js/RepLogTable.js
@@ -55,18 +55,18 @@
     /**
      * A "private" object
      */
-    var Helper = {
-        initialize: function ($wrapper) {
-            this.$wrapper = $wrapper;
-        },
+    var Helper = {};
 
-        calculateTotalWeight: function() {
-            var totalWeight = 0;
-            this.$wrapper.find('tbody tr').each(function () {
-                totalWeight += $(this).data('weight');
-            });
+    Helper.initialize = function ($wrapper) {
+        this.$wrapper = $wrapper;
+    };
+    Helper.calculateTotalWeight = function() {
+        var totalWeight = 0;
+        this.$wrapper.find('tbody tr').each(function () {
+            totalWeight += $(this).data('weight');
+        });
 
-            return totalWeight;
-        }
+        return totalWeight;
     };
+
 })(window, jQuery);
