diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 1aacc8da..890f4401 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -31,9 +31,15 @@ export default class RepLogApp extends Component {
         const quantityInput = this.quantityInput.current;
         const itemSelect = this.itemSelect.current;
 
-        console.log('I love when a good form submits!');
-        console.log(quantityInput.value);
-        console.log(itemSelect.options[itemSelect.selectedIndex].value);
+        const repLogs = this.state.repLogs;
+        const newRep = {
+            id: 'TODO-id',
+            reps: quantityInput.value,
+            itemLabel: itemSelect.options[itemSelect.selectedIndex].text,
+            totalWeightLifted: Math.floor(Math.random() * 50)
+        };
+        repLogs.push(newRep);
+        this.setState({repLogs: repLogs});
     }
 
     calculateTotalWeightLifted() {
