diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 41133224..9628263c 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -33,6 +33,9 @@ export default class RepLogs extends Component {
             itemSelect.options[itemSelect.selectedIndex].text,
             quantityInput.value
         );
+
+        quantityInput.value = '';
+        itemSelect.selectedIndex = 0;
     }
 
     render() {
