diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 13bc55de..0e7659fc 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -22,6 +22,9 @@ export default class RepLogCreator extends Component {
             itemSelect.options[itemSelect.selectedIndex].text,
             quantityInput.value
         );
+
+        quantityInput.value = '';
+        itemSelect.selectedIndex = 0;
     }
 
     render() {
