diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 484d0768..679593a7 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -29,9 +29,12 @@ export default class RepLogApp extends Component {
     handleNewItemSubmit(event) {
         event.preventDefault();
 
+        const quantityInput = this.quantityInput.current;
+        const itemSelect = this.itemSelect.current;
+
         console.log('I love when a good form submits!');
-        console.log(this.quantityInput);
-        console.log(this.itemSelect);
+        console.log(quantityInput.value);
+        console.log(itemSelect.options[itemSelect.selectedIndex].value);
     }
 
     calculateTotalWeightLifted() {
