diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index e45802f45..e85efada8 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -5,6 +5,9 @@ export default class RepLogCreator extends Component {
     constructor(props) {
         super(props);
 
+        this.quantityInput = React.createRef();
+        this.itemSelect = React.createRef();
+
         this.handleFormSubmit = this.handleFormSubmit.bind(this);
     }
 
@@ -13,9 +16,10 @@ export default class RepLogCreator extends Component {
         const { onNewItemSubmit } = this.props;
 
         console.log('I love when a good form submits!');
-        console.log(event.target.elements.namedItem('reps').value);
+        console.log(this.quantityInput);
+        console.log(this.itemSelect);
 
-        onNewItemSubmit('Big Fat Cat', event.target.elements.namedItem('reps').value);
+        //onNewItemSubmit('Big Fat Cat', event.target.elements.namedItem('reps').value);
     }
 
     render() {
@@ -27,7 +31,7 @@ export default class RepLogCreator extends Component {
                         What did you lift?
                     </label>
                     <select id="rep_log_item"
-                            name="item"
+                            ref={this.itemSelect}
                             required="required"
                             className="form-control">
                         <option value="">What did you
@@ -46,7 +50,8 @@ export default class RepLogCreator extends Component {
                         How many times?
                     </label>
                     <input type="number" id="rep_log_reps"
-                           name="reps" required="required"
+                           ref={this.quantityInput}
+                           required="required"
                            placeholder="How many times?"
                            className="form-control"/>
                 </div>
