diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index e4bd40f0..93956c85 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -15,9 +15,12 @@ export default class RepLogCreator extends Component {
         event.preventDefault();
         const { onNewItemSubmit } = this.props;
 
+        const quantityInput = this.quantityInput.current;
+        const itemSelect = this.itemSelect.current;
+
         console.log('I love when a good form submits!');
-        console.log(this.quantityInput);
-        console.log(this.itemSelect);
+        console.log(quantityInput.value);
+        console.log(itemSelect.options[itemSelect.selectedIndex].value);
 
         //onNewItemSubmit('Big Fat Cat', event.target.elements.namedItem('reps').value);
     }
