diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 86b5bdb5..0a977fb9 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -2,6 +2,31 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
 export default class RepLogCreator extends Component {
+    constructor(props) {
+        super(props);
+
+        this.quantityInput = React.createRef();
+        this.itemSelect = React.createRef();
+
+        this.handleFormSubmit = this.handleFormSubmit.bind(this);
+    }
+
+    handleFormSubmit(event) {
+        event.preventDefault();
+        const { onNewItemSubmit } = this.props;
+
+        const quantityInput = this.quantityInput.current;
+        const itemSelect = this.itemSelect.current;
+
+        onNewItemSubmit(
+            itemSelect.options[itemSelect.selectedIndex].text,
+            quantityInput.value
+        );
+
+        quantityInput.value = '';
+        itemSelect.selectedIndex = 0;
+    }
+
     render() {
         return (
             <form className="form-inline" onSubmit={this.handleFormSubmit}>
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 9a0f3bd0..718a8bbb 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -6,11 +6,6 @@ import RepLogCreator from './RepLogCreator';
 export default class RepLogs extends Component {
     constructor(props) {
         super(props);
-
-        this.quantityInput = React.createRef();
-        this.itemSelect = React.createRef();
-
-        this.handleFormSubmit = this.handleFormSubmit.bind(this);
     }
 
     calculateTotalWeightLifted(repLogs) {
@@ -23,22 +18,6 @@ export default class RepLogs extends Component {
         return total;
     }
 
-    handleFormSubmit(event) {
-        event.preventDefault();
-        const { onNewItemSubmit } = this.props;
-
-        const quantityInput = this.quantityInput.current;
-        const itemSelect = this.itemSelect.current;
-
-        onNewItemSubmit(
-            itemSelect.options[itemSelect.selectedIndex].text,
-            quantityInput.value
-        );
-
-        quantityInput.value = '';
-        itemSelect.selectedIndex = 0;
-    }
-
     render() {
         const { withHeart, highlightedRowId, onRowClick, repLogs } = this.props;
 
