diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 12edc2ec..efa181fb 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -4,6 +4,9 @@ import PropTypes from 'prop-types';
 export default function RepLogCreator(props) {
     const { handleNewItemSubmit } = props;
 
+    let itemSelect = null;
+    let quantityInput = null;
+
     return (
         <form className="form-inline" onSubmit={handleNewItemSubmit}>
             <div className="form-group">
@@ -12,7 +15,7 @@ export default function RepLogCreator(props) {
                     What did you lift?
                 </label>
                 <select id="rep_log_item"
-                        ref={this.itemSelect}
+                        ref={(ref) => { itemSelect = ref; }}
                         required="required"
                         className="form-control">
                     <option value="">What did you
@@ -31,7 +34,7 @@ export default function RepLogCreator(props) {
                     How many times?
                 </label>
                 <input type="number" id="rep_log_reps"
-                       ref={this.quantityInput}
+                       ref={(ref) => { quantityInput = ref }}
                        required="required"
                        placeholder="How many times?"
                        className="form-control"/>
