diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 8124d698..6f0d2fa3 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -3,6 +3,41 @@ import PropTypes from 'prop-types';
 
 export default function RepLogCreator(props) {
     return (
-        <div>I'm going to be a form when I grow up!</div>
+        <form className="form-inline" onSubmit={this.handleFormSubmit}>
+            <div className="form-group">
+                <label className="sr-only control-label required"
+                       htmlFor="rep_log_item">
+                    What did you lift?
+                </label>
+                <select id="rep_log_item"
+                        ref={this.itemSelect}
+                        required="required"
+                        className="form-control">
+                    <option value="">What did you
+                        lift?
+                    </option>
+                    <option value="cat">Cat</option>
+                    <option value="fat_cat">Big Fat Cat</option>
+                    <option value="laptop">My Laptop</option>
+                    <option value="coffee_cup">Coffee Cup</option>
+                </select>
+            </div>
+            {' '}
+            <div className="form-group">
+                <label className="sr-only control-label required"
+                       htmlFor="rep_log_reps">
+                    How many times?
+                </label>
+                <input type="number" id="rep_log_reps"
+                       ref={this.quantityInput}
+                       required="required"
+                       placeholder="How many times?"
+                       className="form-control"/>
+            </div>
+            {' '}
+            <button type="submit" className="btn btn-primary">I Lifted
+                it!
+            </button>
+        </form>
     );
 }
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index b5554978..9a0f3bd0 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -76,43 +76,6 @@ export default class RepLogs extends Component {
                 </table>
 
                 <RepLogCreator/>
-
-                <form className="form-inline" onSubmit={this.handleFormSubmit}>
-                    <div className="form-group">
-                        <label className="sr-only control-label required"
-                               htmlFor="rep_log_item">
-                            What did you lift?
-                        </label>
-                        <select id="rep_log_item"
-                                ref={this.itemSelect}
-                                required="required"
-                                className="form-control">
-                            <option value="">What did you
-                                lift?
-                            </option>
-                            <option value="cat">Cat</option>
-                            <option value="fat_cat">Big Fat Cat</option>
-                            <option value="laptop">My Laptop</option>
-                            <option value="coffee_cup">Coffee Cup</option>
-                        </select>
-                    </div>
-                    {' '}
-                    <div className="form-group">
-                        <label className="sr-only control-label required"
-                               htmlFor="rep_log_reps">
-                            How many times?
-                        </label>
-                        <input type="number" id="rep_log_reps"
-                               ref={this.quantityInput}
-                               required="required"
-                               placeholder="How many times?"
-                               className="form-control"/>
-                    </div>
-                    {' '}
-                    <button type="submit" className="btn btn-primary">I Lifted
-                        it!
-                    </button>
-                </form>
             </div>
         );
     }
