diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
new file mode 100644
index 00000000..8124d698
--- /dev/null
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -0,0 +1,8 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function RepLogCreator(props) {
+    return (
+        <div>I'm going to be a form when I grow up!</div>
+    );
+}
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index fefd8dd1..2af2f019 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import RepLogList from './RepLogList';
+import RepLogCreator from './RepLogCreator';
 
 export default class RepLogs extends Component {
     constructor(props) {
@@ -74,6 +75,8 @@ export default class RepLogs extends Component {
                     </tfoot>
                 </table>
 
+                <RepLogCreator/>
+
                 <form className="form-inline" onSubmit={this.handleFormSubmit}>
                     <div className="form-group">
                         <label className="sr-only control-label required"
