diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 717b33c8..e893bb57 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -15,7 +15,13 @@ function calculateTotalWeightLifted(repLogs) {
 const calculateTotalWeightFancier = repLogs => repLogs.reduce((total, log) => total + log.totalWeightLifted, 0);
 
 export default function RepLogs(props) {
-    const { withHeart, highlightedRowId, onRowClick, repLogs, onAddRepLog } = props;
+    const {
+        withHeart,
+        highlightedRowId,
+        onRowClick,
+        repLogs,
+        onAddRepLog
+    } = props;
 
     let heart = '';
     if (withHeart) {
