diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index a57325707..604538847 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -17,14 +17,14 @@ export default class RepLogApp extends Component {
         };
 
         this.handleRowClick = this.handleRowClick.bind(this);
-        this.handleNewItemSubmit = this.handleNewItemSubmit.bind(this);
+        this.handleAddRepLog = this.handleAddRepLog.bind(this);
     }
 
     handleRowClick(repLogId) {
         this.setState({highlightedRowId: repLogId});
     }
 
-    handleNewItemSubmit(itemLabel, reps) {
+    handleAddRepLog(itemLabel, reps) {
         const newRep = {
             id: uuid(),
             reps: reps,
@@ -45,7 +45,7 @@ export default class RepLogApp extends Component {
                 {...this.props}
                 {...this.state}
                 onRowClick={this.handleRowClick}
-                onNewItemSubmit={this.handleNewItemSubmit}
+                onAddRepLog={this.handleAddRepLog}
             />
         )
     }
diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 0e7659fc1..1af3bdb18 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -13,12 +13,12 @@ export default class RepLogCreator extends Component {
 
     handleFormSubmit(event) {
         event.preventDefault();
-        const { onNewItemSubmit } = this.props;
+        const { onAddRepLog } = this.props;
 
         const quantityInput = this.quantityInput.current;
         const itemSelect = this.itemSelect.current;
 
-        onNewItemSubmit(
+        onAddRepLog(
             itemSelect.options[itemSelect.selectedIndex].text,
             quantityInput.value
         );
@@ -70,5 +70,5 @@ export default class RepLogCreator extends Component {
 }
 
 RepLogCreator.propTypes = {
-    onNewItemSubmit: PropTypes.func.isRequired,
+    onAddRepLog: PropTypes.func.isRequired,
 };
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 8f9bf8054..1a3581ba2 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -15,7 +15,7 @@ function calculateTotalWeightLifted(repLogs) {
 const calculateTotalWeightFancier = repLogs => repLogs.reduce((total, log) => total + log.totalWeightLifted, 0);
 
 export default function RepLogs(props) {
-    const { withHeart, highlightedRowId, onRowClick, repLogs, onNewItemSubmit } = props;
+    const { withHeart, highlightedRowId, onRowClick, repLogs, onAddRepLog } = props;
 
     let heart = '';
     if (withHeart) {
@@ -51,7 +51,7 @@ export default function RepLogs(props) {
             </table>
 
             <RepLogCreator
-                onNewItemSubmit={onNewItemSubmit}
+                onAddRepLog={onAddRepLog}
             />
         </div>
     );
@@ -61,6 +61,6 @@ RepLogs.propTypes = {
     withHeart: PropTypes.bool,
     highlightedRowId: PropTypes.any,
     onRowClick: PropTypes.func.isRequired,
-    onNewItemSubmit: PropTypes.func.isRequired,
+    onAddRepLog: PropTypes.func.isRequired,
     repLogs: PropTypes.array.isRequired
 };
