diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 0a977fb9..0e7659fc 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -68,3 +68,7 @@ export default class RepLogCreator extends Component {
         );
     }
 }
+
+RepLogCreator.propTypes = {
+    onNewItemSubmit: PropTypes.func.isRequired,
+};
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 718a8bbb..6490c0cb 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -19,7 +19,7 @@ export default class RepLogs extends Component {
     }
 
     render() {
-        const { withHeart, highlightedRowId, onRowClick, repLogs } = this.props;
+        const { withHeart, highlightedRowId, onRowClick, repLogs, onNewItemSubmit } = this.props;
 
         let heart = '';
         if (withHeart) {
@@ -54,7 +54,7 @@ export default class RepLogs extends Component {
                     </tfoot>
                 </table>
 
-                <RepLogCreator/>
+                <RepLogCreator onNewItemSubmit={onNewItemSubmit}/>
             </div>
         );
     }
