diff --git a/app/Resources/views/base.html.twig b/app/Resources/views/base.html.twig
index ab39ebd..924b36e 100644
--- a/app/Resources/views/base.html.twig
+++ b/app/Resources/views/base.html.twig
@@ -124,7 +124,7 @@
 </div>
 <!-- /container -->
 
-{% block javascripts %}
+    {#
     <script src="{{ asset('assets/vendor/jquery/jquery.js') }}"></script>
     <script src="{{ asset('assets/vendor/bootstrap/dist/js/bootstrap.js') }}"></script>
     <script src="{{ asset('assets/vendor/eventEmitter/EventEmitter.js') }}"></script>
@@ -136,6 +136,6 @@
         // create a global eventEmitter
         window.eventEmitter = new EventEmitter();
     </script>
-{% endblock %}
+    #}
 </body>
 </html>
