diff --git a/web/assets/js/common.js b/web/assets/js/common.js
index c527717..d1835dc 100644
--- a/web/assets/js/common.js
+++ b/web/assets/js/common.js
@@ -18,7 +18,7 @@ requirejs.config({
         eventEmitter: '../vendor/eventEmitter/EventEmitter',
         dust: '../vendor/dustjs-linkedin/dist/dust-full',
         text: '../vendor/requirejs-text/text',
-        routing: '../../bundles/fosjsrouting/js/router'
+        'vendor/routing': '../../bundles/fosjsrouting/js/router'
     },
     shim: {
         /**
@@ -39,11 +39,11 @@ requirejs.config({
             exports: 'dust'
         },
 
-        routing: {
+        "vendor/routing": {
             exports: 'Routing'
         },
 
-        routes: ['routing'],
+        routes: ['vendor/routing'],
 
         'plugins/jquery.userinfopopup': ['jquery', 'routes']
     }
diff --git a/web/assets/js/modules/Leaderboard.js b/web/assets/js/modules/Leaderboard.js
index dc6108b..0763161 100644
--- a/web/assets/js/modules/Leaderboard.js
+++ b/web/assets/js/modules/Leaderboard.js
@@ -1,5 +1,4 @@
-define(['jquery', 'dust', 'text!templates/leaderboard.dust', 'routing', 'routes', 'plugins/jquery.userinfopopup', 'bootstrap'], function ($, dust, leaderboardTemplate, Routing) {
-
+define(['jquery', 'dust', 'text!templates/leaderboard.dust', 'modules/Routing', 'plugins/jquery.userinfopopup', 'bootstrap'], function ($, dust, leaderboardTemplate, Routing) {
     var Leaderboard = function($wrapper, leaders, eventEmitter) {
         this.$wrapper = $wrapper;
         this.leaders = leaders;
diff --git a/web/assets/js/modules/Routing.js b/web/assets/js/modules/Routing.js
new file mode 100644
index 0000000..bc9995a
--- /dev/null
+++ b/web/assets/js/modules/Routing.js
@@ -0,0 +1,6 @@
+/**
+ * Exists just to make a "modules/routing" that grabs the routing and routes
+ */
+define(['vendor/routing', 'routes'], function (routing, routes) {
+    return routing;
+});
