diff --git a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
index 7f9c992..a0ea72d 100644
--- a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
+++ b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
@@ -36,30 +36,8 @@
     </div>
 {% endblock %}
 
-{% block javascripts %}
-    {{ parent() }}
-
-    <script src="{{ asset('assets/vendor/dustjs-linkedin/dist/dust-full.js') }}"></script>
-
-    <script type="text/template" id="js-template-leaderboard">
-        {{ include('AcmeLiftStuffBundle:Lift:leaderboard.dust.twig') }}
-    </script>
-
-    <script src="{{ asset('assets/js/SymfonyErrorUtil.js') }}"></script>
-    <script src="{{ asset('assets/js/ErrorList.js') }}"></script>
-    <script src="{{ asset('assets/js/RepForm.js') }}"></script>
-    <script src="{{ asset('assets/js/Leaderboard.js') }}"></script>
-    <script src="{{ asset('assets/js/jquery/jquery.userinfopopup.js') }}"></script>
-
-    <script>
-        jQuery(document).ready(function() {
-            var repForm = new RepForm($('form.rep-log-form'), eventEmitter);
-            var leaderboard = new Leaderboard(
-                $('#leaderboard'),
-                $('#js-template-leaderboard').html(),
-                {{ leaders|json_encode|raw }},
-                eventEmitter
-            );
-        });
-    </script>
+{% block requirejs %}
+    {{ include('::_requirejs.html.twig', {
+        module: 'app/lift'
+    }) }}
 {% endblock %}
diff --git a/web/assets/js/app/lift.js b/web/assets/js/app/lift.js
new file mode 100644
index 0000000..64d919f
--- /dev/null
+++ b/web/assets/js/app/lift.js
@@ -0,0 +1,16 @@
+/**
+ * Default module that's used on every page
+ */
+define(['jquery', 'eventEmitter'], function ($, EventEmitter) {
+    var emitter = new EventEmitter();
+
+    jQuery(document).ready(function() {
+        var repForm = new RepForm($('form.rep-log-form'), emitter);
+        var leaderboard = new Leaderboard(
+            $('#leaderboard'),
+            $('#js-template-leaderboard').html(),
+            {},
+            emitter
+        );
+    });
+});
\ No newline at end of file
