diff --git a/web/assets/js/common.js b/web/assets/js/common.js
index 19254cf..da1ad4f 100644
--- a/web/assets/js/common.js
+++ b/web/assets/js/common.js
@@ -14,7 +14,8 @@ requirejs.config({
          * shim (beyond what you see for the bootstrap shim).
          */
         jquery: '../vendor/jquery/jquery',
-        bootstrap: '../vendor/bootstrap/dist/js/bootstrap'
+        bootstrap: '../vendor/bootstrap/dist/js/bootstrap',
+        eventEmitter: '../vendor/eventEmitter/EventEmitter'
     },
     shim: {
         /**
