diff --git a/assets/css/main.scss b/assets/css/main.scss
index 977dd99..21794ee 100644
--- a/assets/css/main.scss
+++ b/assets/css/main.scss
@@ -1,3 +1,6 @@
+@import '~bootstrap-sass/assets/stylesheets/_bootstrap.scss';
+@import '~font-awesome/css/font-awesome.css';
+
 .main-content {
     margin-top: 80px;
 }
diff --git a/assets/js/layout.js b/assets/js/layout.js
index 997dc3d..d0cd111 100644
--- a/assets/js/layout.js
+++ b/assets/js/layout.js
@@ -4,9 +4,6 @@ const $ = require('jquery');
 require('bootstrap-sass');
 // make sure the polyfill library is loaded in this main entry
 require('babel-polyfill');
-
-require('bootstrap-sass/assets/stylesheets/_bootstrap.scss');
-require('font-awesome/css/font-awesome.css');
 require('../css/main.scss');
 
 $(document).ready(function() {
