diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index bd00783a..bb82ccfe 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -66,7 +66,9 @@ export default class RepLogApp extends Component {
                 this.setSuccessMessage('Rep Log Saved!');
             })
             .catch(error => {
-                console.log(error.response);
+                error.response.json().then(errorsData => {
+                    console.log(errorsData);
+                })
             })
         ;
     }
