diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index bb82ccfe..3bc91407 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -14,7 +14,8 @@ export default class RepLogApp extends Component {
             numberOfHearts: 1,
             isLoaded: false,
             isSavingNewRepLog: false,
-            successMessage: ''
+            successMessage: '',
+            newRepLogValidationErrorMessage: ''
         };
         this.clearSuccessMessageTimeout = 0;
 
@@ -60,6 +61,7 @@ export default class RepLogApp extends Component {
                     return {
                         repLogs: newRepLogs,
                         isSavingNewRepLog: false,
+                        newRepLogValidationErrorMessage: '',
                     };
                 });
 
@@ -67,7 +69,12 @@ export default class RepLogApp extends Component {
             })
             .catch(error => {
                 error.response.json().then(errorsData => {
-                    console.log(errorsData);
+                    const errors = errorsData.errors;
+                    const firstError = errors[Object.keys(errors)[0]];
+
+                    this.setState({
+                        newRepLogValidationErrorMessage: firstError
+                    });
                 })
             })
         ;
diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index ee2e47ea..b2e0748e 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -53,9 +53,15 @@ export default class RepLogCreator extends Component {
 
     render() {
         const { quantityInputError } = this.state;
+        const { validationErrorMessage } = this.props;
 
         return (
             <form onSubmit={this.handleFormSubmit}>
+                {validationErrorMessage ? (
+                    <div className="alert alert-danger">
+                        {validationErrorMessage}
+                    </div>
+                ) : null}
                 <div className="form-group">
                     <label className="sr-only control-label required"
                            htmlFor="rep_log_item">
@@ -97,4 +103,5 @@ export default class RepLogCreator extends Component {
 
 RepLogCreator.propTypes = {
     onAddRepLog: PropTypes.func.isRequired,
+    validationErrorMessage: PropTypes.string.isRequired,
 };
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index ce10dfa2..5f5ee84d 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -27,7 +27,8 @@ export default function RepLogs(props) {
         onDeleteRepLog,
         isLoaded,
         isSavingNewRepLog,
-        successMessage
+        successMessage,
+        newRepLogValidationErrorMessage
     } = props;
 
     let heart = '';
@@ -87,6 +88,7 @@ export default function RepLogs(props) {
                 <div className="col-md-6">
                     <RepLogCreator
                         onAddRepLog={onAddRepLog}
+                        validationErrorMessage={newRepLogValidationErrorMessage}
                     />
                 </div>
             </div>
@@ -105,5 +107,6 @@ RepLogs.propTypes = {
     onDeleteRepLog: PropTypes.func.isRequired,
     isLoaded: PropTypes.bool.isRequired,
     isSavingNewRepLog: PropTypes.bool.isRequired,
-    successMessage: PropTypes.string.isRequired
+    successMessage: PropTypes.string.isRequired,
+    newRepLogValidationErrorMessage: PropTypes.string.isRequired,
 };
diff --git a/src/Form/Type/RepLogType.php b/src/Form/Type/RepLogType.php
index 296d24d4..494cadc3 100644
--- a/src/Form/Type/RepLogType.php
+++ b/src/Form/Type/RepLogType.php
@@ -17,7 +17,8 @@ class RepLogType extends AbstractType
             ->add('reps', IntegerType::class)
             ->add('item', ChoiceType::class, array(
                 'choices' => RepLog::getThingsYouCanLiftChoices(),
-                'placeholder' => 'What did you lift?'
+                'placeholder' => 'What did you lift?',
+                'invalid_message' => 'Please lift something that is understood by our scientists.'
             ))
         ;
     }
