diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index ebc22a3..b097ee3 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -98,7 +98,7 @@ export default class RepLogApp extends Component {
                         return repLog;
                     }
 
-                    return Object.assign({}, repLog, {isDeleting: true});
+                    return {...repLog, isDeleting: true};
                 })
             };
         });
