diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 0c90554..bd04dda 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -65,6 +65,9 @@ export default class RepLogApp extends Component {
 
                 this.setSuccessMessage('Rep Log Saved!');
             })
+            .catch(error => {
+                console.log(error.response);
+            })
         ;
     }
 
