diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 06d44bce5..ee2e47ea5 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -17,6 +17,7 @@ export default class RepLogCreator extends Component {
             { id: 'fat_cat', text: 'Big Fat Cat' },
             { id: 'laptop', text: 'My Laptop' },
             { id: 'coffee_cup', text: 'Coffee Cup' },
+            { id: 'invalid_item', text: 'Dark Matter' }
         ];
 
         this.handleFormSubmit = this.handleFormSubmit.bind(this);
diff --git a/assets/js/api/rep_log_api.js b/assets/js/api/rep_log_api.js
index b92e8e3c5..3aa8e0005 100644
--- a/assets/js/api/rep_log_api.js
+++ b/assets/js/api/rep_log_api.js
@@ -2,6 +2,7 @@ function fetchJson(url, options) {
     return fetch(url, Object.assign({
         credentials: 'same-origin',
     }, options))
+        .then(checkStatus)
         .then(response => {
             // decode JSON, but avoid problems with empty responses
             return response.text()
@@ -9,6 +10,17 @@ function fetchJson(url, options) {
         });
 }
 
+function checkStatus(response) {
+    if (response.status >= 200 && response.status < 400) {
+        return response;
+    }
+
+    const error = new Error(response.statusText);
+    error.response = response;
+
+    throw error
+}
+
 /**
  * Returns a promise where the data is the rep log collection
  *
