diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 6fa693c8..c8ee8c33 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -62,7 +62,6 @@ export default class RepLogApp extends Component {
 
                     return {
                         repLogs: newRepLogs,
-                        savingNewRepLog: false,
                         newRepLogValidationErrorMessage: '',
                     };
                 });
@@ -79,6 +78,11 @@ export default class RepLogApp extends Component {
                     });
                 })
             })
+            .finally(() => {
+                this.setState({
+                    savingNewRepLog: false
+                });
+            })
         ;
     }
 
