diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 0f7d4757..99654c15 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -14,7 +14,8 @@ export default class RepLogApp extends Component {
             numberOfHearts: 1,
             isLoaded: false,
             isSavingNewRepLog: false,
-            successMessage: ''
+            successMessage: '',
+            newRepLogValidationErrorMessage: ''
         };
 
         this.handleRowClick = this.handleRowClick.bind(this);
@@ -61,6 +62,7 @@ export default class RepLogApp extends Component {
                     return {
                         repLogs: newRepLogs,
                         isSavingNewRepLog: false,
+                        newRepLogValidationErrorMessage: '',
                     };
                 });
 
@@ -68,7 +70,12 @@ export default class RepLogApp extends Component {
             })
             .catch(error => {
                 error.response.json().then(errorsData => {
-                    console.log(errorsData);
+                    const errors = errorsData.errors;
+                    const firstError = errors[Object.keys(errors)[0]];
+
+                    this.setState({
+                        newRepLogValidationErrorMessage: firstError
+                    });
                 })
             })
         ;
diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 47638ce0..6e3d5b8b 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -53,6 +53,7 @@ export default class RepLogCreator extends Component {
 
     render() {
         const { quantityInputError } = this.state;
+        const { validationErrorMessage } = this.props;
 
         let quantityError = '';
         if (quantityInputError) {
@@ -61,6 +62,11 @@ export default class RepLogCreator extends Component {
 
         return (
             <form onSubmit={this.handleFormSubmit}>
+                {validationErrorMessage ? (
+                    <div className="alert alert-danger">
+                        {validationErrorMessage}
+                    </div>
+                ) : null}
                 <div className="form-group">
                     <label className="sr-only control-label required"
                            htmlFor="rep_log_item">
@@ -102,4 +108,5 @@ export default class RepLogCreator extends Component {
 
 RepLogCreator.propTypes = {
     onAddRepLog: PropTypes.func.isRequired,
+    validationErrorMessage: PropTypes.string.isRequired,
 };
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 7fddf0ee..f7387c6e 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -15,7 +15,7 @@ function calculateTotalWeightLifted(repLogs) {
 }
 
 export default function RepLogs(props) {
-    const { withHeart, highlightedRowId, onRowClick, repLogs, onAddRepLog, numberOfHearts, onHeartChange, onDeleteRepLog, isLoaded, isSavingNewRepLog, successMessage } = props;
+    const { withHeart, highlightedRowId, onRowClick, repLogs, onAddRepLog, numberOfHearts, onHeartChange, onDeleteRepLog, isLoaded, isSavingNewRepLog, successMessage, newRepLogValidationErrorMessage } = props;
 
     let heart = '';
     if (withHeart) {
@@ -74,6 +74,7 @@ export default function RepLogs(props) {
                 <div className="col-md-6">
                     <RepLogCreator
                         onAddRepLog={onAddRepLog}
+                        validationErrorMessage={newRepLogValidationErrorMessage}
                     />
                 </div>
             </div>
@@ -92,5 +93,6 @@ RepLogs.propTypes = {
     onDeleteRepLog: PropTypes.func.isRequired,
     isLoaded: PropTypes.bool.isRequired,
     isSavingNewRepLog: PropTypes.bool.isRequired,
-    successMessage: PropTypes.string.isRequired
+    successMessage: PropTypes.string.isRequired,
+    newRepLogValidationErrorMessage: PropTypes.string.isRequired,
 };
diff --git a/src/Form/Type/RepLogType.php b/src/Form/Type/RepLogType.php
index 296d24d4..494cadc3 100644
--- a/src/Form/Type/RepLogType.php
+++ b/src/Form/Type/RepLogType.php
@@ -17,7 +17,8 @@ class RepLogType extends AbstractType
             ->add('reps', IntegerType::class)
             ->add('item', ChoiceType::class, array(
                 'choices' => RepLog::getThingsYouCanLiftChoices(),
-                'placeholder' => 'What did you lift?'
+                'placeholder' => 'What did you lift?',
+                'invalid_message' => 'Please lift something that is understood by our scientists.'
             ))
         ;
     }
