diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 07aeaa80..55a973aa 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import RepLogList from './RepLogList';
+import RepLogs from './RepLogs';
 
 export default class RepLogApp extends Component {
     constructor(props) {
@@ -20,75 +21,6 @@ export default class RepLogApp extends Component {
         const { highlightedRowId } = this.state;
         const { withHeart } = this.props;
 
-        let heart = '';
-        if (withHeart) {
-            heart = <span>❤️</span>;
-        }
-
-        return (
-            <div className="col-md-7">
-                <h2>Lift Stuff! {heart}</h2>
-
-                <table className="table table-striped">
-                    <thead>
-                    <tr>
-                        <th>What</th>
-                        <th>How many times?</th>
-                        <th>Weight</th>
-                        <th>&nbsp;</th>
-                    </tr>
-                    </thead>
-                    <RepLogList
-                        highlightedRowId={highlightedRowId}
-                        onRowClick={this.handleRowClick}
-                    />
-                    <tfoot>
-                    <tr>
-                        <td>&nbsp;</td>
-                        <th>Total</th>
-                        <th>TODO</th>
-                        <td>&nbsp;</td>
-                    </tr>
-                    </tfoot>
-                </table>
-
-                <form className="form-inline">
-                    <div className="form-group">
-                        <label className="sr-only control-label required"
-                               htmlFor="rep_log_item">
-                            What did you lift?
-                        </label>
-                        <select id="rep_log_item"
-                                name="item"
-                                required="required"
-                                className="form-control">
-                            <option value="">What did you
-                                lift?
-                            </option>
-                            <option value="cat">Cat</option>
-                            <option value="fat_cat">Big Fat Cat</option>
-                            <option value="laptop">My Laptop</option>
-                            <option value="coffee_cup">Coffee Cup</option>
-                        </select>
-                    </div>
-                    {' '}
-                    <div className="form-group">
-                        <label className="sr-only control-label required"
-                               htmlFor="rep_log_reps">
-                            How many times?
-                        </label>
-                        <input type="number" id="rep_log_reps"
-                               name="reps" required="required"
-                               placeholder="How many times?"
-                               className="form-control"/>
-                    </div>
-                    {' '}
-                    <button type="submit" className="btn btn-primary">I Lifted
-                        it!
-                    </button>
-                </form>
-            </div>
-
-        );
+        return <RepLogs/>
     }
 }
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
new file mode 100644
index 00000000..918f5e64
--- /dev/null
+++ b/assets/js/RepLog/RepLogs.js
@@ -0,0 +1,73 @@
+import React from 'react';
+
+export default function RepLogs(props) {
+    let heart = '';
+    if (withHeart) {
+        heart = <span>❤️</span>;
+    }
+
+    return (
+        <div className="col-md-7">
+            <h2>Lift Stuff! {heart}</h2>
+
+            <table className="table table-striped">
+                <thead>
+                <tr>
+                    <th>What</th>
+                    <th>How many times?</th>
+                    <th>Weight</th>
+                    <th>&nbsp;</th>
+                </tr>
+                </thead>
+                <RepLogList
+                    highlightedRowId={highlightedRowId}
+                    onRowClick={this.handleRowClick}
+                />
+                <tfoot>
+                <tr>
+                    <td>&nbsp;</td>
+                    <th>Total</th>
+                    <th>TODO</th>
+                    <td>&nbsp;</td>
+                </tr>
+                </tfoot>
+            </table>
+
+            <form className="form-inline">
+                <div className="form-group">
+                    <label className="sr-only control-label required"
+                           htmlFor="rep_log_item">
+                        What did you lift?
+                    </label>
+                    <select id="rep_log_item"
+                            name="item"
+                            required="required"
+                            className="form-control">
+                        <option value="">What did you
+                            lift?
+                        </option>
+                        <option value="cat">Cat</option>
+                        <option value="fat_cat">Big Fat Cat</option>
+                        <option value="laptop">My Laptop</option>
+                        <option value="coffee_cup">Coffee Cup</option>
+                    </select>
+                </div>
+                {' '}
+                <div className="form-group">
+                    <label className="sr-only control-label required"
+                           htmlFor="rep_log_reps">
+                        How many times?
+                    </label>
+                    <input type="number" id="rep_log_reps"
+                           name="reps" required="required"
+                           placeholder="How many times?"
+                           className="form-control"/>
+                </div>
+                {' '}
+                <button type="submit" className="btn btn-primary">I Lifted
+                    it!
+                </button>
+            </form>
+        </div>
+    );
+}
\ No newline at end of file
