diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 55a973a..3db9f86 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -1,5 +1,4 @@
 import React, { Component } from 'react';
-import RepLogList from './RepLogList';
 import RepLogs from './RepLogs';
 
 export default class RepLogApp extends Component {
@@ -21,6 +20,12 @@ export default class RepLogApp extends Component {
         const { highlightedRowId } = this.state;
         const { withHeart } = this.props;
 
-        return <RepLogs/>
+        return (
+            <RepLogs
+                highlightedRowId={highlightedRowId}
+                withHeart={withHeart}
+                onRowClick={this.handleRowClick}
+            />
+        )
     }
 }
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 918f5e6..3935c27 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -1,6 +1,9 @@
 import React from 'react';
+import RepLogList from './RepLogList';
 
 export default function RepLogs(props) {
+    const { withHeart, highlightedRowId, onRowClick } = props;
+
     let heart = '';
     if (withHeart) {
         heart = <span>❤️</span>;
@@ -21,7 +24,7 @@ export default function RepLogs(props) {
                 </thead>
                 <RepLogList
                     highlightedRowId={highlightedRowId}
-                    onRowClick={this.handleRowClick}
+                    onRowClick={onRowClick}
                 />
                 <tfoot>
                 <tr>
