diff --git a/webpack.config.js b/webpack.config.js
index cd76420..f539b4e 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -4,11 +4,15 @@ const CopyWebpackPlugin = require('copy-webpack-plugin');
 
 const styleLoader = {
     loader: 'style-loader',
-    options: {}
+    options: {
+        sourceMap: true
+    }
 };
 const cssLoader = {
     loader: 'css-loader',
-    options: {}
+    options: {
+        sourceMap: true
+    }
 };
 const sassLoader = {
     loader: 'sass-loader',
@@ -18,7 +22,9 @@ const sassLoader = {
 };
 const resolveUrlLoader = {
     loader: 'resolve-url-loader',
-    options: {}
+    options: {
+        sourceMap: true
+    }
 };
 
 
