diff --git a/webpack.config.js b/webpack.config.js
index ddc1d2aa..0ae481c9 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -28,16 +28,16 @@ module.exports = {
             {
                 test: /\.css$/,
                 use: [
-                    'style-loader',
-                    'css-loader',
+                    'style-loader?sourceMap',
+                    'css-loader?sourceMap',
                 ]
             },
             {
                 test: /\.scss$/,
                 use: [
-                    'style-loader',
-                    'css-loader',
-                    'resolve-url-loader',
+                    'style-loader?sourceMap',
+                    'css-loader?sourceMap',
+                    'resolve-url-loader?sourceMap',
                     'sass-loader?sourceMap',
                 ]
             },
