diff --git a/webpack.config.js b/webpack.config.js
index e84f572..f9358b5 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -2,6 +2,26 @@ const path = require('path');
 const webpack = require('webpack');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 
+const styleLoader = {
+    loader: 'style-loader',
+    options: {}
+};
+const cssLoader = {
+    loader: 'css-loader',
+    options: {}
+};
+const sassLoader = {
+    loader: 'sass-loader',
+    options: {
+        sourceMap: true
+    }
+};
+const resolveUrlLoader = {
+    loader: 'resolve-url-loader',
+    options: {}
+};
+
+
 module.exports = {
     entry: {
         rep_log: './assets/js/rep_log.js',
@@ -28,17 +48,17 @@ module.exports = {
             {
                 test: /\.css$/,
                 use: [
-                    'style-loader',
-                    'css-loader',
+                    styleLoader,
+                    cssLoader,
                 ]
             },
             {
                 test: /\.scss$/,
                 use: [
-                    'style-loader',
-                    'css-loader',
-                    'resolve-url-loader',
-                    'sass-loader?sourceMap',
+                    styleLoader,
+                    cssLoader,
+                    resolveUrlLoader,
+                    sassLoader,
                 ]
             },
             {
