diff --git a/assets/js/Components/username_validation_error.js b/assets/js/Components/username_validation_error.js
new file mode 100644
index 0000000..ba1ebe4
--- /dev/null
+++ b/assets/js/Components/username_validation_error.js
@@ -0,0 +1,4 @@
+export default function($usernameInput) {
+    const $warning = $('<div class="login-long-username-warning">This is a really long username - are you sure that is right?</div>');
+    $usernameInput.before($warning);
+}
diff --git a/assets/js/login.js b/assets/js/login.js
index afb516d..102430a 100644
--- a/assets/js/login.js
+++ b/assets/js/login.js
@@ -2,6 +2,7 @@
 
 import $ from 'jquery';
 import '../css/login.css';
+import username_validation_error from './Components/username_validation_error';
 
 $(document).ready(function() {
     $('.js-recommended-login').on('click', '.js-show-login', function(e) {
@@ -16,8 +17,8 @@ $(document).ready(function() {
         $('.login-long-username-warning').remove();
 
         if ($usernameInput.val().length >= 20) {
-            const $warning = $('<div class="login-long-username-warning">This is a really long username - are you sure that is right?</div>');
-            $usernameInput.before($warning);
+            console.log(username_validation_error);
+            username_validation_error($usernameInput);
         }
     });
 });
