diff --git a/assets/js/login.js b/assets/js/login.js
index 102430a..0615f07 100644
--- a/assets/js/login.js
+++ b/assets/js/login.js
@@ -2,7 +2,6 @@
 
 import $ from 'jquery';
 import '../css/login.css';
-import username_validation_error from './Components/username_validation_error';
 
 $(document).ready(function() {
     $('.js-recommended-login').on('click', '.js-show-login', function(e) {
@@ -17,8 +16,11 @@ $(document).ready(function() {
         $('.login-long-username-warning').remove();
 
         if ($usernameInput.val().length >= 20) {
-            console.log(username_validation_error);
-            username_validation_error($usernameInput);
+            // use code splitting to lazily load this "chunk"
+            import('./Components/username_validation_error').then(username_validation_error => {
+                console.log(username_validation_error);
+                username_validation_error($usernameInput);
+            });
         }
     });
 });
