diff --git a/assets/js/login.js b/assets/js/login.js
index 0615f07..076a229 100644
--- a/assets/js/login.js
+++ b/assets/js/login.js
@@ -18,8 +18,7 @@ $(document).ready(function() {
         if ($usernameInput.val().length >= 20) {
             // use code splitting to lazily load this "chunk"
             import('./Components/username_validation_error').then(username_validation_error => {
-                console.log(username_validation_error);
-                username_validation_error($usernameInput);
+                username_validation_error.default($usernameInput);
             });
         }
     });
