diff --git a/play.js b/play.js
index ab67c0d..844f61b 100644
--- a/play.js
+++ b/play.js
@@ -1,7 +1,5 @@
-let printThreeThings = function(thing1, thing2, thing3) {
-    console.log(thing1, thing2, thing3);
-};
-
 let yummyThings = ['pizza', 'gelato', 'sushi', 'cheeseburger'];
 
-printThreeThings(...yummyThings);
+let greatThings = ['swimming', 'sunsets', ...yummyThings, 'New Orleans'];
+
+console.log(greatThings);
