diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index b600119b..9571bad3 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -23,15 +23,11 @@ export default class RepLogApp extends Component {
     }
 
     render() {
-        const { highlightedRowId, repLogs } = this.state;
-        const { withHeart } = this.props;
-
         return (
             <RepLogs
-                highlightedRowId={highlightedRowId}
-                withHeart={withHeart}
+                {...this.props}
+                {...this.state}
                 onRowClick={this.handleRowClick}
-                repLogs={repLogs}
             />
         )
     }
