diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 5054d52cd..8307956be 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -11,9 +11,10 @@ export default class RepLogApp extends Component {
 
     render() {
         const { highlightedRowId } = this.state;
+        const { withHeart } = this.props;
 
         let heart = '';
-        if (this.props.withHeart) {
+        if (withHeart) {
             heart = <span>❤️</span>;
         }
 
