diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 660269948..5054d52cd 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -1,7 +1,17 @@
 import React, { Component } from 'react';
 
 export default class RepLogApp extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            highlightedRowId: null
+        };
+    }
+
     render() {
+        const { highlightedRowId } = this.state;
+
         let heart = '';
         if (this.props.withHeart) {
             heart = <span>❤️</span>;
@@ -28,7 +38,10 @@ export default class RepLogApp extends Component {
                     </thead>
                     <tbody>
                     {repLogs.map((repLog) => (
-                        <tr key={repLog.id}>
+                        <tr
+                            key={repLog.id}
+                            className={highlightedRowId === repLog.id ? 'info' : ''}
+                        >
                             <td>{repLog.itemLabel}</td>
                             <td>{repLog.reps}</td>
                             <td>{repLog.totalWeightLifted}</td>
