diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index b89bf601..dfcffbb3 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -9,6 +9,10 @@ export default class RepLogApp extends Component {
         };
     }
 
+    handleRowClick(repLogId, event) {
+        this.setState({highlightedRowId: repLogId});
+    }
+
     render() {
         const { highlightedRowId } = this.state;
         const { withHeart } = this.props;
@@ -42,7 +46,7 @@ export default class RepLogApp extends Component {
                         <tr
                             key={repLog.id}
                             className={highlightedRowId === repLog.id ? 'info' : ''}
-                            onClick={(event) => this.setState({highlightedRowId: repLog.id})}
+                            onClick={(event) => this.handleRowClick(repLog.id, event)}
                         >
                             <td>{repLog.itemLabel}</td>
                             <td>{repLog.reps}</td>
