diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 30bef582..67508fc7 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -22,6 +22,16 @@ export default class RepLogApp extends Component {
         this.setState({highlightedRowId: repLogId});
     }
 
+    calculateTotalWeightLifted() {
+        let total = 0;
+
+        for (let repLog of this.state.repLogs) {
+            total += repLog.totalWeightLifted;
+        }
+
+        return total;
+    }
+
     render() {
         const { highlightedRowId, repLogs } = this.state;
 
@@ -52,7 +62,7 @@ export default class RepLogApp extends Component {
                     <tr>
                         <td>&nbsp;</td>
                         <th>Total</th>
-                        <th>TODO</th>
+                        <th>{this.calculateTotalWeightLifted()}</th>
                         <td>&nbsp;</td>
                     </tr>
                     </tfoot>
