diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 050c38b17..7eb2ef86a 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -42,7 +42,7 @@ export default class RepLogApp extends Component {
                         <tr
                             key={repLog.id}
                             className={highlightedRowId === repLog.id ? 'info' : ''}
-                            onClick={() => console.log('OMG an onClick!') }
+                            onClick={() => this.setState({highlightedRowId: repLog.id}) }
                         >
                             <td>{repLog.itemLabel}</td>
                             <td>{repLog.reps}</td>
