diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index b66f94e0..77469373 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -1,7 +1,17 @@
 import React, { Component } from 'react';
 
 export default class RepLogApp extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            highlightedRowId: null
+        };
+    }
+
     render() {
+        const { highlightedRowId } = this.state;
+
         let heart = '';
         if (this.props.withHeart) {
             heart = <span>❤️</span>;
@@ -15,7 +25,7 @@ export default class RepLogApp extends Component {
 
         const repLogElements = repLogs.map((repLog) => {
             return (
-                <tr key={repLog.id}>
+                <tr key={repLog.id} className={highlightedRowId === repLog.id ? 'info' : ''}>
                     <td>{repLog.itemLabel}</td>
                     <td>{repLog.reps}</td>
                     <td>{repLog.totalWeightLifted}</td>
