diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index c2136e73..d8373e14 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -2,6 +2,16 @@ import React from 'react';
 import RepLogList from './RepLogs';
 import PropTypes from 'prop-types';
 
+function calculateTotalWeightLifted(repLogs) {
+    let total = 0;
+
+    for (let repLog of repLogs) {
+        total += repLog.totalWeightLifted;
+    }
+
+    return total;
+}
+
 export default function RepLogs(props) {
     const { withHeart, highlightedRowId, onRowClick, repLogs } = props;
 
@@ -32,7 +42,7 @@ export default function RepLogs(props) {
                 <tr>
                     <td>&nbsp;</td>
                     <th>Total</th>
-                    <th>TODO</th>
+                    <th>{calculateTotalWeightLifted(repLogs)}</th>
                     <td>&nbsp;</td>
                 </tr>
                 </tfoot>
