diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index b6f8cbc..a4a3e16 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -66,20 +66,4 @@
             var repLogApp = new RepLogApp($wrapper);
         });
     </script>
-
-    <script type="text/template" id="js-rep-log-row-template">
-        <tr data-weight="<%= totalWeightLifted %>">
-            <td><%= itemLabel %></td>
-            <td><%= reps %></td>
-            <td><%= totalWeightLifted %></td>
-            <td>
-                <a href="#"
-                   class="js-delete-rep-log"
-                   data-url="<%= links._self %>"
-                >
-                    <span class="fa fa-trash"></span>
-                </a>
-            </td>
-        </tr>
-    </script>
 {% endblock %}
diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 9ca8a8f..69f62c6 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -1,6 +1,7 @@
 'use strict';
 
 (function(window, $, Routing, swal) {
+
     class RepLogApp {
         constructor($wrapper) {
             this.$wrapper = $wrapper;
@@ -169,7 +170,7 @@
             // let {id, itemLabel, reps, totallyMadeUpKey = 'whatever!'} = repLog;
             // console.log(id, itemLabel, reps, totallyMadeUpKey);
 
-            const tplText = $('#js-rep-log-row-template').html();
+            const tplText = rowTemplate;
             const tpl = _.template(tplText);
 
             const html = tpl(repLog);
@@ -213,7 +214,7 @@
         }
     }
 
-    const rowTemplate = '
+    const rowTemplate = `
 <tr data-weight="<%= totalWeightLifted %>">
     <td><%= itemLabel %></td>
     <td><%= reps %></td>
@@ -227,7 +228,7 @@
         </a>
     </td>
 </tr>
-';
+`;
 
     window.RepLogApp = RepLogApp;
 })(window, jQuery, Routing, swal);
