diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 1998b33..f3bd6ab 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -211,7 +211,13 @@
         }
     }
 
-    const rowTemplate = (repLog) => `
+    function upper(template, ...expressions) {
+        return template.reduce((accumulator, part, i) => {
+            return accumulator + (expressions[i - 1].toUpperCase ? expressions[i - 1].toUpperCase() : expressions[i - 1]) + part
+        })
+    }
+
+    const rowTemplate = (repLog) => upper`
 <tr data-weight="${repLog.totalWeightLifted}">
     <td>${repLog.itemLabel}</td>
     <td>${repLog.reps}</td>
