diff --git a/app/Resources/views/base.html.twig b/app/Resources/views/base.html.twig
index 10e722a..e94e1f9 100644
--- a/app/Resources/views/base.html.twig
+++ b/app/Resources/views/base.html.twig
@@ -91,6 +91,7 @@
 {% block javascripts %}
     <script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js"></script>
 {% endblock %}
 
 </body>
diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 7b403e9..99ab475 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -66,4 +66,20 @@
             var repLogApp = new RepLogApp($wrapper);
         });
     </script>
+
+    <script type="text/template" id="js-rep-log-row-template">
+        <tr data-weight="<%= totalWeightLifted %>">
+            <td><%= itemLabel %></td>
+            <td><%= reps %></td>
+            <td><%= totalWeightLifted %></td>
+            <td>
+                <a href="#"
+                   class="js-delete-rep-log"
+                   data-url="<%= links._self %>"
+                >
+                    <span class="fa fa-trash"></span>
+                </a>
+            </td>
+        </tr>
+    </script>
 {% endblock %}
diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 2fc2505..6a3ef10 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -120,7 +120,13 @@
         },
 
         _addRow: function(repLog) {
-            console.log(repLog);
+            var tplText = $('#js-rep-log-row-template').html();
+            var tpl = _.template(tplText);
+
+            var html = tpl(repLog);
+            this.$wrapper.find('tbody').append($.parseHTML(html));
+
+            this.updateTotalWeightLifted();
         }
     });
 
