diff --git a/src/AppBundle/Controller/LiftController.php b/src/AppBundle/Controller/LiftController.php
index 31a9f43..4ba1d5b 100644
--- a/src/AppBundle/Controller/LiftController.php
+++ b/src/AppBundle/Controller/LiftController.php
@@ -28,13 +28,6 @@ class LiftController extends BaseController
             $em->persist($repLog);
             $em->flush();
 
-            // return a blank form after success
-            if ($request->isXmlHttpRequest()) {
-                return $this->render('lift/_repRow.html.twig', [
-                    'repLog' => $repLog
-                ]);
-            }
-
             $this->addFlash('notice', 'Reps crunched!');
 
             return $this->redirectToRoute('lift');
@@ -48,15 +41,6 @@ class LiftController extends BaseController
             $totalWeight += $repLog->getTotalWeightLifted();
         }
 
-        // render just the form for AJAX, there is a validation error
-        if ($request->isXmlHttpRequest()) {
-            $html = $this->renderView('lift/_form.html.twig', [
-                'form' => $form->createView()
-            ]);
-
-            return new Response($html, 400);
-        }
-
         return $this->render('lift/index.html.twig', array(
             'form' => $form->createView(),
             'repLogs' => $repLogs,
