diff --git a/app/Resources/views/lift/_form.html.twig b/app/Resources/views/lift/_form.html.twig
index 92993e5..a98b268 100644
--- a/app/Resources/views/lift/_form.html.twig
+++ b/app/Resources/views/lift/_form.html.twig
@@ -1,4 +1,4 @@
-<form class="form-inline js-new-rep-log-form" novalidate>
+<form class="form-inline js-new-rep-log-form" novalidate data-url="{{ path('rep_log_new') }}">
     <div class="form-group">
         <label class="sr-only control-label required" for="rep_log_item">
             What did you lift?
diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index ce9b134..05ba124 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -63,19 +63,21 @@
             e.preventDefault();
 
             var $form = $(e.currentTarget);
-            var $tbody = this.$wrapper.find('tbody');
-            var self = this;
+            var formData = {};
+            $.each($form.serializeArray(), function(key, fieldData) {
+                formData[fieldData.name] = fieldData.value
+            });
             $.ajax({
-                url: $form.attr('action'),
+                url: $form.data('url'),
                 method: 'POST',
-                data: $form.serialize(),
+                data: JSON.stringify(formData),
                 success: function(data) {
-                    $tbody.append(data);
-                    self.updateTotalWeightLifted();
+                    // todo
+                    console.log('success!');
                 },
                 error: function(jqXHR) {
-                    $form.closest('.js-new-rep-log-form-wrapper')
-                        .html(jqXHR.responseText);
+                    // todo
+                    console.log('error :(');
                 }
             });
         }
