diff --git a/src/Acme/LiftStuffBundle/Controller/DefaultController.php b/src/Acme/LiftStuffBundle/Controller/DefaultController.php
index b92eccc..6eec138 100644
--- a/src/Acme/LiftStuffBundle/Controller/DefaultController.php
+++ b/src/Acme/LiftStuffBundle/Controller/DefaultController.php
@@ -23,7 +23,8 @@ class DefaultController extends Controller
      * @Route(
      *      "/users/{username}.{_format}",
      *      name="user_show",
-     *      requirements= {"_format": "json"}
+     *      requirements= {"_format": "json"},
+     *      options={"expose": true}
      * )
      */
     public function userAction($username)
diff --git a/web/assets/js/jquery/jquery.userinfopopup.js b/web/assets/js/jquery/jquery.userinfopopup.js
index 379dd28..ffcfdfc 100644
--- a/web/assets/js/jquery/jquery.userinfopopup.js
+++ b/web/assets/js/jquery/jquery.userinfopopup.js
@@ -27,12 +27,16 @@
      */
     var app = function(element, options) {
         this.$element = $(element);
+        this.options = $.extend({}, $.fn.userInfoPopup.defaults, options);
 
         this.initialize();
     };
 
     $.extend(app.prototype, {
         initialize: function() {
+            this.$element.on('mouseover', this.options.selector, $.proxy(this._handleMouseOver, this));
+            this.$element.on('mouseout', this.options.selector, $.proxy(this._handleMouseOut, this));
+
             this.$element.on('shown.bs.popover', this.options.selector, $.proxy(this._handlePopoverShown, this));
             this.$element.on('hidden.bs.popover', this.options.selector, $.proxy(this._handlePopoverHidden, this));
         },
@@ -78,9 +82,6 @@
 
             var username = $.trim($el.text());
 
-            // temporarily just stopping here
-            return username;
-
             var html = $el.data('userinfo.html');
             if (html) {
                 return html;
@@ -113,6 +114,16 @@
             return 'Loading...';
         },
 
+        _handleMouseOver: function(e) {
+            var $target = $(e.currentTarget);
+            this.show($target);
+        },
+
+        _handleMouseOut: function(e) {
+            var $target = $(e.currentTarget);
+            this.hide($target);
+        },
+
         /**
          * Just listens to popover so we know later if we're in a "shown" state or not
          */
