diff --git a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
index c3fbde3..7f9c992 100644
--- a/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
+++ b/src/Acme/LiftStuffBundle/Resources/views/Lift/index.html.twig
@@ -49,6 +49,7 @@
     <script src="{{ asset('assets/js/ErrorList.js') }}"></script>
     <script src="{{ asset('assets/js/RepForm.js') }}"></script>
     <script src="{{ asset('assets/js/Leaderboard.js') }}"></script>
+    <script src="{{ asset('assets/js/jquery/jquery.userinfopopup.js') }}"></script>
 
     <script>
         jQuery(document).ready(function() {
diff --git a/src/Acme/LiftStuffBundle/Resources/views/Lift/leaderboard.dust.twig b/src/Acme/LiftStuffBundle/Resources/views/Lift/leaderboard.dust.twig
index 6025732..7a2c413 100644
--- a/src/Acme/LiftStuffBundle/Resources/views/Lift/leaderboard.dust.twig
+++ b/src/Acme/LiftStuffBundle/Resources/views/Lift/leaderboard.dust.twig
@@ -10,7 +10,7 @@
     <tbody>
     {{ '{#leaders}' }}
         <tr>
-            <td>{username}</td>
+            <td class="js-username">{username}</td>
             <td style="width: 175px;">
                 {weight}lbs
 
diff --git a/web/assets/js/Leaderboard.js b/web/assets/js/Leaderboard.js
index 2278f86..db29a37 100644
--- a/web/assets/js/Leaderboard.js
+++ b/web/assets/js/Leaderboard.js
@@ -17,8 +17,10 @@ window.Leaderboard = (function($, dust, Routing, undefined){
             this.render();
 
             this.$wrapper.on('click', '.js-leaderboard-update', $.proxy(this._handleUpdateLeaderboard, this));
-
             this.eventEmitter.addListener('reps.update', $.proxy(this._handleRepsUpdate, this));
+
+            // trigger the userInfoBox behavior
+            this.$wrapper.find('.js-username').userInfoPopup();
         },
 
         /**
diff --git a/web/assets/js/jquery/jquery.userinfopopup.js b/web/assets/js/jquery/jquery.userinfopopup.js
new file mode 100644
index 0000000..80822a3
--- /dev/null
+++ b/web/assets/js/jquery/jquery.userinfopopup.js
@@ -0,0 +1,21 @@
+(function($, Routing, undefined) {
+
+    $.fn.userInfoPopup = function() {
+
+        this.each( function() {
+            var self = this;
+            var username = $.trim($(this).text());
+
+            $(this).on('mouseover', function() {
+                $(this).data('original', $(this).html());
+
+                $(this).html('Hallo!');
+            });
+
+            $(this).on('mouseout', function() {
+                $(this).html($(this).data('original'));
+            });
+        });
+    }
+
+}(jQuery, Routing));
\ No newline at end of file
