diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index e9171fa6..1824bd41 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -51,6 +51,11 @@ export default class RepLogCreator extends Component {
     }
 
     render() {
+        let quantityError = '';
+        if (this.state.quantityInputError) {
+            quantityError = <span className="help-block">{this.state.quantityInputError}</span>
+        }
+
         return (
             <form className="form-inline" onSubmit={this.handleSubmit}>
                 <div className="form-group">
@@ -81,6 +86,7 @@ export default class RepLogCreator extends Component {
                            required="required"
                            placeholder="How many times?"
                            className="form-control"/>
+                    {quantityError}
                 </div>
                 {' '}
                 <button type="submit" className="btn btn-primary">I Lifted
