diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 61827bc4..7744ba61 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -5,6 +5,10 @@ export default class RepLogCreator extends Component {
     constructor(props) {
         super(props);
 
+        this.state = {
+            quantityInputError: ''
+        };
+
         this.quantityInput = React.createRef();
         this.itemSelect = React.createRef();
 
@@ -26,7 +30,9 @@ export default class RepLogCreator extends Component {
         const itemSelect = this.itemSelect.current;
 
         if (quantityInput.value <= 0) {
-            // TODO - print some validation error!
+            this.setState({
+                quantityInputError: 'Please enter a value greater than 0'
+            });
 
             // don't submit, or clear the form
             return;
@@ -39,9 +45,14 @@ export default class RepLogCreator extends Component {
 
         quantityInput.value = '';
         itemSelect.selectedIndex = 0;
+        this.setState({
+            quantityInputError: ''
+        });
     }
 
     render() {
+        const { quantityInputError } = this.state;
+
         return (
             <form className="form-inline" onSubmit={this.handleFormSubmit}>
                 <div className="form-group">
@@ -62,7 +73,7 @@ export default class RepLogCreator extends Component {
                     </select>
                 </div>
                 {' '}
-                <div className="form-group">
+                <div className={`form-group ${quantityInputError ? 'has-error' : ''}`}>
                     <label className="sr-only control-label required"
                            htmlFor="rep_log_reps">
                         How many times?
