diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 47b40cd8..b2da46f7 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 
 export default class RepLogCreator extends Component {
     render() {
-        const { handleAddRepLog } = props;
+        const { handleAddRepLog } = this.props;
 
         let itemSelect = null;
         let quantityInput = null;
