diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 870aad0d..5e5cb495 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -59,7 +59,7 @@ export default class RepLogCreator extends Component {
         }
 
         return (
-            <form className="form-inline" onSubmit={this.handleFormSubmit}>
+            <form onSubmit={this.handleFormSubmit}>
                 <div className="form-group">
                     <label className="sr-only control-label required"
                            htmlFor="rep_log_item">
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index 65257534..c779eb59 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -54,7 +54,11 @@ export default class RepLogs extends Component {
                     </tfoot>
                 </table>
 
-                <RepLogCreator onAddRepLog={onAddRepLog}/>
+                <div className="row">
+                    <div className="col-md-6">
+                        <RepLogCreator onAddRepLog={onAddRepLog}/>
+                    </div>
+                </div>
             </div>
         );
     }
