diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 18363152..4a391dc5 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import RepLogList from './RepLogList';
-//import RepLogCreator from './RepLogCreator';
-import RepLogCreator from './RepLogCreatorControlledComponents';
+import RepLogCreator from './RepLogCreator';
+//import RepLogCreator from './RepLogCreatorControlledComponents';
 import uuid from 'uuid/v4';
 
 export default class RepLogApp extends Component {
