diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index cdd762f3c..3ebc2894f 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -54,7 +54,7 @@ export default class RepLogCreator extends Component {
         const { quantityInputError } = this.state;
 
         return (
-            <form className="form-inline" onSubmit={this.handleFormSubmit}>
+            <form onSubmit={this.handleFormSubmit}>
                 <div className="form-group">
                     <label className="sr-only control-label required"
                            htmlFor="rep_log_item">
diff --git a/assets/js/RepLog/RepLogs.js b/assets/js/RepLog/RepLogs.js
index e4bbcb411..e77b32702 100644
--- a/assets/js/RepLog/RepLogs.js
+++ b/assets/js/RepLog/RepLogs.js
@@ -56,9 +56,13 @@ export default function RepLogs(props) {
                 </tfoot>
             </table>
 
-            <RepLogCreator
-                onAddRepLog={onAddRepLog}
-            />
+            <div className="row">
+                <div className="col-md-6">
+                    <RepLogCreator
+                        onAddRepLog={onAddRepLog}
+                    />
+                </div>
+            </div>
         </div>
     );
 }
