diff --git a/assets/js/RepLog/RepLogApp.js b/assets/js/RepLog/RepLogApp.js
index 16e46cf0..3ef7108c 100644
--- a/assets/js/RepLog/RepLogApp.js
+++ b/assets/js/RepLog/RepLogApp.js
@@ -84,7 +84,11 @@ export default class RepLogApp extends Component {
                     </tfoot>
                 </table>
 
-                <RepLogCreator handleAddRepLog={this.handleAddRepLog} />
+                <div className="row">
+                    <div className="col-md-12">
+                        <RepLogCreator handleAddRepLog={this.handleAddRepLog} />
+                    </div>
+                </div>
             </div>
         );
     }
diff --git a/assets/js/RepLog/RepLogCreator.js b/assets/js/RepLog/RepLogCreator.js
index 1824bd41..0e7fa225 100644
--- a/assets/js/RepLog/RepLogCreator.js
+++ b/assets/js/RepLog/RepLogCreator.js
@@ -57,7 +57,7 @@ export default class RepLogCreator extends Component {
         }
 
         return (
-            <form className="form-inline" onSubmit={this.handleSubmit}>
+            <form onSubmit={this.handleSubmit}>
                 <div className="form-group">
                     <label className="sr-only control-label required"
                            htmlFor="rep_log_item">
