diff --git a/package.json b/package.json
index 5c9ba66..53edf61 100644
--- a/package.json
+++ b/package.json
@@ -29,6 +29,7 @@
     "style-loader": "^0.18.2",
     "sweetalert2": "^6.6.6",
     "webpack": "^3.4.1",
+    "webpack-chunk-hash": "^0.4.0",
     "webpack-dev-server": "^2.6.1",
     "webpack-manifest-plugin": "^1.2.1"
   },
diff --git a/webpack.config.js b/webpack.config.js
index cf98a43..51cc96a 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -3,6 +3,7 @@ const webpack = require('webpack');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const ManifestPlugin = require('webpack-manifest-plugin');
+const WebpackChunkHash = require('webpack-chunk-hash');
 
 const useDevServer = false;
 const useVersioning = true;
@@ -44,7 +45,7 @@ const webpackConfig = {
     },
     output: {
         path: path.resolve(__dirname, 'web', 'build'),
-        filename: useVersioning ? '[name].[hash:6].js' : '[name].js',
+        filename: useVersioning ? '[name].[chunkhash:6].js' : '[name].js',
         publicPath: publicPath,
     },
     module: {
@@ -136,6 +137,9 @@ const webpackConfig = {
             // always dump manifest
             writeToFileEmit: true
         }),
+
+        // allows for [chunkhash]
+        new WebpackChunkHash(),
     ],
     devtool: useSourcemaps ? 'inline-source-map' : false,
     devServer: {
diff --git a/yarn.lock b/yarn.lock
index 2d11ffa..1612cea 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -4374,6 +4374,10 @@ wbuf@^1.1.0, wbuf@^1.7.2:
   dependencies:
     minimalistic-assert "^1.0.0"
 
+webpack-chunk-hash@^0.4.0:
+  version "0.4.0"
+  resolved "https://registry.yarnpkg.com/webpack-chunk-hash/-/webpack-chunk-hash-0.4.0.tgz#6b40c3070fbc9ff0cfe0fe781c7174af6c7c16a4"
+
 webpack-dev-middleware@^1.11.0:
   version "1.11.0"
   resolved "https://registry.yarnpkg.com/webpack-dev-middleware/-/webpack-dev-middleware-1.11.0.tgz#09691d0973a30ad1f82ac73a12e2087f0a4754f9"
