diff --git a/webpack.config.js b/webpack.config.js
index 51cc96a..e8905da 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -140,6 +140,10 @@ const webpackConfig = {
 
         // allows for [chunkhash]
         new WebpackChunkHash(),
+
+        // keep module ids consistent between builds
+        // so that hashes doesn't suddenly change
+        isProduction ? new webpack.HashedModuleIdsPlugin() : new webpack.NamedModulesPlugin()
     ],
     devtool: useSourcemaps ? 'inline-source-map' : false,
     devServer: {
