diff --git a/webpack.config.js b/webpack.config.js
index b98e8c7..f9727d6 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -4,6 +4,7 @@ const CopyWebpackPlugin = require('copy-webpack-plugin');
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 const useDevServer = false;
+const useVersioning = true;
 const publicPath = useDevServer ? 'http://localhost:8080/build/' : '/build/';
 const isProduction = process.env.NODE_ENV === 'production';
 const useSourcemaps = !isProduction;
@@ -42,7 +43,7 @@ const webpackConfig = {
     },
     output: {
         path: path.resolve(__dirname, 'web', 'build'),
-        filename: '[name].js',
+        filename: useVersioning ? '[name].[hash:6].js' : '[name].js',
         publicPath: publicPath,
     },
     module: {
@@ -125,7 +126,9 @@ const webpackConfig = {
             minChunks: Infinity
         }),
 
-        new ExtractTextPlugin('[name].css'),
+        new ExtractTextPlugin(
+            useVersioning ? '[name].[contenthash:6].css' : '[name].css'
+        ),
     ],
     devtool: useSourcemaps ? 'inline-source-map' : false,
     devServer: {
