diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index adc1760e..a66e2e26 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -63,7 +63,7 @@ const Helper = require('./RepLogHelper');
             const $link = $(e.currentTarget);
 
             swal({
-                title: 'Delete this log?',
+                title: 'Delete this log???',
                 text: 'What? Did you not actually lift this?',
                 showCancelButton: true,
                 showLoaderOnConfirm: true,
