diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 961fe05..640f9be 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -8,7 +8,6 @@
         constructor($wrapper) {
             this.$wrapper = $wrapper;
             HelperInstances.set(this, new Helper($wrapper));
-            return;
 
             this.loadRepLogs();
 
@@ -231,11 +230,5 @@
 </tr>
 `;
 
-    new RepLogApp($('body'));
-    new RepLogApp($('body'));
-    new RepLogApp($('body'));
-    new RepLogApp($('body'));
-    console.log(HelperInstances);
-
     window.RepLogApp = RepLogApp;
 })(window, jQuery, Routing, swal);
