diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 8b3ba28..b8ed949 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -71,7 +71,7 @@
     <script src="{{ asset('assets/js/RepLogApp.js') }}"></script>
 
     <script>
-        console.log(window);
+        console.log(window === this);
         $(document).ready(function() {
             var $table = $('.js-rep-log-table');
             RepLogApp.initialize($table);
