diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index a160603..1bfb565 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -82,6 +82,7 @@
     <script src="{{ asset('assets/js/RepLogTable.js') }}"></script>
 
     <script>
+        console.log(window);
         $(document).ready(function() {
             var $table = $('.js-rep-log-table');
             RepLogTable.initialize($table);
diff --git a/web/assets/js/RepLogTable.js b/web/assets/js/RepLogTable.js
index 464d3f6..aa6f281 100644
--- a/web/assets/js/RepLogTable.js
+++ b/web/assets/js/RepLogTable.js
@@ -53,7 +53,7 @@
     /**
      * A "private" object
      */
-    var helper = {
+    helper = {
         initialize: function ($wrapper) {
             this.$wrapper = $wrapper;
         },
