diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index b8ed949..4213658 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -71,7 +71,6 @@
     <script src="{{ asset('assets/js/RepLogApp.js') }}"></script>
 
     <script>
-        console.log(window === this);
         $(document).ready(function() {
             var $table = $('.js-rep-log-table');
             RepLogApp.initialize($table);
diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 565dd21..0e0c44f 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -1,3 +1,5 @@
+'use strict';
+
 (function(window, $) {
     window.RepLogApp = {
         initialize: function ($wrapper) {
