diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 79e099a..2626354 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -86,6 +86,7 @@
     <script src="{{ asset('assets/js/RepLogApp.js') }}"></script>
 
     <script>
+        console.log(window);
         $(document).ready(function() {
             var $table = $('.js-rep-log-table');
             RepLogApp.initialize($table);
diff --git a/web/assets/js/RepLogApp.js b/web/assets/js/RepLogApp.js
index 1af351a..565dd21 100644
--- a/web/assets/js/RepLogApp.js
+++ b/web/assets/js/RepLogApp.js
@@ -53,7 +53,7 @@
     /**
      * A "private" object
      */
-    var Helper = {
+    Helper = {
         initialize: function ($wrapper) {
             this.$wrapper = $wrapper;
         },
