diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 66bb4a0..a160603 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -82,7 +82,6 @@
     <script src="{{ asset('assets/js/RepLogTable.js') }}"></script>
 
     <script>
-        console.log(window === this);
         $(document).ready(function() {
             var $table = $('.js-rep-log-table');
             RepLogTable.initialize($table);
diff --git a/web/assets/js/RepLogTable.js b/web/assets/js/RepLogTable.js
index aa6f281..a228abe 100644
--- a/web/assets/js/RepLogTable.js
+++ b/web/assets/js/RepLogTable.js
@@ -1,3 +1,5 @@
+'use strict';
+
 (function(window, $) {
     window.RepLogTable = {
         initialize: function ($table) {
